<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html id="nojs" xmlns="http://www.w3.org/1999/xhtml" xmlns:m="http://www.w3.org/1998/Math/MathML" xmlns:og="http://ogp.me/ns#" xml:lang="en-GB" lang="en-GB">

<head>

   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            
    <title>BMC Evolutionary Biology | Full text | A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes</title>
<link rel="shortcut icon" type="image/x-icon" href="/images/icon/1006/favicon.ico" />
<link rel="apple-touch-icon" sizes="72x72" href="/sites/1006/images/70.gif" />
<meta http-equiv="X-UA-Compatible" content="IE=7" />


                
    
		
		
		
		
<!-- <?xml version="1.0" encoding="UTF-8"?>
<rdf:RDF xmlns:cc="http://web.resource.org/cc/" xmlns="http://purl.org/rss/1.0/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:prism="http://prismstandard.org/namespaces/1.2/basic/">
<cc:Work rdf:about="http://www.biomedcentral.com/1471-2148/13/93">
<cc:license rdf:resource="http://creativecommons.org/licenses/by/2.0/"/>
</cc:Work>
<cc:License rdf:about="http://creativecommons.org/licenses/by/2.0/">
<cc:permits rdf:resource="http://web.resource.org/cc/Reproduction"/>
<cc:permits rdf:resource="http://web.resource.org/cc/Distribution"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Notice"/>
<cc:requires rdf:resource="http://web.resource.org/cc/Attribution"/>
<cc:permits rdf:resource="http://web.resource.org/cc/DerivativeWorks"/>
</cc:License>
<item rdf:about="http://www.biomedcentral.com/1471-2148/13/93">
<title>A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes</title>
<dc:title>A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes</dc:title>
<dc:creator>Pyron, R A</dc:creator>
<dc:creator>Burbrink, Frank T</dc:creator>
<dc:creator>Wiens, John J</dc:creator>
<dc:identifier>info:doi/10.1186/1471-2148-13-93</dc:identifier>
<dc:identifier>info:pmid/23627680</dc:identifier>
<dc:source>BMC Evolutionary Biology 2013, 13:93</dc:source>
<dc:date>2013-04-29</dc:date>

<prism:publicationName>BMC Evolutionary Biology</prism:publicationName>
<prism:publicationDate>2013-04-29</prism:publicationDate>
<prism:volume>13</prism:volume>
<prism:number>1</prism:number>
<prism:section>Research article</prism:section>
<prism:startingPage>93</prism:startingPage>
<prism:copyright>2013 Pyron et al.; licensee BioMed Central Ltd.</prism:copyright>
</item>
</rdf:RDF>
 -->
 
 
<meta name="citation_journal_title" content="BMC Evolutionary Biology" />
<meta name="citation_publisher" content="BioMed Central Ltd" />


<meta name="citation_author" content="R A Pyron" />
<meta name="citation_author_institution" content="Department of Biological Sciences, The George Washington University, 2023 G St. NW, Washington, DC 20052, USA" />
<meta name="citation_author" content="Frank T Burbrink" />
<meta name="citation_author_institution" content="Department of Biology, The College of Staten Island, The City University of New York, 2800 Victory Blvd., Staten Island, NY 10314, USA" />
<meta name="citation_author_institution" content="Department of Biology, The Graduate School and University Center, The City University of New York, 365 5th Ave., New York, NY 10016, USA" />
<meta name="citation_author" content="John J Wiens" />
<meta name="citation_author_institution" content="Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ 85721-0088, USA" />


<meta name="citation_issn" content="1471-2148" />
<meta name="citation_title" content="A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes" />
<meta name="citation_volume" content="13" />
<meta name="citation_issue" content="1" />
<meta name="citation_date" content="2013-04-29" />
<meta name="citation_firstpage" content="93" />
<meta name="citation_doi" content="10.1186/1471-2148-13-93" />
<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2148-13-93.pdf" />
<meta name="citation_pmid" content="23627680" />
<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2148-13-93.pdf" />
<meta name="title" content="A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes" />
<meta name="description" content="The extant squamates (&gt;9400 known species of lizards and snakes) are one of the most diverse and conspicuous radiations of terrestrial vertebrates, but no studies have attempted to reconstruct a phylogeny for the group with large-scale taxon sampling. Such an estimate is invaluable for comparative evolutionary studies, and to address their classification. Here, we present the first large-scale phylogenetic estimate for Squamata." />

<meta name="dc.title" content="A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes" />
<meta name="dc.creator" content="Pyron, R A" />
<meta name="dc.creator" content="Burbrink, Frank T" />
<meta name="dc.creator" content="Wiens, John J" />
<meta name="dc.description" content="The extant squamates (&gt;9400 known species of lizards and snakes) are one of the most diverse and conspicuous radiations of terrestrial vertebrates, but no studies have attempted to reconstruct a phylogeny for the group with large-scale taxon sampling. Such an estimate is invaluable for comparative evolutionary studies, and to address their classification. Here, we present the first large-scale phylogenetic estimate for Squamata." />
<meta name="dc.source" content="BMC Evolutionary Biology 2013 13:93" />
<meta name="dc.format" content="text/html" />
<meta name="dc.publisher" content="BioMed Central Ltd" />
<meta name="dc.date" content="2013-04-29" />
<meta name="dc.type" content="Research article" />
<meta name="dc.identifier" content="10.1186/1471-2148-13-93" />
<meta name="dc.identifier" content="info:pmid/23627680" />
<meta name="dc.language" content="en" />
<meta name="dc.copyright" content="2013 Pyron et al.; licensee BioMed Central Ltd." />
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/" />
<meta name="dc.rightsAgent" content="reprints@biomedcentral.com" />

<meta name="prism.issn" content="1471-2148" />
<meta name="prism.publicationName" content="BMC Evolutionary Biology" />
<meta name="prism.publicationDate" content="2013-04-29" />
<meta name="prism.volume" content="13" />
<meta name="prism.number" content="1" />
<meta name="prism.section" content="Research article" />
<meta name="prism.startingPage" content="93" />
<meta name="prism.copyright" content="2013 Pyron et al.; licensee BioMed Central Ltd." />
<meta name="prism.rightsAgent" content="reprints@biomedcentral.com" />


        <meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2148/13/93/abstract" />
            <meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2148/13/93" />
            <link rel='alternate' type='text/xml' title='XML version' href='http://www.biomedcentral.com/content/download/xml/1471-2148-13-93.xml' />
            <link rel="image_src" href="http://www.biomedcentral.com/content/figures/1471-2148-13-93-toc.gif" />
    
    <link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-13-93-s1.txt" title="Additional file 1: Data File S1" />
<link rel="section" href="http://www.biomedcentral.com/content/download/supplementary/1471-2148-13-93-s2.docx" title="Additional file 2: Table S1" />

    
    
                    
    
<script type="text/javascript">
// must run before css
document.documentElement.id = "js";
</script>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.8.2/jquery.min.js"></script>

						
   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            



						


	                <link rel="stylesheet" type="text/css" href="/css/main-0.css" media="screen, print"/>
    
                <link rel="stylesheet" type="text/css" href="/css/plugins-0.css" media="screen, print"/>
    



    

			                <link rel="stylesheet" type="text/css" href="http://www.bmcimg.com/css/articles-0.css" media="screen, print"/>
    	

    

    <link rel="stylesheet" type="text/css" href="/bmcevolbiol/css/themes/1006.css" media="screen, print"/>


                <link rel="stylesheet" type="text/css" href="/css/media/print-0.css" media="print"/>
    



<!--[if IE]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie-0.css"/>
    <![endif]-->

<!--[if IE 7]>
                    <link rel="stylesheet" type="text/css" href="/css/hacks/ie7-0.css"/>
    <![endif]-->
			
<script type="text/javascript">
    //configuration
    OAS_url = '//oas.biomedcentral.com/RealMedia/ads/';

	// control of advert's position on page [AMC]
    
    	
	
    OAS_sitepage = "biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93";
    	
		OAS_listpos = 'Top,x96,Bottom,Right3';
	
	
			
		OAS_query = '';
		
	

    OAS_target = '_top';
    //end of configuration
    OAS_version = 10;
    OAS_rn = '001234567890'; OAS_rns = '1234567890';
    OAS_rn = new String (Math.random()); OAS_rns = OAS_rn.substring (2, 11);
    function OAS_NORMAL(pos) {
		document.write('<a href="' + OAS_url + 'click_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" target="' + OAS_target + '">');
		document.write('<img src="' + OAS_url + 'adstream_nx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '!' + pos + '?' + OAS_query + '" border="0"/></a>');
    }
    
	function refererTerms(referer) {

	if(referer.match("q=") != null) {
		referer = referer.split("q=");
		referer = referer[1].split("&");
		referer = referer[0];
	} else {
		referer = referer;
	}

		if(referer.match("%20") != null && referer.match("%20").length >= 1) {
			referer =  referer.replace(/%20/gi,"+");
			return referer;
		} else { 
			return referer; 
		}
	}
 </script>

 
 <div id="oas-campaign" style="display:none">biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93</div>
 <div id="oas-positions" style="display:none">Bottom,Top</div> 


 <script type="text/javascript">
	// Set the version of JavaScript to 10 for a Browser 'Mozilla/3' on webTV 
    OAS_version = 11;
    if ((navigator.userAgent.indexOf('Mozilla/3') != -1) || (navigator.userAgent.indexOf('Mozilla/4.0 WebTV') != -1)) {
     OAS_version = 10; }
    if (OAS_version >= 11) {
    document.write('<scr' + 'ipt type="text/javascript" src="' + OAS_url + 'adstream_mjx.ads/' + OAS_sitepage + '/1' + OAS_rns + '@' + OAS_listpos + '?' + OAS_query + '"><\/script>'); }
</script>
	
<script type="text/javascript">
    document.write('');
	
    function OAS_AD(pos) {
        if (OAS_version >= 11)
         OAS_RICH(pos);
        else
         OAS_NORMAL(pos);
    }
 </script>
	





<script type="text/javascript">
    jsonLoaders = function(){
        var _loaders = [];
        return {
            register: function(config){
                config.prefix = config.prefix || '/webapi/1.0/blogs/';
                config.feedLength = config.feedLength || 3;

                _loaders.push(config);
            },
            list: function(){return _loaders}
        };
    }();

    function totext() {
	    document.write('<a href="javascript:returnToText();">Return to text</a>');
    }

    function retrieve_doi(){
        var metas = document.getElementsByTagName('meta');
        var i;
        for (i = 0; i < metas.length; i++)
            if (metas[i].getAttribute('name') == "citation_doi")
                break;
        var citation_doi = metas[i].getAttribute('content');
        return citation_doi;
    }

</script>




        <link title="Latest articles" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/latest/rss" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Most viewed" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/mostviewed/rss/" type="application/rss+xml" exposeToHeader="true"/>
    <link title="Latest comments" rel="alternate" href="http://www.biomedcentral.com/bmcevolbiol/latestcomments/rss" type="application/rss+xml" exposeToHeader="true"/>
    


	<script type="text/javascript">
	var _gaq = _gaq || [];
	_gaq.push(['_setAccount', 'UA-72326-1']);
	_gaq.push(['_gat._anonymizeIp']);
	_gaq.push(['_setLocalRemoteServerMode']);
	
	/*for cross domain tracking*/
	//_gaq.push(['_setAllowLinker', true]);
    //_gaq.push(['_setDomainName', 'biomedcentral.com/bmcevolbiol']); //this journal's domain
	/**/
	
		_gaq.push(['_trackPageview']); 

	(function() {
		var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
		ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
		var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);

		//var iframe = document.getElementById('casesdb-search-widget');
		//if(iframe !== null) {
		//	var pageTracker = _gat._getTrackerByName();
		//	iframe.src = pageTracker._getLinkerUrl('http://www.casesdatabase.com/');
		//}
	})();
	</script>

	
        </head>


<body class=" theme    ">

                                                



   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            

                        
<div id="branding" role="banner">
                			
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Top' begin -->
		<script type="text/javascript">
		OAS_AD('Top');

		</script>
		<!-- OAS AD 'Top' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad wide noscript">  
    <dt class="hide"></dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/12896078029@Top?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/12114979449@Top?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
			            <div class="sup-panel-outer">
        <div class="sup-panel-inner">

            <style>
  #nojs li.greeting, #nojs li#loginLink {display:none;}
</style>


<ul id="login" class="login">

    <noscript><li><a href="/log"><img class="greeting" src="/images/log.gif"/></a></li></noscript>
    
    <li id="welcome"><span id="username"></span></li>

    <li id="loginLink"><a href=""></a></li>
    
    
</ul>

<script>

function login() {

/* cookie getter/setter */
var allCookies = {
  getItem: function (sKey) {
    if (!sKey || !this.hasItem(sKey)) { return null; }
    return unescape(document.cookie.replace(new RegExp("(?:^|.*;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=\\s*((?:[^;](?!;))*[^;]?).*"), "$1"));
  },
  setItem: function (sKey, sValue, vEnd, sPath, sDomain, bSecure) {
    if (!sKey || /^(?:expires|max\-age|path|domain|secure)$/i.test(sKey)) { return; }
    var sExpires = "";
    if (vEnd) {
      switch (vEnd.constructor) {
        case Number:
          sExpires = vEnd === Infinity ? "; expires=Tue, 19 Jan 2038 03:14:07 GMT" : "; max-age=" + vEnd;
          break;
        case String:
          sExpires = "; expires=" + vEnd;
          break;
        case Date:
          sExpires = "; expires=" + vEnd.toGMTString();
          break;
      }
    }
    document.cookie = escape(sKey) + "=" + escape(sValue) + sExpires + (sDomain ? "; domain=" + sDomain : "") + (sPath ? "; path=" + sPath : "") + (bSecure ? "; secure" : "");
  },
  removeItem: function (sKey, sPath) {
    if (!sKey || !this.hasItem(sKey)) { return; }
    document.cookie = escape(sKey) + "=; expires=Thu, 01 Jan 1970 00:00:00 GMT" + (sPath ? "; path=" + sPath : "");
  },
  hasItem: function (sKey) {
    return (new RegExp("(?:^|;\\s*)" + escape(sKey).replace(/[\-\.\+\*]/g, "\\$&") + "\\s*\\=")).test(document.cookie);
  },
  keys: /* optional method: you can safely remove it! */ function () {
    var aKeys = document.cookie.replace(/((?:^|\s*;)[^\=]+)(?=;|$)|^\s*|\s*(?:\=[^;]*)?(?:\1|$)/g, "").split(/\s*(?:\=[^;]*)?;\s*/);
    for (var nIdx = 0; nIdx < aKeys.length; nIdx++) { aKeys[nIdx] = unescape(aKeys[nIdx]); }
    return aKeys;
  }
};

// converts cookie into js object
function parseCookie(cookie) {

    cookie = cookie.split("&");
    var obj = {};

    for(var i = 0; i < cookie.length; i++) {

        var c = cookie[i].split("=");

        c[0] = c[0].replace('"','');
        
        obj[c[0]] = c[1];

    }
    return obj;
}

    var cookie = allCookies.getItem("bmccookie");

    var obj = (cookie != null) ? parseCookie(cookie) : {};
    

        var welcome = "";

        if(obj.hasOwnProperty("fname")) {

            // insert user welcome
            welcome = obj.fname.replace(/\+/g, " ") + " " + obj['lname'].replace(/\+/g," ");

            if(jQuery("#welcome").length > 0) {
              jQuery("#welcome").prepend("<strong>Welcome</strong> ");
              jQuery("#username").prepend('<strong>' + welcome + '</strong><span class="divider"></span>');
            } 

            // update link url
            jQuery("#loginLink a").text("Log off");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logoff");

            window.bmcloggedon = true;


        } else if(obj.hasOwnProperty("institution_name")) {

            // insert institution name
            welcome = obj["institution_name"].replace(/\+/g," ");
            
            if(jQuery("#welcome").length > 0) { 
                jQuery("#welcome").prepend("<strong>Welcome</strong> ");
                jQuery("#username").prepend('<strong>' + welcome + '</strong><span class="divider"></span>');
            } 

            // update link url
            jQuery("#loginLink a").text("Log on");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logon");

            window.bmcloggedon = false;


        } else {
            jQuery("#welcome").innerHTML = "";
            jQuery("#loginLink a").text("Log on");
            jQuery("#loginLink a").attr("href", window.location.protocol + "//" + window.location.host + "/logon");

            window.bmcloggedon = false;

        }


        // if athens login cookie doesn't exist and the athens login link is visible then remove it.
        if(allCookies.hasItem("athens") != true && document.getElementById("loginAthensLink") != null) {
          jQuery("#loginAthensLink").hide();
        }

}

login();


</script>

            <ul class="nav-sup">

    
        <li id="BMC" class="current">
            <a href="http://www.biomedcentral.com/"><span class="img-title">&nbsp;</span>
            <span class="text-title">BioMed Central</span></a>
        </li>
		
        		<li><a href="http://www.biomedcentral.com/journals"><span>Journals</span></a></li>
		
								<li>
				<a href="http://www.biomedcentral.com/gateways"><span>Gateways</span></a>
			</li>
					    	
</ul>
        </div>
    </div>
    <div class="branding-inner">
                         

    
<style>
.survey {
	margin: 10px 0;
}
.survey, .survey .survey-sentence {
	font-size: 13px;
}
.survey-button a, .survey-button a:active, .survey-button a:visited, .survey-button a:hover  {
    color: #FFFFFF;
    text-decoration: none;
}
.survey-button {
	background-color: #FFA500;
    border-radius: 5px;
    color: #FFFFFF;
    font-weight: bold;
    font-size: 13px;
    font-weight: bold;
    padding: 5px 13px;
    background: #ef8142; /* Old browsers */
	background: -moz-linear-gradient(top,  #ef8142 0%, #ef8142 50%, #a64510 100%, #a64510 100%); /* FF3.6+ */
	background: -webkit-gradient(linear, left top, left bottom, color-stop(0%,#ef8142), color-stop(50%,#ef8142), color-stop(100%,#a64510), color-stop(100%,#a64510)); /* Chrome,Safari4+ */
	background: -webkit-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Chrome10+,Safari5.1+ */
	background: -o-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* Opera 11.10+ */
	background: -ms-linear-gradient(top,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* IE10+ */
	background: linear-gradient(to bottom,  #ef8142 0%,#ef8142 50%,#a64510 100%,#a64510 100%); /* W3C */
	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#ef8142', endColorstr='#a64510',GradientType=0 ); /* IE6-9 */

}
</style>




          <div class="logo">
        <a href="/bmcevolbiol/"><img alt="BMC Evolutionary Biology" title="BMC Evolutionary Biology Logo" src="/sites/1006/images/logo.gif" /></a>     
      </div>    
        
                    



            <div id="impact-factor" class="official">
    <img src="/images/branding/official.gif" alt="official impact factor" title="3.29"/>
    <span id="impact-factor-value">3.29</span>
</div>
    
                            

<div class="module search gray small  " role="search">
    <div class="module-inner padded">
        <form method="get" class="searchForm"
              action=" /quickSearchRedirectURL "
              onsubmit="window.location.href=this['url'].value + '?terms=' + encodeURIComponent(this['terms'].value); return false;">
            <fieldset class="search">
			                <span>
                    Search <select name="url">
                                <option value="http://www.biomedcentral.com/bmcevolbiol/search/results" selected>this journal</option>
                                <option value="http://www.biomedcentral.com/search/results">BioMed Central</option>
                           </select> for
                </span>
                            <input id="searchTerms" name="terms" type="text" alt="Search terms" class="text " />
                 <button class="w37" value="Go" type="submit">Go</button>
            </fieldset>
        </form>
    </div>
	<a class="advanced-search" href="/bmcevolbiol/search">Advanced search</a>
</div> 

                        <div id="mobile-nav-list" >

	<ul class="primary-nav" role="navigation">
		<li     ><a href="/bmcevolbiol/"><span id="home-tab">Home</span></a></li>
														<li             class="current"
    ><a href="/bmcevolbiol/content"><span id="articles-tab">Articles</span></a></li>
										
	

	
            <li     ><a href="/bmcevolbiol/authors/instructions"><span id="authors-tab">Authors</span></a></li>
        <li     ><a href="/bmcevolbiol/about/reviewers"><span id="reviewers-tab">Reviewers</span></a></li>
    		<li     ><a href="/bmcevolbiol/about"><span id='about-tab'>About this journal</span></a></li>
	<li     ><a href="/bmcevolbiol/my"><span id="my-tab">My BMC Evolutionary Biology</span></a></li>
		
    		</ul>
</div>
            </div>
</div>

<hr class="hide" />
<div class="container">
	<div id="content"  class="overflow-visible"  role="main">
			    
	

<!--div class="branding-mobile">
		<h1 class="logo">
	    <a href="/bmcevolbiol/">
		    <img alt="BMC Evolutionary Biology" title="BMC Evolutionary Biology Logo" src="/sites/1006/images/logo.gif" />
		</a>
		</h1>
	</div><br/-->



			
	



<div class="outline-wrapper block mobile-hidden" id="left-article-box">
<ul id="box-outline" class="box">
<ul id="outline" class="outline">
   <li>
      <a href="#">Top</a>
   </li>
   <li>
      <a href="#abs">Abstract</a>
   </li>
   <li>
      <a href="#sec1">Background</a>
   </li>
   <li>
      <a href="#sec2">Results</a>
   </li>
   <li>
      <a href="#sec3">Discussion</a>
   </li>
   <li>
      <a href="#sec4">Conclusions</a>
   </li>
   <li>
      <a href="#sec5">Methods</a>
   </li>
   <li>
      <a href="#sec6">Appendix I</a>
   </li>
   <li>
      <a href="#sec7">Squamata</a>
   </li>
   <li>
      <a href="#sec8">Competing interests</a>
   </li>
   <li>
      <a href="#sec9">Authorsâ€™ contributions</a>
   </li>
   <li>
      <a href="#ack">Acknowledgements</a>
   </li>
   <li class="lowest">
      <a href="#refs">References</a>
   </li>
</ul>
</ul>

<dl id="box-outline1">
</dl>


</div>



<div class="mobile-hidden" id="right-panel">
 	
<div id="mobile-sidebar">
	<div id="mobile-sidebar-tab">
		<span>
			<a href="#" id="mobile-sidebar-tab-link"><img src="/images/sidebar-hidden.png"><img style="display:none;" src="/images/sidebar-shown.png"></a>
		</span>
	</div>	<div id="mobile-sidebar-list">

<div id="article-navigation-bar">
    <div class="issue-information block">
        <h5><a href="http://www.biomedcentral.com/bmcevolbiol"><strong>BMC Evolutionary Biology</strong></a></h5>
 		        <ul class="square normal" id="section-name">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/sections/phylogenetics_and_phylogeography">Phylogenetics and phylogeography</a></li>
        </ul>
                <ul class="square normal">
            <li><a href="http://www.biomedcentral.com/bmcevolbiol/content/13">Volume 13</a></li>
				
                    </ul>
    </div>

    <div class="article-information block" id="article-info">
        


<div id="viewing-options-links">
	<h5>Viewing options</h5>
	<ul class="square normal">
        
            <li>
                             <a href="/1471-2148/13/93/abstract">Abstract</a>
                        </li>
        				<li>
							<strong>Full text</strong>
					</li>
				
		<li class="pdfFileSize">
			<a onclick="_gaq.push(['_trackEvent', 'PDF download', 'Article Sidebar', '/1471-2148/13/93', 1, true]);" href="/content/pdf/1471-2148-13-93.pdf">PDF</a>
			<span>(4.2MB)</span>
		</li>

                    <li>
                <a href="/content/epub/1471-2148-13-93.epub">ePUB</a>
                (3.7MB)
            </li>
        
				<li>
						<a href="/1471-2148/13/93/additional">Additional files</a>
					</li>
			</ul>
</div>
		
<div id="associated-material-links">
    <h5>Associated material</h5>
    <ul class="square normal">
                        <li>
            <a href="/pubmed/23627680">PubMed record</a>
        </li>
                            <li><a href="/1471-2148/13/93/about">Article metrics</a></li>
                <li>
                        <a href="/1471-2148/13/93/comments">Readers' comments </a>
                    </li>
        <li>
                    </li>
    </ul>
</div>
		




<div id="related-literature-links">
    <h5>Related literature</h5>
    <ul class="normal">
                    <li><a href="http://www.biomedcentral.com/1471-2148/13/93/about#citations">Cited by</a></li>
                <li id="google-blog-search"><a href="http://www.google.com/search?tbm=blg&hl=en&amp;source=hp&amp;biw=1280&amp;bih=899&amp;q=%22A+phylogeny+and+revised+classification+of+Squamata%2C+including+4161+species+of+lizards+and+snakes%22&amp;btnG=Search">Google blog search</a></li>
        <h6>Other articles by authors</h6>
        <li>
            <a class="collapser"><i>&nbsp;</i>on Google Scholar</a>
<ul id="authg" class="hidebeforeload">                         <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22RA+Pyron%22">Pyron RA</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22FT+Burbrink%22">Burbrink FT</a>
            </li>
                                <li>
                <a href="http://scholar.google.com/scholar?q=author%3A%22JJ+Wiens%22">Wiens JJ</a>
            </li>
            </ul>          </li>
                <li>
            		<a class="collapser"><i>&nbsp;</i>on PubMed</a>
	<ul id="authpm" class="hidebeforeload"> 									<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Pyron_RA%20[Author]">Pyron RA</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Burbrink_FT%20[Author]">Burbrink FT</a>
				</li>
												<li><a href="http://www.ncbi.nlm.nih.gov/pubmed?term=Wiens_JJ%20[Author]">Wiens JJ</a>
				</li>
						</ul > 	        </li>
    </ul>     <h6>Related articles/pages</h6>
    <ul class="square normal">
        <li><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2148/13/93">on Google</a></li>
        <li><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2148/13/93">on Google Scholar</a></li>
                    <li><a href="/pubmed/related/23627680">on PubMed</a></li>
            </ul>

</div>		
        
    
    
        

    	






<h5>Tools</h5>
	<ul class="square normal">
									<li>
						<a href="/1471-2148/13/93/citation">Download references</a>
					</li>
				<li><a href="/content/download/xml/1471-2148-13-93.xml">Download XML</a></li>
				<li>
						<a href="/1471-2148/13/93/email">Email to a friend</a>
					</li>
		<li><a href="https://www.odysseypress.com/onlinehost/reprint_order.php?type=A&page=0&journal=738&doi=10.1186/1471-2148-13-93&volume=13&issue=1&title=A+phylogeny+and+revised+classification+of+Squamata%2C+including+4161+species+of+lizards+and+snakes&author_name=R Alexander Pyron&start_page=1&end_page=54">Order reprints</a></li>
		<li>
						<a href="/1471-2148/13/93/postcomment">Post a comment</a>
					</li>			
		<ul id="download-to-links">
			<a class="downloadto_title btn" href="#"><span class="icon">&nbsp;</span>Download to ...</a>
			
			<ul id="downloadto" class="iconlist hidebeforeload" style="display: none;">
				<li>
					<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F13%2F93" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
				</li>
				<li>
					<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/13/93" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
				</li>
			</ul>
		</ul>
		
		<ul id="mobile-downloadto" class="iconlist">
		
		<h6>Download to ...</h6>
			<li>
				<a href="http://redirect.papersapp.com/redirect?url=http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F13%2F93" onclick="largepopup(this.href,'papers',800,600);return false"><span class="share-icons papers"></span>Papers</a>
			</li>
			
			<li>
				<a href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/13/93" onclick="largepopup(this.href,'mendeley',800,600);return false"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
		</ul>

		
	</ul>
		




<h5>Share this article</h5>
<ul id="social-networking-links">
	<li>
	<div class="fb-like" data-href="http://www.biomedcentral.com/1471-2148/13/93" data-send="false" data-layout="button_count" data-width="100" data-show-faces="false" data-action="recommend"></div>
	</li>
	<li>
		<a href="http://twitter.com/share" data-url="http://www.biomedcentral.com/1471-2148/13/93" data-counturl="http://www.biomedcentral.com/1471-2148/13/93" data-hashtags="bmcevolutionarybiology
" data-text="A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes" class="twitter-share-button">Tweet</a>
	</li>
	<li>
			<g:plusone size="medium" width="100"></g:plusone>
	</li>

		<a class="sharethis_title btn" href="#"><span class="icon">&nbsp;</span>More options...</a>
	<ul class="iconlist hidebeforeload" id="sharethis">
			<li>
				<a onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false" href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2148/13/93"><span class="share-icons citeulike"></span>Citeulike</a>
			</li>
				<li>
			<a onclick="largepopup(this.href,'linkedin',800,600);return false" href="http://www.linkedin.com/shareArticle?mini=true&url=%26%2335%3Bbmcevolutionarybiology%0D%0AA%20phylogeny%20and%20revised%20classification%20of%20Squamata%2C%20including%204161%20species%20of%20lizards%20and%20snakes%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F13%2F93&title=A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes"><span class="share-icons linkedin"></span>LinkedIn</a>  
	</li>
			<li>
				<a onclick="largepopup(this.href,'Del.icio.us',800,600);return false" href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2148/13/93&title=A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes"><span class="share-icons delicious"></span>Del.icio.us</a>
	</li>
			<li>
				<a href="/1471-2148/13/93/email"><span class="share-icons email"></span>Email</a>
	</li>
			<li>
				<a onclick="largepopup(this.href,'facebook',800,600);return false" href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2148/13/93"><span class="share-icons facebook"></span>Facebook</a>
	</li>
	<li class="gp">
			<div class="googlehider mobile-hide"> 
				<div class="g-plusone" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus mobile-hide"></span>
				<div class="googlehider2 mobile-hide"> 
				<div class="g-plusone mobile-hide" data-annotation="none"></div>
			</div>
				<span class="share-icons googleplus"></span>
				<span class="gp-link"><a href="#">Google+</a></span>
	</li>
			<li>
				<a onclick="largepopup(this.href,'mendeley',800,600);return false" href="http://www.mendeley.com/import/?url=http://www.biomedcentral.com/1471-2148/13/93"><span class="share-icons mendeley"></span>Mendeley</a>
			</li>
			<li>
			<a onclick="largepopup(this.href,'twitter',800,600);return false" href="http://twitter.com/?status=A%20phylogeny%20and%20revised%20classification%20of%20Squamata%2C%20including%204161%20species%20of%20lizards%20and%20snakes%20http%3A%2F%2Fwww.biomedcentral.com%2F1471-2148%2F13%2F93+%23bmcevolutionarybiology
"><span class="share-icons twitter"></span>Twitter</a>  
			</li>
			
		</ul>
</ul>

    </div>

    
					<div class="block block-buttons hide_nav submit-register-panel">
			<a class="left" href="http://biomedcentral.com/researchawards/nominate?doi=10.1186/1471-2148-13-93"><img src="/images/nominate.png" class="left" alt="Reasearch Awards nomination" title="Nominate this article for the next Research Awards"></a>
			</div>
				
    			
		














		                                    		                                    		
	

	    
   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            

<div id="signup-to-etoc" class="button-collection-margins block mobile-border-bottom">	
					<h3>Email updates</h3>
			<p id="receive_issue_alerts_journal_updates_text">Keep up to date with the latest news and content from BMC Evolutionary Biology and BioMed Central.</p>
			
				

<form  id="articleSignupForUpdates"  method="post" action="/bmcevolbiol/signuptoupdates.html">
	<div id="message-box"></div>
    <fieldset class="block-form">
        <input type="text" name="emailAddress" onblur="javascript: if (this.value == '') { this.value = 'email address'; }" onfocus="javascript: if (this.value == 'email address') { this.value = ''; }" value="email address"  class="text " id="email" />
        <input value="/bmcevolbiol" type="hidden" name="returnUrl" id="returnUrl" />
        <input value="" type="hidden" name="listName" id="listName" />
        <input value="BMC Evolutionary Biology" type="hidden" name="journalName" id="journalName" />
        <button class="w74 right" type="submit" onmouseover="this.className='w74 hover right'" onmouseout="this.className='w74 right'" value="Sign up">Sign up</button>
    </fieldset>
</form>	
</div>
	



			<div id="advert-offset">
					
		<dl class="google-ad  ">
	<dt class="hide">
				
					<a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
			</dt>
		<dd>

		<!-- OAS AD 'Right3' begin -->
		<script type="text/javascript">
		OAS_AD('Right3');

		</script>
		<!-- OAS AD 'Right3' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/18412849424@Right3?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/19414091656@Right3?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
					</div>
		
</div>
		<div class="mobile-sidebar-gradient top"></div>
		<div class="mobile-sidebar-gradient bottom"></div>
	</div>

</div>








 </div>
<div class="rounded custom white article full-text">
<div class="wrap-inner content">
    <div class="padded-inner">            


    
   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            

<div id="topmatter">
	


	   








				<a href="/about/access"><img alt="Open Access" src="/images/articles/openaccess-large.png" class="access mr15"/></a>
			<a href="http://www.biomedcentral.com/about/mostviewed/"><img alt="Highly Accessed" src="/images/articles/highlyaccessed-large.png" class="access mr15"/></a>
				<span class="articletype">Research article</span>

<h1>A phylogeny and revised classification of Squamata, including 4161 species of lizards and snakes</h1>
        
			<div class="multipleins">
		
	<p class="authors">
        																																																																																																								         <strong>R Alexander Pyron</strong><sup><a href="/1471-2148/13/93/#ins1">1</a></sup><sup>&#42;</sup>, <strong>Frank T Burbrink</strong><sup><a href="/1471-2148/13/93/#ins2">2</a></sup><sup><a href="/1471-2148/13/93/#ins3">3</a></sup> and <strong>John J Wiens</strong><sup><a href="/1471-2148/13/93/#ins4">4</a></sup>	</p>

			<div id="affiliations">
		<div class="module gray  inner">
<div class="module-inner padded-inner">
		<ul>
			<li>                        
                                                    		<p class="authors" >
							<span>*</span>
									 Corresponding author:										            R A Pyron <a href='mailto:rpyron@colubroid.org'>rpyron@colubroid.org</a>    					</p>
      
</li>
			
                                                                                                    		</ul>

		
		 <p class="options"><a class="affiliations-toggle" href="#"><i class="arrow"></i>Author Affiliations</a></p>            <section> 
						<div class="collapsible-content">
							
				<div id="ins_container" style="display: block;">
										    						    <p >
						        <sup id="ins1">1</sup>                                Department of Biological Sciences, The George Washington University, 2023 G St. NW, Washington, DC 20052, USA
                            </p>
																    						    <p >
						        <sup id="ins2">2</sup>                                Department of Biology, The Graduate School and University Center, The City University of New York, 365 5th Ave., New York, NY 10016, USA
                            </p>
																    						    <p >
						        <sup id="ins3">3</sup>                                Department of Biology, The College of Staten Island, The City University of New York, 2800 Victory Blvd., Staten Island, NY 10314, USA
                            </p>
																    						    <p >
						        <sup id="ins4">4</sup>                                Department of Ecology and Evolutionary Biology, University of Arizona, Tucson, AZ 85721-0088, USA
                            </p>
																
				</div>
										<p id="authoremails">            For all author emails, please <a href="/bmcevolbiol/logon">log on</a>.
    </p>
						</div>
			</section>
				</div>
</div>
		</div>
	  </div>

            <section class="cit">
                        <div class="collapsible-content">
        <p><em>BMC Evolutionary Biology</em> 2013, <strong>13</strong>:93&nbsp;
			<span class="pseudotab">doi:10.1186/1471-2148-13-93</span></p>
	    				    
    
        		<br/>
		<p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2148/13/93">http://www.biomedcentral.com/1471-2148/13/93</a></p>
		<br/>
                    <table cellspacing="0" cellpadding="0">
        <tbody>
        <tr><td>Received:</td><td>30 January 2013</td></tr>                <tr><td>Accepted:</td><td>19 March 2013</td></tr>        <tr><td>Published:</td><td>29 April 2013</td></tr>        </tbody>
        </table>
    		<!--<br/>-->
		<div  style="line-height:140%">
        <p>
            &copy; 2013 Pyron et al.; licensee BioMed Central Ltd. <br />
        </p>
					<p>
				This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href='http://creativecommons.org/licenses/by/2.0'>http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.
			</p>
		    	</div>
	</section>

</div>
<div id="article-body">
    <section><a name="abs"></a><h3>Abstract</h3>
   <div class="collapsible-content">
      <h4>Background</h4>
      <p style="line-height:160%">The extant squamates (&gt;9400 known species of lizards and snakes) are one of the most
         diverse and conspicuous radiations of terrestrial vertebrates, but no studies have
         attempted to reconstruct a phylogeny for the group with large-scale taxon sampling.
         Such an estimate is invaluable for comparative evolutionary studies, and to address
         their classification. Here, we present the first large-scale phylogenetic estimate
         for Squamata.
      </p>
      <h4>Results</h4>
      <p style="line-height:160%">The estimated phylogeny contains 4161 species, representing all currently recognized
         families and subfamilies. The analysis is based on up to 12896 base pairs of sequence
         data per species (average = 2497 bp) from 12 genes, including seven nuclear loci (BDNF,
         c-mos, NT3, PDC, R35, RAG-1, and RAG-2), and five mitochondrial genes (12S, 16S, cytochrome
         <em>b</em>, ND2, and ND4). The tree provides important confirmation for recent estimates of
         higher-level squamate phylogeny based on molecular data (but with more limited taxon
         sampling), estimates that are very different from previous morphology-based hypotheses.
         The tree also includes many relationships that differ from previous molecular estimates
         and many that differ from traditional taxonomy.
      </p>
      <h4>Conclusions</h4>
      <p style="line-height:160%">We present a new large-scale phylogeny of squamate reptiles that should be a valuable
         resource for future comparative studies. We also present a revised classification
         of squamates at the family and subfamily level to bring the taxonomy more in line
         with the new phylogenetic hypothesis. This classification includes new, resurrected,
         and modified subfamilies within gymnophthalmid and scincid lizards, and boid, colubrid,
         and lamprophiid snakes.
      </p>
   </div>
</section><span id="keywords">
   <h5 class="inline">Keywords: </h5>Amphisbaenia; Lacertilia; Likelihood support measures; Missing data; Serpentes; Squamata; Phylogenetics; Reptilia; Supermatrices; Systematics</span><section><a name="sec1"></a><h3>Background</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Squamate reptiles (lizards, snakes, and amphisbaenians ["worm lizards"]) are among
         the most diverse radiations of terrestrial vertebrates. Squamata includes more than
         9400 species as of December 2012 <a name="d6347e173"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The rate of new species descriptions shows no signs of slowing, with a record 168
         new species described in 2012 <a name="d6347e177"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], greater than the highest yearly rates of the 18th and 19th centuries (e.g. 1758,
         118 species; 1854, 144 species <a name="d6347e181"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]). Squamates are presently found on every continent except Antarctica, and in the
         Indian and Pacific Oceans, and span many diverse ecologies and body forms, from limbless
         burrowers to arboreal gliders (summarized in <a name="d6347e185"></a><a name="d6347e187"></a><a name="d6347e189"></a>[<a onclick="LoadInParent('#B2'); return false;" href="#B2">2</a>-<a onclick="LoadInParent('#B4'); return false;" href="#B4">4</a>]).
      </p>
      <p style="line-height:160%">Squamates are key study organisms in numerous fields, from evolution, ecology, and
         behavior <a name="d6347e195"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>] to medicine <a name="d6347e199"></a><a name="d6347e201"></a>[<a onclick="LoadInParent('#B5'); return false;" href="#B5">5</a></a>,<a onclick="LoadInParent('#B6'); return false;" href="#B6">6</a>] and applied physics <a name="d6347e205"></a>[<a onclick="LoadInParent('#B7'); return false;" href="#B7">7</a>]. They have also been the focus of many pioneering studies using phylogenies to address
         questions about trait evolution (e.g. <a name="d6347e209"></a><a name="d6347e211"></a>[<a onclick="LoadInParent('#B8'); return false;" href="#B8">8</a></a>,<a onclick="LoadInParent('#B9'); return false;" href="#B9">9</a>]). Phylogenies are now recognized as being integral to all comparative studies of
         squamate biology (e.g. <a name="d6347e215"></a><a name="d6347e217"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a></a>,<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]). However, hypotheses about squamate phylogeny have changed radically in recent years
         <a name="d6347e222"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>], especially when comparing trees generated from morphological <a name="d6347e226"></a><a name="d6347e228"></a><a name="d6347e230"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>] and molecular data <a name="d6347e234"></a><a name="d6347e236"></a><a name="d6347e238"></a><a name="d6347e240"></a><a name="d6347e242"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. Furthermore, despite extensive work on squamate phylogeny at all taxonomic levels,
         a large-scale phylogeny (i.e. including thousands of species and multiple genes) has
         never been attempted using morphological or molecular data.
      </p>
      <p style="line-height:160%">Squamate phylogenetics has changed radically in the last 10 years, revealing major
         conflicts between the results of morphological and molecular analyses <a name="d6347e248"></a>[<a onclick="LoadInParent('#B12'); return false;" href="#B12">12</a>]. Early estimates of squamate phylogeny <a name="d6347e252"></a>[<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>] and recent studies based on morphological data <a name="d6347e256"></a><a name="d6347e258"></a><a name="d6347e260"></a><a name="d6347e262"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>] consistently supported a basal division between Iguania (including chameleons, agamids,
         and iguanids, <em>sensu lato</em>), and Scleroglossa, which comprises all other squamates (including skinks, geckos,
         snakes, and amphisbaenians). Within Scleroglossa, many phylogenetic analyses of morphological
         data have also supported a clade containing limb-reduced taxa, including various combinations
         of snakes, dibamids, amphisbaenians, and (in some analyses) limb-reduced skinks and
         anguids <a name="d6347e269"></a><a name="d6347e271"></a><a name="d6347e273"></a><a name="d6347e275"></a><a name="d6347e277"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>], though some of these authors also acknowledged that this clade was likely erroneous.
      </p>
      <p style="line-height:160%">In contrast, recent molecular analyses have estimated very different relationships.
         Novel arrangements include placement of dibamids and gekkotans near the root of the
         squamate tree, a sister-group relationship between amphisbaenians and lacertids, and
         a clade (Toxicofera) uniting Iguania with snakes and anguimorphs within Scleroglossa
         <a name="d6347e283"></a><a name="d6347e285"></a><a name="d6347e287"></a><a name="d6347e289"></a><a name="d6347e291"></a><a name="d6347e293"></a><a name="d6347e295"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a></a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. These molecular results (and the results of combined morphological and molecular
         analyses) suggest that some estimates of squamate phylogeny based on morphology may
         have been misled, especially by convergence associated with adaptations to burrowing
         <a name="d6347e299"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>]. However, there have also been disagreements among molecular studies, such as placement
         of dibamids relative to gekkotans and other squamates, and relationships among snakes,
         iguanians, and anguimorphs (e.g. <a name="d6347e303"></a><a name="d6347e305"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]).
      </p>
      <p style="line-height:160%">Analyses of higher-level squamate relationships based on molecular data have so far
         included relatively few (less than 200) species, and none have included representatives
         from all described families and subfamilies <a name="d6347e311"></a><a name="d6347e313"></a><a name="d6347e315"></a><a name="d6347e317"></a><a name="d6347e319"></a><a name="d6347e321"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a></a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. This limited taxon sampling makes existing molecular phylogenies difficult to use
         for broad-scale comparative studies, with some exceptions based on supertrees <a name="d6347e325"></a><a name="d6347e327"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a></a>,<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>]. In addition, limited taxon sampling is potentially a serious issue for phylogenetic
         accuracy <a name="d6347e331"></a><a name="d6347e333"></a><a name="d6347e335"></a><a name="d6347e337"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a>-<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>]. Thus, an analysis with extensive taxon sampling is critically important to test
         hypotheses based on molecular datasets with more limited sampling, and to provide
         a framework for comparative analyses.
      </p>
      <p style="line-height:160%">Despite the lack of a large-scale phylogeny across squamates, recent molecular studies
         have produced phylogenetic estimates for many of the major groups of squamates, including
         iguanian lizards <a name="d6347e343"></a><a name="d6347e345"></a><a name="d6347e347"></a><a name="d6347e349"></a><a name="d6347e351"></a><a name="d6347e353"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>-<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>], higher-level snake groups <a name="d6347e357"></a><a name="d6347e359"></a><a name="d6347e361"></a>[<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>-<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>], typhlopoid snakes <a name="d6347e365"></a><a name="d6347e367"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a></a>,<a onclick="LoadInParent('#B39'); return false;" href="#B39">39</a>], colubroid snakes <a name="d6347e371"></a><a name="d6347e373"></a><a name="d6347e375"></a><a name="d6347e377"></a><a name="d6347e379"></a><a name="d6347e381"></a><a name="d6347e383"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>-<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>], booid snakes <a name="d6347e387"></a><a name="d6347e389"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a></a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>], scincid lizards <a name="d6347e394"></a><a name="d6347e396"></a><a name="d6347e398"></a><a name="d6347e400"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>-<a onclick="LoadInParent('#B52'); return false;" href="#B52">52</a>], gekkotan lizards <a name="d6347e404"></a><a name="d6347e406"></a><a name="d6347e408"></a><a name="d6347e410"></a><a name="d6347e412"></a><a name="d6347e414"></a><a name="d6347e416"></a><a name="d6347e418"></a>[<a onclick="LoadInParent('#B53'); return false;" href="#B53">53</a>-<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>], teiioid lizards <a name="d6347e422"></a><a name="d6347e424"></a><a name="d6347e426"></a><a name="d6347e428"></a>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>-<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>], lacertid lizards <a name="d6347e432"></a><a name="d6347e434"></a><a name="d6347e436"></a><a name="d6347e438"></a><a name="d6347e440"></a>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>-<a onclick="LoadInParent('#B69'); return false;" href="#B69">69</a>], and amphisbaenians <a name="d6347e444"></a><a name="d6347e446"></a>[<a onclick="LoadInParent('#B70'); return false;" href="#B70">70</a></a>,<a onclick="LoadInParent('#B71'); return false;" href="#B71">71</a>]. These studies have done an outstanding job of clarifying the phylogeny and taxonomy
         of these groups, but many were limited in some ways by the number of characters and
         taxa that they sampled (and which were available at the time for sequencing).
      </p>
      <p style="line-height:160%">Here, we present a phylogenetic estimate for Squamata based on combining much of the
         existing sequence data for squamate reptiles, using the increasingly well-established
         supermatrix approach <a name="d6347e452"></a><a name="d6347e454"></a><a name="d6347e456"></a><a name="d6347e458"></a><a name="d6347e460"></a><a name="d6347e462"></a><a name="d6347e464"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B72'); return false;" href="#B72">72</a>-<a onclick="LoadInParent('#B77'); return false;" href="#B77">77</a>]. We present a new phylogenetic estimate including 4161 squamate species. The dataset
         includes up to 12896 bp per species from 12 genes (7 nuclear, 5 mitochondrial). We
         include species from all currently described families and subfamilies. In terms of
         species sampled, this is 5 times larger than any previous phylogeny for any one squamate
         group <a name="d6347e468"></a><a name="d6347e470"></a>[<a onclick="LoadInParent('#B30'); return false;" href="#B30">30</a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>], 3 times larger than the largest supertree estimate <a name="d6347e474"></a>[<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>], and 25 times larger than the largest molecular study of higher-level squamate relationships
         <a name="d6347e478"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. While we did not sequence any new taxa specifically for this project, much of the
         data in the combined matrix were generated in our labs or from our previous collaborative
         projects <a name="d6347e482"></a><a name="d6347e484"></a><a name="d6347e486"></a><a name="d6347e488"></a><a name="d6347e490"></a><a name="d6347e492"></a><a name="d6347e494"></a><a name="d6347e496"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a></a>,<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>],<a name="d6347e501"></a><a name="d6347e503"></a><a name="d6347e505"></a><a name="d6347e507"></a><a name="d6347e509"></a>[<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>-<a onclick="LoadInParent('#B82'); return false;" href="#B82">82</a>], including thousands of gene sequences from hundreds of species (&gt;550 species; ~13%
         of the total).
      </p>
      <p style="line-height:160%">The supermatrix approach can provide a relatively comprehensive phylogeny, and uncover
         novel relationships not seen in any of the separate analyses in which the data were
         generated. Such novel relationships can be revealed via three primary mechanisms.
         First, different studies may have each sampled different species from a given group
         for the same genes, and combining these data may reveal novel relationships not apparent
         in the separate analyses. Second, different studies may have used different genetic
         markers for the same taxa, and combining these markers can dramatically increase character
         sampling, potentially revealing new relationships and providing stronger support for
         previous hypotheses. Third, even for clades that were previously studied using complete
         taxon sampling and multiple loci, novel relationships may be revealed by including
         these lineages with other related groups in a large-scale phylogeny.
      </p>
      <p style="line-height:160%">The estimated tree and branch-lengths should be useful for comparative studies of
         squamate biology. However, this phylogeny is based on a supermatrix with extensive
         missing data (mean = 81% per species). Some authors have suggested that matrices with
         missing cells may yield misleading estimates of topology, support, and branch lengths
         <a name="d6347e517"></a>[<a onclick="LoadInParent('#B83'); return false;" href="#B83">83</a>]. Nevertheless, most empirical and simulation studies have not found this to be the
         case, at least for topology and support <a name="d6347e521"></a><a name="d6347e523"></a><a name="d6347e525"></a><a name="d6347e527"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>,<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a></a>,<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]. Though fewer studies have examined the effects of missing data on branch lengths
         <a name="d6347e531"></a><a name="d6347e533"></a><a name="d6347e535"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a></a>,<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>], these also suggest that missing data do not strongly impact estimates. Here, we
         test whether branch lengths for terminal taxa are related to their completeness.
      </p>
      <p style="line-height:160%">In general, our results corroborate those of many recent molecular studies with regard
         to higher-level relationships, species-level relationships, and the monophyly, composition,
         and relationships of most families, subfamilies, and genera. However, our results
         differ from previous estimates for some groups, and reveal (or corroborate) numerous
         problems in the existing classification of squamates. We therefore provide a conservative,
         updated classification of extant squamates at the family and subfamily level based
         on the new phylogeny, while highlighting problematic taxonomy at the genus level,
         without making changes. The generic composition of all families and subfamilies under
         our revised taxonomy are provided in Appendix I.
      </p>
      <p style="line-height:160%">We note dozens of problems in the genus-level taxonomy suggested by our tree, but
         we acknowledge in advance that we do not provide a comprehensive review of the previous
         literature dealing with all these taxonomic issues (this would require a monographic
         treatment). Similarly, we do not attempt to fix these genus-level problems here, as
         most will require more extensive taxon (and potentially character) sampling to adequately
         resolve.
      </p>
      <p style="line-height:160%">Throughout the paper, we address only extant squamates. Squamata also includes numerous
         extinct species classified in both extant and extinct families, subfamilies, and genera.
         Relationships and classification of extinct squamates based on morphological data
         from fossils have been addressed by numerous authors (e.g. <a name="d6347e546"></a><a name="d6347e548"></a><a name="d6347e550"></a><a name="d6347e552"></a><a name="d6347e554"></a><a name="d6347e556"></a><a name="d6347e558"></a><a name="d6347e560"></a><a name="d6347e562"></a><a name="d6347e564"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a></a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B88'); return false;" href="#B88">88</a>-<a onclick="LoadInParent('#B93'); return false;" href="#B93">93</a>]). A classification based only on living taxa may create some problems for classifying
         fossil taxa, but these can be addressed in future studies that integrate molecular
         and fossil data <a name="d6347e568"></a><a name="d6347e570"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a>].
      </p>
   </div>
</section>
<section><a name="sec2"></a><h3>Results</h3>
   <div class="collapsible-content">
      <h4>Supermatrix phylogeny</h4>
      <p style="line-height:160%">We generated the final tree (lnL = âˆ’2609551.07) using Maximum Likelihood (ML) in RAxMLv7.2.8.
         Support was assessed using the non-parametric Shimodaira-Hasegawa-Like (SHL) implementation
         of the approximate likelihood-ratio test (aLRT; see <a name="d6347e584"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>]). The tree and data matrix are available in NEXUS format in DataDryad repository
         10.5061/dryad.82h0m and as Additional file <a name="d6347e587"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S1">1</a>: Data File S1. A skeletal representation of the tree (excluding several species which
         are <em>incertae sedis</em>) is shown in Figure&nbsp;<a name="d6347e593"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>. The full species-level phylogeny (minus the outgroup <em>Sphenodon</em>) is shown in Figures&nbsp;<a name="d6347e600"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>, <a name="d6347e603"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3">3</a>, <a name="d6347e606"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F4">4</a>, <a name="d6347e609"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>, <a name="d6347e612"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e615"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e619"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e622"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e625"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>, <a name="d6347e628"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>, <a name="d6347e631"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12">12</a>, <a name="d6347e634"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>, <a name="d6347e638"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>, <a name="d6347e641"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15">15</a>, <a name="d6347e644"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>, <a name="d6347e647"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e650"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e653"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>, <a name="d6347e657"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>, <a name="d6347e660"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>, <a name="d6347e663"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>, <a name="d6347e666"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>, <a name="d6347e669"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24">24</a>, <a name="d6347e672"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e676"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>, <a name="d6347e679"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>, <a name="d6347e682"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>. The analysis yields a generally well-supported phylogenetic estimate for squamates
         (i.e. 70% of nodes have SHL values &gt;85, indicating they are strongly supported). There
         is no relationship between proportional completeness (bp of non-missing data in species
         / 12896 bp of complete data) and branch length (<em>r</em> = âˆ’0.29, <em>P</em> = 0.14) for terminal taxa, strongly suggesting that the estimated branch lengths
         are not consistently biased by missing data.
      </p><div class="figs">
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S1','S1',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S1"></a><strong>Additional file 1: Data File S1.</strong> The 4162-species ML phylogeny in Newick format; taxonomic changes are given in Additional
         file <a name="d6347e698"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S2">2</a>: Table S1.                                                                      
         </p><p>
         	Format: TXT 
         	Size: 264KB <a href="/content/supplementary/1471-2148-13-93-s1.txt">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div>
      <div class="addfile"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S2','S2',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S2"></a><strong>Additional file 2: Table S1.</strong> GenBank accession numbers for all taxa included in this analysis.</p><p>
         	Format: DOCX 
         	Size: 682KB <a href="/content/supplementary/1471-2148-13-93-s2.docx">Download file</a><span class="open-data-icon"><a href="/about/access/#opendata"><img title="Open Data" alt="Open Data" src="/bmcimages/article/opendata.gif"></a></span></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','F1',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1"><img align="top" src="/content/figures/1471-2148-13-93-1.gif" alt="thumbnail" class="thumbnail"><strong>Figure 1.</strong></a> <strong>Higher-level squamate phylogeny.</strong> Skeletal representation of the 4161-species tree from maximum-likelihood analysis
         of 12 genes, with tips representing families and subfamilies (following our taxonomic
         revision; species considered <em>incertae sedis</em> are not shown). Numbers at nodes are SHL values greater than 50%. The full tree is
         presented in Figures&nbsp;<a name="d6347e729"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>, <a name="d6347e732"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3">3</a>, <a name="d6347e735"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F4">4</a>, <a name="d6347e738"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>, <a name="d6347e742"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e745"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e748"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e751"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e754"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>, <a name="d6347e757"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>, <a name="d6347e761"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12">12</a>, <a name="d6347e764"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>, <a name="d6347e767"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>, <a name="d6347e770"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15">15</a>, <a name="d6347e773"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>, <a name="d6347e776"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e780"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e783"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>, <a name="d6347e786"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>, <a name="d6347e789"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>, <a name="d6347e792"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>, <a name="d6347e795"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>, <a name="d6347e799"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24">24</a>, <a name="d6347e802"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e805"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>, <a name="d6347e808"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>, <a name="d6347e811"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>.
      </div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','F2',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2"><img align="top" src="/content/figures/1471-2148-13-93-2.gif" alt="thumbnail" class="thumbnail"><strong>Figure 2.</strong></a> <strong>Species-level squamate phylogeny.</strong> Large-scale maximum likelihood estimate of squamate phylogeny, containing 4161 species.
         Numbers at nodes are SHL values greater than 50%. A skeletal version of this tree
         is presented in Figure&nbsp;<a name="d6347e827"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>. Bold italic letters indicate figure panels (<strong>A-AA</strong>). Within panels, branch lengths are proportional to expected substitutions per site,
         but the relative scale differs between panels.
      </div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','F3',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3"><img align="top" src="/content/figures/1471-2148-13-93-3.gif" alt="thumbnail" class="thumbnail"><strong>Figure 3.</strong></a> <strong>Species-level squamate phylogeny continued (B).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F4','F4',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F4"><img align="top" src="/content/figures/1471-2148-13-93-4.gif" alt="thumbnail" class="thumbnail"><strong>Figure 4.</strong></a> <strong>Species-level squamate phylogeny continued (C).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','F5',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5"><img align="top" src="/content/figures/1471-2148-13-93-5.gif" alt="thumbnail" class="thumbnail"><strong>Figure 5.</strong></a> <strong>Species-level squamate phylogeny continued (D).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','F6',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6"><img align="top" src="/content/figures/1471-2148-13-93-6.gif" alt="thumbnail" class="thumbnail"><strong>Figure 6.</strong></a> <strong>Species-level squamate phylogeny continued (E).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','F7',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7"><img align="top" src="/content/figures/1471-2148-13-93-7.gif" alt="thumbnail" class="thumbnail"><strong>Figure 7.</strong></a> <strong>Species-level squamate phylogeny continued (F).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','F8',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8"><img align="top" src="/content/figures/1471-2148-13-93-8.gif" alt="thumbnail" class="thumbnail"><strong>Figure 8.</strong></a> <strong>Species-level squamate phylogeny continued (G).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','F9',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9"><img align="top" src="/content/figures/1471-2148-13-93-9.gif" alt="thumbnail" class="thumbnail"><strong>Figure 9.</strong></a> <strong>Species-level squamate phylogeny continued (H).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','F10',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10"><img align="top" src="/content/figures/1471-2148-13-93-10.gif" alt="thumbnail" class="thumbnail"><strong>Figure 10.</strong></a> <strong>Species-level squamate phylogeny continued (I).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','F11',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11"><img align="top" src="/content/figures/1471-2148-13-93-11.gif" alt="thumbnail" class="thumbnail"><strong>Figure 11.</strong></a> <strong>Species-level squamate phylogeny continued (J).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','F12',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12"><img align="top" src="/content/figures/1471-2148-13-93-12.gif" alt="thumbnail" class="thumbnail"><strong>Figure 12.</strong></a> <strong>Species-level squamate phylogeny continued (K).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','F13',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13"><img align="top" src="/content/figures/1471-2148-13-93-13.gif" alt="thumbnail" class="thumbnail"><strong>Figure 13.</strong></a> <strong>Species-level squamate phylogeny continued (L).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','F14',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14"><img align="top" src="/content/figures/1471-2148-13-93-14.gif" alt="thumbnail" class="thumbnail"><strong>Figure 14.</strong></a> <strong>Species-level squamate phylogeny continued (M).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','F15',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15"><img align="top" src="/content/figures/1471-2148-13-93-15.gif" alt="thumbnail" class="thumbnail"><strong>Figure 15.</strong></a> <strong>Species-level squamate phylogeny continued (N).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','F16',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16"><img align="top" src="/content/figures/1471-2148-13-93-16.gif" alt="thumbnail" class="thumbnail"><strong>Figure 16.</strong></a> <strong>Species-level squamate phylogeny continued (O).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','F17',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17"><img align="top" src="/content/figures/1471-2148-13-93-17.gif" alt="thumbnail" class="thumbnail"><strong>Figure 17.</strong></a> <strong>Species-level squamate phylogeny continued (P).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','F18',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18"><img align="top" src="/content/figures/1471-2148-13-93-18.gif" alt="thumbnail" class="thumbnail"><strong>Figure 18.</strong></a> <strong>Species-level squamate phylogeny continued (Q).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','F19',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19"><img align="top" src="/content/figures/1471-2148-13-93-19.gif" alt="thumbnail" class="thumbnail"><strong>Figure 19.</strong></a> <strong>Species-level squamate phylogeny continued (R).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','F20',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20"><img align="top" src="/content/figures/1471-2148-13-93-20.gif" alt="thumbnail" class="thumbnail"><strong>Figure 20.</strong></a> <strong>Species-level squamate phylogeny continued (S).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','F21',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21"><img align="top" src="/content/figures/1471-2148-13-93-21.gif" alt="thumbnail" class="thumbnail"><strong>Figure 21.</strong></a> <strong>Species-level squamate phylogeny continued (T).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','F22',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22"><img align="top" src="/content/figures/1471-2148-13-93-22.gif" alt="thumbnail" class="thumbnail"><strong>Figure 22.</strong></a> <strong>Species-level squamate phylogeny continued (U).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','F23',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23"><img align="top" src="/content/figures/1471-2148-13-93-23.gif" alt="thumbnail" class="thumbnail"><strong>Figure 23.</strong></a> <strong>Species-level squamate phylogeny continued (V).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','F24',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24"><img align="top" src="/content/figures/1471-2148-13-93-24.gif" alt="thumbnail" class="thumbnail"><strong>Figure 24.</strong></a> <strong>Species-level squamate phylogeny continued (W).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','F25',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25"><img align="top" src="/content/figures/1471-2148-13-93-25.gif" alt="thumbnail" class="thumbnail"><strong>Figure 25.</strong></a> <strong>Species-level squamate phylogeny continued (X).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','F26',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26"><img align="top" src="/content/figures/1471-2148-13-93-26.gif" alt="thumbnail" class="thumbnail"><strong>Figure 26.</strong></a> <strong>Species-level squamate phylogeny continued (Y).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','F27',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27"><img align="top" src="/content/figures/1471-2148-13-93-27.gif" alt="thumbnail" class="thumbnail"><strong>Figure 27.</strong></a> <strong>Species-level squamate phylogeny continued (Z).</strong></div>
      <div class="fig"><p><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','F28',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28"><img align="top" src="/content/figures/1471-2148-13-93-28.gif" alt="thumbnail" class="thumbnail"><strong>Figure 28.</strong></a> <strong>Species-level squamate phylogeny continued (AA).</strong></div></div>
      <h4>Higher-level relationships</h4>
      <p style="line-height:160%">Our tree (Figure&nbsp;<a name="d6347e1160"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>) is broadly congruent with most previous molecular studies of higher-level squamate
         phylogeny using both nuclear data and combined nuclear and mitochondrial data (e.g.
         <a name="d6347e1164"></a><a name="d6347e1166"></a><a name="d6347e1168"></a><a name="d6347e1170"></a><a name="d6347e1172"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]), providing important confirmation of previous molecular studies based on more limited
         taxon sampling. Specifically we support (Figure&nbsp;<a name="d6347e1175"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>): (i) the placement of dibamids and gekkotans near the base of the tree (Figure&nbsp;<a name="d6347e1178"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>A); (ii) a sister-group relationship between Scincoidea (scincids, cordylids, gerrhosaurids,
         and xantusiids; Figure&nbsp;<a name="d6347e1181"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>B) and a clade (Episquamata; Figure&nbsp;<a name="d6347e1185"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>C) containing the rest of the squamates excluding dibamids and gekkotans; (iii) Lacertoidea
         (lacertids, amphisbaenians, teiids, and gymnophthalmids; Figure&nbsp;<a name="d6347e1188"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>D), and (iv) a clade (Toxicofera; Figure&nbsp;<a name="d6347e1191"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>E) containing anguimorphs (Figure&nbsp;<a name="d6347e1194"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>F), iguanians (Figure&nbsp;<a name="d6347e1197"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>G), and snakes (Figure&nbsp;<a name="d6347e1200"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>H) as the sister taxon to Lacertoidea.
      </p>
      <p style="line-height:160%">These relationships are strongly supported in general (Figure&nbsp;<a name="d6347e1205"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), but differ sharply from most trees based on morphological data <a name="d6347e1209"></a><a name="d6347e1211"></a><a name="d6347e1213"></a><a name="d6347e1215"></a><a name="d6347e1217"></a><a name="d6347e1219"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B95'); return false;" href="#B95">95</a>]. Nevertheless, many clades found in previous morphological taxonomies and phylogenies
         are also present in this tree in some form, including Amphisbaenia, Anguimorpha, Gekkota,
         Iguania, Lacertoidea (but including amphisbaenians), Scincoidea, Serpentes, and many
         families and subfamilies. In contrast, the relationships among these groups differ
         strongly between molecular analyses <a name="d6347e1223"></a><a name="d6347e1225"></a><a name="d6347e1227"></a><a name="d6347e1229"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>] and morphological analyses <a name="d6347e1233"></a><a name="d6347e1235"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a></a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]. Our results demonstrate that this incongruence is not explained by limited taxon
         sampling in the molecular data sets. In fact, our species-level sampling is far more
         extensive than in any morphological analyses (e.g. <a name="d6347e1239"></a><a name="d6347e1241"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a></a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]), by an order of magnitude.
      </p>
      <p style="line-height:160%">We find that the basal squamate relationships are strongly supported in our tree.
         The family Dibamidae is the sister group to all other squamates, and Gekkota is the
         sister group to all squamates excluding Dibamidae (Figure&nbsp;<a name="d6347e1246"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), as in some previous studies (e.g. <a name="d6347e1250"></a><a name="d6347e1252"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B18'); return false;" href="#B18">18</a>]). Other recent molecular analyses have also placed Dibamidae near the squamate root,
         but differed in placing it as either the sister taxon to all squamates excluding Gekkota
         <a name="d6347e1256"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>], or the sister- group of Gekkota <a name="d6347e1260"></a><a name="d6347e1262"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. Our results also corroborate that the New World genus <em>Anelytropsis</em> is nested within the Old World genus <em>Dibamus</em><a name="d6347e1272"></a>[<a onclick="LoadInParent('#B96'); return false;" href="#B96">96</a>], but the associated branches are weakly supported (Figure&nbsp;<a name="d6347e1275"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>).
      </p>
      <h4>Gekkota</h4>
      <p style="line-height:160%">Within Gekkota, we corroborate both earlier morphological <a name="d6347e1285"></a>[<a onclick="LoadInParent('#B97'); return false;" href="#B97">97</a>] and recent molecular estimates <a name="d6347e1289"></a><a name="d6347e1291"></a><a name="d6347e1293"></a><a name="d6347e1295"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a></a>,<a onclick="LoadInParent('#B56'); return false;" href="#B56">56</a>,<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>,<a onclick="LoadInParent('#B98'); return false;" href="#B98">98</a>] in supporting a clade containing the Australian radiation of "diplodactylid" geckos
         (Carphodactylidae and Diplodactylidae) and the snakelike pygopodids (Figures&nbsp;<a name="d6347e1298"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e1301"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>). As in previous studies <a name="d6347e1305"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>], Carphodactylidae is the weakly supported sister group to Pygopodidae, and this clade
         is the sister group of Diplodactylidae (Figures&nbsp;<a name="d6347e1309"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e1312"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>). We recover clades within the former Gekkonidae that correspond to the strongly
         supported families Eublepharidae, Sphaerodactylidae, Phyllodactylidae, and Gekkonidae
         as in previous studies, and similar relationships among these groups <a name="d6347e1316"></a><a name="d6347e1318"></a><a name="d6347e1320"></a><a name="d6347e1322"></a><a name="d6347e1324"></a><a name="d6347e1326"></a><a name="d6347e1328"></a><a name="d6347e1330"></a>[<a onclick="LoadInParent('#B55'); return false;" href="#B55">55</a>-<a onclick="LoadInParent('#B57'); return false;" href="#B57">57</a>,<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a></a>,<a onclick="LoadInParent('#B60'); return false;" href="#B60">60</a>,<a onclick="LoadInParent('#B98'); return false;" href="#B98">98</a>-<a onclick="LoadInParent('#B100'); return false;" href="#B100">100</a>].
      </p>
      <p style="line-height:160%">Within Gekkota, we find evidence for non-monophyly of many genera. Many relationships
         among the New Caledonian diplodactylids are weakly supported (Figure&nbsp;<a name="d6347e1335"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>), and there is apparent non-monophyly of the genera <em>Rhacodactylus, Bavayia,</em> and <em>Eurydactylodes</em> with respect to each other and <em>Oedodera, Dierogekko, Paniegekko, Correlophus</em>, and <em>Mniarogekko</em><a name="d6347e1350"></a>[<a onclick="LoadInParent('#B101'); return false;" href="#B101">101</a>]. In the Australian diplodactylids, <em>Strophurus taenicauda</em> is strongly supported as belonging to a clade that is only distantly related to the
         other sampled <em>Strophurus</em> species (Figure&nbsp;<a name="d6347e1360"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>). The two species of the North African sphaerodactylid genus <em>Saurodactylus</em> are divided between the two major sphaerodactylid clades (Figure&nbsp;<a name="d6347e1366"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3">3</a>), but the associated branches are weakly supported. The South American phyllodactylid
         genus <em>Homonota</em> is strongly supported as being paraphyletic with respect to <em>Phyllodactylus</em> (Figure&nbsp;<a name="d6347e1376"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3">3</a>).
      </p>
      <p style="line-height:160%">A number of gekkonid genera (Figure&nbsp;<a name="d6347e1381"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F4">4</a>) also appear to be non-monophyletic, including the Asian genera <em>Cnemaspis</em> (sampled species divided into two non-sister clades), <em>Lepidodactylus</em> (with respect to <em>Pseudogekko</em> and some <em>Luperosaurus</em>), <em>Gekko</em> (with respect to <em>Ptychozoon</em> and <em>Lu. iskandari</em>), <em>Luperosaurus</em> (with respect to <em>Lepidodactylus</em> and <em>Gekko</em>), <em>Mediodactylus</em> (with respect to <em>Pseudoceramodactylus</em>, <em>Tropiocolotes</em>, <em>Stenodactylus, Cyrtopodion, Bunopus, Crossobamon,</em> and <em>Agamura</em>), and <em>Bunopus</em> (with respect to <em>Crossobamon</em>), and the African <em>Afrogecko</em> (with respect to <em>Afroedura, Christinus, Cryptactites,</em> and <em>Matoatoa</em>)<em>, Afroedura</em> (with respect to <em>Afrogecko, Blaesodactylus, Christinus, Geckolepis</em>, <em>Pachydactylus, Rhoptropus,</em> and numerous other genera), <em>Chondrodactylus</em> (with respect to <em>Pachydactylus laevigatus</em>), and <em>Pachydactylus</em> (with respect to <em>Chondrodactylus</em> and <em>Colopus</em>). Many of these taxonomic problems in gekkotan families have been identified in previous
         studies (e.g. <a name="d6347e1474"></a><a name="d6347e1476"></a><a name="d6347e1478"></a>[<a onclick="LoadInParent('#B59'); return false;" href="#B59">59</a>,<a onclick="LoadInParent('#B99'); return false;" href="#B99">99</a>,<a onclick="LoadInParent('#B102'); return false;" href="#B102">102</a>]), and extensive changes will likely be required to fix them.
      </p>
      <h4>Scincoidea</h4>
      <p style="line-height:160%">We strongly support (SHL = 100; Figures&nbsp;<a name="d6347e1487"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e1490"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>, <a name="d6347e1493"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e1496"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1499"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1503"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1506"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>) the monophyly of Scincoidea (Scincidae, Xantusiidae, Gerrhosauridae, and Cordylidae),
         as in other recent studies <a name="d6347e1510"></a><a name="d6347e1512"></a><a name="d6347e1514"></a><a name="d6347e1516"></a><a name="d6347e1518"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. All four families are strongly supported (Figures&nbsp;<a name="d6347e1521"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>, <a name="d6347e1524"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e1527"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1531"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1534"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1537"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). A similar clade is also recognized in morphological phylogenies <a name="d6347e1541"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a>], though without Xantusiidae in some <a name="d6347e1545"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>].
      </p>
      <p style="line-height:160%">Within the New World family Xantusiidae, we corroborate previous analyses <a name="d6347e1551"></a><a name="d6347e1553"></a>[<a onclick="LoadInParent('#B103'); return false;" href="#B103">103</a></a>,<a onclick="LoadInParent('#B104'); return false;" href="#B104">104</a>] that found strong support for a sister-group relationship between <em>Xantusia</em> and <em>Lepidophyma</em>, excluding <em>Cricosaura</em> (Figure&nbsp;<a name="d6347e1565"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>). These relationships support the subfamily Cricosaurinae for <em>Cricosaura</em><a name="d6347e1572"></a>[<a onclick="LoadInParent('#B105'); return false;" href="#B105">105</a>]. We also recognize Xantusiinae for the North American genus <em>Xantusia</em> and Lepidophyminae for the Central American genus <em>Lepidophyma</em><a name="d6347e1581"></a><a name="d6347e1583"></a>[<a onclick="LoadInParent('#B106'); return false;" href="#B106">106</a></a>,<a onclick="LoadInParent('#B107'); return false;" href="#B107">107</a>].
      </p>
      <p style="line-height:160%">Within the African and Madagascan family Gerrhosauridae (Figure&nbsp;<a name="d6347e1588"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>), the genus <em>Gerrhosaurus</em> is weakly supported as being paraphyletic with respect to the clade comprising <em>Tetradactylus + Cordylosaurus,</em> with <em>G. major</em> placed as the sister group to all other gerrhosaurids. Within Cordylidae (Figure&nbsp;<a name="d6347e1600"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>), we use the generic taxonomy from a recent phylogenetic analysis and re-classification
         based on multiple nuclear and mitochondrial genes <a name="d6347e1605"></a>[<a onclick="LoadInParent('#B108'); return false;" href="#B108">108</a>]. This classification broke up the non-monophyletic <em>Cordylus</em><a name="d6347e1611"></a>[<a onclick="LoadInParent('#B109'); return false;" href="#B109">109</a>] into several smaller genera, and we corroborate the non-monophyly of the former <em>Cordylus</em> and support the monophyly of the newly recognized genera (Figure&nbsp;<a name="d6347e1617"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>). We support the distinctiveness of <em>Platysaurus</em> (Figure&nbsp;<a name="d6347e1624"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>) and recognition of the subfamily Platysaurinae <a name="d6347e1628"></a>[<a onclick="LoadInParent('#B108'); return false;" href="#B108">108</a>].
      </p>
      <p style="line-height:160%">We strong support (SHL = 100) for the monophyly of Scincidae (Figure&nbsp;<a name="d6347e1633"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>) as in previous studies (e.g. <a name="d6347e1637"></a><a name="d6347e1639"></a><a name="d6347e1641"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a></a>,<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]). We strongly support the basal placement of the monophyletic subfamily Acontiinae
         (Figure&nbsp;<a name="d6347e1644"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>), as found in some previous studies (e.g. <a name="d6347e1648"></a><a name="d6347e1650"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]) but not others (e.g. <a name="d6347e1654"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>]). Similar to earlier studies, we find that the subfamily Scincinae (sensu <a name="d6347e1659"></a>[<a onclick="LoadInParent('#B110'); return false;" href="#B110">110</a>]) is non-monophyletic, as Feylininae is nested within Scincinae (also found in <a name="d6347e1663"></a><a name="d6347e1665"></a><a name="d6347e1667"></a><a name="d6347e1669"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a></a>,<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>,<a onclick="LoadInParent('#B111'); return false;" href="#B111">111</a>]). Based on these results, synonymizing Feylininae with Scincinae produces a monophyletic
         Scincinae (SHL = 97), which is then sister to a monophyletic Lygosominae (SHL = 100
         excluding <em>Ateuchosaurus</em>; see below) with 94% SHL support (Figures&nbsp;<a name="d6347e1675"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e1678"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1681"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1685"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1688"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). This yields a new classification in which all three subfamilies (Acontiinae, Lygosominae,
         Scincinae) are strongly supported. Importantly, these definitions approxi\mate the
         traditional content of the three subfamilies <a name="d6347e1692"></a><a name="d6347e1694"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>,<a onclick="LoadInParent('#B110'); return false;" href="#B110">110</a>], except for recognition of Feylininae.
      </p>
      <p style="line-height:160%">We note that a recent revision of the New World genus <em>Mabuya</em> introduced a nontraditional family-level classification for Scincidae <a name="d6347e1703"></a>[<a onclick="LoadInParent('#B112'); return false;" href="#B112">112</a>]. These authors divided Scincidae into seven families: Acontiidae, Egerniidae, Eugongylidae,
         Lygosomidae, Mabuyidae, Scincidae and Sphenomorphidae. However, there was no phylogenetic
         need for considering these clades as families, since the family Scincidae is clearly
         monophyletic, based on our results and others (see above). Thus, their new taxonomy
         changes the long-standing definition of Scincidae unnecessarily (see <a name="d6347e1707"></a>[<a onclick="LoadInParent('#B113'); return false;" href="#B113">113</a>]). Furthermore, these changes were done without defining the full content (beyond
         a type genus) of any of these families other than Scincidae (the former Scincinae
         + Feylininae) and Acontiidae (the former Acontiinae).
      </p>
      <p style="line-height:160%">Most importantly, the new taxonomy proposed by these authors <a name="d6347e1713"></a>[<a onclick="LoadInParent('#B112'); return false;" href="#B112">112</a>] is at odds with the phylogeny estimated here, with respect to the familial and subfamilial
         classification of &gt;1000 skink species (Figures&nbsp;<a name="d6347e1716"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e1719"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1722"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1725"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1729"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). For instance, <em>Sphenomorphus stellatus</em> is found in a strongly supported clade containing <em>Lygosoma</em> (presumably Lygosomidae; Figure&nbsp;<a name="d6347e1738"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>), which is separate from the other clade (presumably Sphenomorphidae) containing
         the other sampled <em>Sphenomorphus</em> species (Figure&nbsp;<a name="d6347e1744"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>; note that these <em>Sphenomorphus</em> species are divided among several subclades within this latter clade). An additional
         problem is that <em>Egernia</em>, <em>Lygosoma</em>, and <em>Sphenomorphus</em> are the type genera of Egerniidae, Lygosomidae, and Sphenomorphidae, but are paraphyletic
         as currently defined (Figures&nbsp;<a name="d6347e1760"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1763"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1767"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1770"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>), leading to further uncertainty in the content and definition of these putative
         families.
      </p>
      <p style="line-height:160%">Furthermore, Mabuyidae apparently refers to the clade (Figure&nbsp;<a name="d6347e1775"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>) containing <em>Chioninia</em>, <em>Dasia</em>, <em>Mabuya</em>, <em>Trachylepis</em>, with each of these genera placed in its own subfamily (Chioniniinae, Dasiinae, Mabuyinae,
         and Trachylepidinae). However, several other genera are strongly placed in this group,
         such as <em>Eumecia, Eutropis</em>, <em>Lankaskincus</em>, and <em>Ristella</em> (Figure&nbsp;<a name="d6347e1800"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>). These other genera cannot be readily fit into these subfamilial groups (i.e. they
         are not the sister group of any genera in those subfamilies), and <em>Trachylepis</em> is paraphyletic with respect to <em>Eumecia, Chioninia,</em> and <em>Mabuya</em> (Figure&nbsp;<a name="d6347e1813"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>). Also, we find that <em>Emoia</em> is divided between clades containing <em>Lygosoma</em> (Lygosomidae) and <em>Eugongylus</em> (Eugongylidae), and many of these relationships have strong support (Figure&nbsp;<a name="d6347e1825"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). Finally, <em>Ateuchosaurus</em> is apparently not accounted for in their classification, and here is weakly placed
         as the sister-group to a clade comprising their Sphenomorphidae, Egerniidae, Mabuyidae,
         and Lygosomidae (i.e. Lygosominae as recognized here; Figures&nbsp;<a name="d6347e1832"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1835"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1838"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1841"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>).
      </p>
      <p style="line-height:160%">These authors <a name="d6347e1847"></a>[<a onclick="LoadInParent('#B112'); return false;" href="#B112">112</a>] argued that a more heavily subdivided classification for skinks may be desirable
         for facilitating future taxonomic revisions and species descriptions. However, this
         classification seems likely to only exacerbate existing taxonomic problems (e.g. placing
         congeneric species in different families without revising the genus-level taxonomy).
         Here, we retain the previous definition of <em>Mabuya</em> (restricted to the New World clade; sensu <a name="d6347e1854"></a>[<a onclick="LoadInParent('#B114'); return false;" href="#B114">114</a>]), and we support the traditional definitions of Scincidae, Acontiinae, Scincinae
         (but including Feylininae), and Lygosominae (Figures&nbsp;<a name="d6347e1857"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e1860"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1864"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1867"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1870"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>; note that we leave <em>Ateuchosaurus</em> as <em>incertae sedis</em>). The other taxonomic issues in Scincidae identified here and elsewhere should be
         resolved in future studies. Our phylogeny provides a framework in which these analyses
         can take place (i.e. identifying major subclades within skinks), which we think may
         be more useful than a classification lacking clear taxon definitions.
      </p>
      <p style="line-height:160%">Of the 133 scincid genera <a name="d6347e1882"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], we can assign the 113 sampled in our tree to one of the three subfamilies in our
         classification (Acontiinae, Lygosominae, and Scincinae; Appendix I). We place 19 of
         the remaining genera into one of the three subfamilies based on previous classifications
         (e.g. <a name="d6347e1886"></a>[<a onclick="LoadInParent('#B110'); return false;" href="#B110">110</a>]), with <em>Ateuchosaurus</em> as <em>incertae sedis</em> in Scincidae. Below, we review the non-monophyletic genera in our tree. Many of these
         problems have been reported by previous authors <a name="d6347e1896"></a><a name="d6347e1898"></a><a name="d6347e1900"></a><a name="d6347e1902"></a><a name="d6347e1904"></a><a name="d6347e1906"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>,<a onclick="LoadInParent('#B111'); return false;" href="#B111">111</a>,<a onclick="LoadInParent('#B115'); return false;" href="#B115">115</a>-<a onclick="LoadInParent('#B118'); return false;" href="#B118">118</a>], and for brevity we do not distinguish between cases reported in previous studies,
         and potentially new instances found here.
      </p>
      <p style="line-height:160%">Within Acontiinae, we find that the two genera are both monophyletic (Figure&nbsp;<a name="d6347e1911"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>). Within Scincinae, many genera are now strongly monophyletic (thanks in part to
         the dismantling of <em>Eumeces</em>; <a name="d6347e1918"></a><a name="d6347e1920"></a><a name="d6347e1922"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a></a>,<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>,<a onclick="LoadInParent('#B119'); return false;" href="#B119">119</a>]), but some problems remain (Figure&nbsp;<a name="d6347e1925"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>). The genera <em>Scincus</em> and <em>Scincopus</em> are strongly supported as being nested inside of the remaining <em>Eumeces</em>. Among Malagasy scincines (see <a name="d6347e1939"></a><a name="d6347e1941"></a>[<a onclick="LoadInParent('#B49'); return false;" href="#B49">49</a>,<a onclick="LoadInParent('#B120'); return false;" href="#B120">120</a>]), <em>Pseudacontias</em> is nested inside <em>Madascincus</em>, and the genera <em>Androngo</em>, <em>Pygomeles</em>, and <em>Voeltzkowia</em> are all nested in <em>Amphiglossus</em> (Figure&nbsp;<a name="d6347e1963"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>).
      </p>
      <p style="line-height:160%">We also find numerous taxonomic problems within lygosomines (Figures&nbsp;<a name="d6347e1969"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e1972"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e1975"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e1978"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). Species of <em>Sphenomorphus</em> are widely dispersed among other lygosomine genera. The genus <em>Tropidophorus</em> is paraphyletic with respect to a clade containing many other genera (Figure&nbsp;<a name="d6347e1988"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>). The sampled species of <em>Asymblepharus</em> are only distantly related to each other, including one species (<em>A. sikimmensis</em>) nested inside of <em>Scincella</em> (Figure&nbsp;<a name="d6347e2000"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>). The genus <em>Lipinia</em> is polyphyletic, with one species (<em>L. vittigera</em>) strongly placed as the sister taxon to <em>Isopachys</em>, and with two other species (<em>L. pulchella</em> and <em>L. noctua</em>) placed in a well-supported clade that also includes <em>Papuascincus</em> (Figure&nbsp;<a name="d6347e2023"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>).
      </p>
      <p style="line-height:160%">Among Australian skinks, the genus <em>Eulamprus</em> is polyphyletic with respect to <em>Nangura</em>, <em>Calyptotis</em>, <em>Gnypetoscincus</em>, <em>Coggeria</em>, <em>Coeranoscincus</em>, <em>Ophioscincus</em>, <em>Saiphos</em>, <em>Anomalopus</em>, <em>Eremiascincus</em>, <em>Hemiergis</em>, <em>Glaphyromorphus</em>, <em>Notoscincus</em>, <em>Ctenotus</em>, and <em>Lerista</em>, and most of the relevant nodes are strongly supported (Figure&nbsp;<a name="d6347e2075"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>). The genera <em>Coeranoscincus</em> and <em>Ophioscincus</em> are polyphyletic with respect to each other and to <em>Saiphos</em> and <em>Coggeria</em> (Figure&nbsp;<a name="d6347e2091"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>). The genus <em>Glaphyromorphus</em> is paraphyletic with respect to a clade of <em>Eulamprus</em> (Figure&nbsp;<a name="d6347e2101"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>). The genus <em>Egernia</em> is paraphyletic with respect to <em>Bellatorias</em> (which is paraphyletic with respect to <em>Egernia</em> and <em>Lissolepis</em>) and <em>Lissolepis</em>, although many of the relevant nodes are not strongly supported (Figure&nbsp;<a name="d6347e2120"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>). The genera <em>Cyclodomorphus</em> and <em>Tiliqua</em> are paraphyletic with respect to each other (Figure&nbsp;<a name="d6347e2129"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>).
      </p>
      <p style="line-height:160%">Among other lygosomines, <em>Trachylepis</em> is non-monophyletic <a name="d6347e2138"></a>[<a onclick="LoadInParent('#B121'); return false;" href="#B121">121</a>], with two species (<em>T. aurata</em> and <em>T. vittata</em>) that fall outside the strongly supported clade containing the other species (Figure&nbsp;<a name="d6347e2147"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>). The latter clade is weakly supported as the sister group to a clade containing
         <em>Chioninia</em>, <em>Eumecia</em>, and <em>Mabuya</em>. In <em>Mabuya</em>, a few species (<em>M. altamazonica</em>, <em>M. bistriata</em>, and <em>M. nigropuncata</em>) have unorthodox placements within a monophyletic <em>Mabuya</em>, potentially due to uncertain taxonomic assignment of specimens by previous authors
         <a name="d6347e2177"></a><a name="d6347e2179"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>,<a onclick="LoadInParent('#B122'); return false;" href="#B122">122</a>]. The genus <em>Lygosoma</em> is paraphyletic with respect to <em>Lepidothyris</em> and <em>Mochlus</em>, and many of the relevant nodes are strongly supported (Figure&nbsp;<a name="d6347e2192"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). Among New Caledonian skinks, the genus <em>Lioscincus</em> is polyphyletic with respect to <em>Marmorosphax</em>, <em>Celatiscincus,</em> and <em>Tropidoscincus</em>, and both <em>Lioscincus</em> and <em>Tropidoscincus</em> are paraphyletic with respect to, <em>Kanakysaurus, Lacertoides, Phoboscincus, Sigaloseps, Tropidoscincus, Graciliscincus,
            Simiscincus,</em> and <em>Caledoniscincus</em>, with strong support for most relevant nodes (Figure&nbsp;<a name="d6347e2220"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). The genera <em>Emoia</em> and <em>Bassiana</em> are massively polyphyletic and divided across multiple lygosomine clades (Figure&nbsp;<a name="d6347e2230"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>). The genus <em>Lygisaurus</em> appears to be nested inside of <em>Carlia</em>, although many of the relevant branches are only weakly supported (Figure&nbsp;<a name="d6347e2239"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>).
      </p>
      <h4>Lacertoidea</h4>
      <p style="line-height:160%">Within Lacertoidea (Figure&nbsp;<a name="d6347e2248"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), we corroborate recent molecular analyses (e.g. <a name="d6347e2252"></a><a name="d6347e2254"></a><a name="d6347e2256"></a><a name="d6347e2258"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a></a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]) and morphology-based phylogenies and classifications (e.g. <a name="d6347e2262"></a><a name="d6347e2264"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]) in supporting the clade including the New World families Gymnophthalmidae and Teiidae
         (Figure&nbsp;<a name="d6347e2267"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). Within a weakly supported Teiidae (Figure&nbsp;<a name="d6347e2270"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>), the subfamilies Tupinambinae and Teiinae are each strongly supported as monophyletic,
         as in previous studies <a name="d6347e2275"></a>[<a onclick="LoadInParent('#B61'); return false;" href="#B61">61</a>]. In Tupinambinae, <em>Callopistes</em> is the sister group to a clade containing <em>Tupinambis</em>, <em>Dracaena</em>, and <em>Crocodilurus</em>. The clade <em>Dracaena + Crocodilurus</em> is nested within <em>Tupinambis</em>, and the associated clades have strong support (Figure&nbsp;<a name="d6347e2297"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). We find that the teiine genera <em>Ameiva</em> and <em>Cnemidophorus</em> are non-monophyletic (Figure&nbsp;<a name="d6347e2306"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>), interdigitating with each other and the monophyletic genera <em>Aspidoscelis</em>, <em>Dicrodon</em> (monotypic), and <em>Kentropyx</em>, as in previous phylogenies <a name="d6347e2320"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>].
      </p>
      <p style="line-height:160%">A recent study <a name="d6347e2326"></a>[<a onclick="LoadInParent('#B123'); return false;" href="#B123">123</a>] proposed a re-classification of the family Teiidae based on analysis of 137 morphological
         characters for 101 terminal species (with ~150 species in the family <a name="d6347e2330"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]). Those authors erected several new genera and subfamilies in an attempt to deal
         with the apparent non-monophyly of currently recognized taxa in their tree. However,
         in our tree, some of these new taxa conflict strongly with the phylogeny or are rendered
         unnecessary. First, they recognize Callopistinae as a distinct subfamily for <em>Callopistes</em>, arguing that failure to do so would produce a taxonomy inconsistent with teiid phylogeny.
         However, we find strong support for <em>Callopistes</em> in its traditional placement as part of Tupinambinae (Figure&nbsp;<a name="d6347e2339"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>), and this change is thus not needed based on our results. The genus <em>Ameiva</em> is paraphyletic under traditional definitions <a name="d6347e2347"></a>[<a onclick="LoadInParent('#B62'); return false;" href="#B62">62</a>]. In our tree, their conception of <em>Ameiva</em> is also non-monophyletic, with species found in three distinct clades (Figure&nbsp;<a name="d6347e2353"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). We also find non-monophyly of many of their species groups within <em>Ameiva</em>, including the <em>ameiva, bifrontata, dorsalis,</em> and <em>erythrocephala</em> groups (Figure&nbsp;<a name="d6347e2366"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). Their genera <em>Aurivela</em> (<em>Cnemidophorus longicaudus</em>) and <em>Contomastyx</em> (<em>Cnemidophorus lacertoides</em>) are strongly supported as sister taxa in our tree, and are nested within <em>Ameiva</em> in their <em>erythrocephala</em> species group, along with <em>Dicrodon</em> (Figure&nbsp;<a name="d6347e2391"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>).
      </p>
      <p style="line-height:160%">On the positive side, many of the genera they recognize are monophyletic and are not
         nested in other genera in our tree, including their <em>Ameivula</em> (<em>Cnemidophorus ocellifer</em>)<em>, Aspidoscelis</em> (unchanged from previous definitions), <em>Cnemidophorus</em> (excluding <em>C. ocellifer, C. lacertoides,</em> and <em>C. longicaudus</em>)<em>, Holcosus</em> (<em>Ameiva undulata</em>, <em>A. festiva</em>, and <em>A. quadrilineatus</em>)<em>, Kentropyx</em> (unchanged from previous definitions), <em>Salvator</em> (<em>Tupinambis rufescens, T. duseni</em>, and <em>T. merianae</em>), and <em>Teius</em> (unchanged from previous definitions). We did not sample <em>Ameiva edracantha</em> (their <em>Medopheos</em>).
      </p>
      <p style="line-height:160%">Given our results, major taxonomic rearrangements within Teiidae seem problematic
         at present, especially with the extensive paraphyly of many traditional and re-defined
         teiid genera, the lack of strong resolution of many of these relationships based on
         molecular and morphological data, and incomplete taxon sampling in all studies so
         far. Thus, we provisionally retain the traditional taxonomy of Teiidae, pending additional
         data and analyses. However, we note that <em>Ameiva, Cnemidophorus</em>, and <em>Tupinambis</em> are clearly non-monophyletic based on both our results and those of recent authors
         <a name="d6347e2458"></a>[<a onclick="LoadInParent('#B123'); return false;" href="#B123">123</a>], and will require taxonomic changes in the future. We anticipate that many of these
         newly proposed genera <a name="d6347e2462"></a>[<a onclick="LoadInParent('#B123'); return false;" href="#B123">123</a>] will be useful in such revisions.
      </p>
      <p style="line-height:160%">We find strong support (SHL = 98) for monophyly of Gymnophthalmidae (Figure&nbsp;<a name="d6347e2467"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). Within Gymnophthalmidae, we find strong support for the monophyly of the previously
         recognized subfamilies <a name="d6347e2471"></a><a name="d6347e2473"></a><a name="d6347e2475"></a>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a></a>,<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>,<a onclick="LoadInParent('#B124'); return false;" href="#B124">124</a>], with the exception of Cercosaurinae (Figure&nbsp;<a name="d6347e2478"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). Previous researchers considered the genus <em>Bachia</em> a distinct tribe (Bachiini) within Cercosaurinae, based on a poorly supported sister-group
         relationship with the tribe Cercosaurini <a name="d6347e2485"></a><a name="d6347e2487"></a>[<a onclick="LoadInParent('#B63'); return false;" href="#B63">63</a></a>,<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>]. Here, we find a moderately well supported relationship (SHL = 84) between <em>Bachia</em> and Gymnophthalminae + Rhachisaurinae, and we find that this clade is only distantly
         related to other Cercosaurinae. Therefore, we restrict Cercosaurinae to the tribe
         Cercosaurini, and elevate the tribe Bachiini <a name="d6347e2495"></a>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>] to the subfamily level. The subfamily Bachiinae contains only the genus <em>Bachia</em> (Figure&nbsp;<a name="d6347e2501"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>), identical in content to the previously recognized tribe <a name="d6347e2505"></a>[<a onclick="LoadInParent('#B64'); return false;" href="#B64">64</a>]. Within Cercosaurinae, we find that the genus <em>Petracola</em> is nested within <em>Proctoporus</em> (Figure&nbsp;<a name="d6347e2515"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>). In Ecpleopinae, <em>Leposoma</em> is divided into two clades, separated by <em>Anotosaura</em>, <em>Colobosauroides</em>, and <em>Arthrosaura</em> (Figure&nbsp;<a name="d6347e2531"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>), and many of the relevant nodes are very strongly supported. These issues should
         be addressed in future studies.
      </p>
      <p style="line-height:160%">Our results show strong support for a clade uniting Lacertidae and Amphisbaenia (Figure&nbsp;<a name="d6347e2536"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), as in many previous studies <a name="d6347e2540"></a><a name="d6347e2542"></a><a name="d6347e2544"></a><a name="d6347e2546"></a><a name="d6347e2548"></a><a name="d6347e2550"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. We also find strong support for monophyly of amphisbaenians (SHL = 99), in contrast
         to some molecular analyses <a name="d6347e2554"></a><a name="d6347e2556"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. Relationships among amphisbaenian families are generally strongly supported and
         similar to those in earlier molecular studies (Figure&nbsp;<a name="d6347e2559"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12">12</a>), including the placement of the New World family Rhineuridae as sister group to
         all other amphisbaenians <a name="d6347e2563"></a><a name="d6347e2565"></a><a name="d6347e2567"></a>[<a onclick="LoadInParent('#B70'); return false;" href="#B70">70</a></a>,<a onclick="LoadInParent('#B71'); return false;" href="#B71">71</a>,<a onclick="LoadInParent('#B125'); return false;" href="#B125">125</a>]. The family Cadeidae is placed as the sister-group to Amphisbaenidae + Trogonophiidae
         (Figures&nbsp;<a name="d6347e2571"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a> and Figure&nbsp;<a name="d6347e2574"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12">12</a>) with weak support, but has been placed with Blanidae in previous studies, with strong
         support but less- extensive taxon sampling <a name="d6347e2578"></a><a name="d6347e2580"></a>[<a onclick="LoadInParent('#B125'); return false;" href="#B125">125</a></a>,<a onclick="LoadInParent('#B126'); return false;" href="#B126">126</a>].
      </p>
      <p style="line-height:160%">We find strong support for monophyly of the Old World family Lacertidae (Figure&nbsp;<a name="d6347e2585"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>). Within Lacertidae, branch support for the monophyly of most genera and for the
         subfamilies Gallotiinae and Lacertinae is very high (Figure&nbsp;<a name="d6347e2588"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>). However, we find that relationships among many genera are poorly supported, as
         in previous studies <a name="d6347e2592"></a><a name="d6347e2594"></a><a name="d6347e2596"></a>[<a onclick="LoadInParent('#B65'); return false;" href="#B65">65</a>,<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a></a>,<a onclick="LoadInParent('#B68'); return false;" href="#B68">68</a>]. Our results (Figure&nbsp;<a name="d6347e2599"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>) also indicate that several lacertid genera are non-monophyletic with strong support
         for the associated nodes, including <em>Algyroides</em> (paraphyletic with respect to <em>Dinarolacerta</em>), <em>Ichnotropis</em> (paraphyletic with respect to <em>Meroles</em>), <em>Meroles</em> (paraphyletic with respect to <em>Ichnotropis</em>), <em>Nucras</em> (polyphyletic with respect to several genera, including <em>Pedioplanis</em>, <em>Poromera</em>, <em>Latastia</em>, <em>Philocortus</em>, <em>Pseuderemias</em>, and <em>Heliobolus</em>), and <em>Pedioplanis</em> (paraphyletic with respect to <em>Nucras</em>).
      </p>
      <h4>Higher-level phylogeny of Toxicofera</h4>
      <p style="line-height:160%">We find strong support (SHL = 96) for monophyly of Toxicofera (Anguimorpha, Iguania,
         and Serpentes; Figure&nbsp;<a name="d6347e2656"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), and moderate support for a sister-group relationship between Iguania and Anguimorpha
         (SHL = 79). Relationships among Anguimorpha, Iguania, and Serpentes were weakly supported
         in some Bayesian and likelihood analyses <a name="d6347e2660"></a><a name="d6347e2662"></a><a name="d6347e2664"></a><a name="d6347e2666"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>], but strongly supported in others <a name="d6347e2670"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. We further corroborate previous studies in also placing Anguimorpha with Iguania
         <a name="d6347e2674"></a><a name="d6347e2676"></a><a name="d6347e2678"></a><a name="d6347e2680"></a><a name="d6347e2682"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. In contrast, some other studies have placed anguimorphs with snakes as the sister
         group to iguanians <a name="d6347e2686"></a><a name="d6347e2688"></a>[<a onclick="LoadInParent('#B127'); return false;" href="#B127">127</a></a>,<a onclick="LoadInParent('#B128'); return false;" href="#B128">128</a>].
      </p>
      <h4>Anguimorpha</h4>
      <p style="line-height:160%">Our hypothesis for family-level anguimorphan relationships (Figures&nbsp;<a name="d6347e2697"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e2700"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>) is generally similar to that of other recent studies <a name="d6347e2704"></a><a name="d6347e2706"></a><a name="d6347e2708"></a><a name="d6347e2710"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B129'); return false;" href="#B129">129</a>], and is strongly supported. Our results differ from some analyses based only on morphology,
         which place Anguidae near the base of Anguimorpha <a name="d6347e2714"></a>[<a onclick="LoadInParent('#B130'); return false;" href="#B130">130</a>]. Here, Shinisauridae is strongly supported as the sister taxon to a well-supported
         clade of Varanidae + Lanthanotidae (Figures&nbsp;<a name="d6347e2717"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e2721"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>). Varanid relationships are similar to previous estimates (e.g. <a name="d6347e2725"></a>[<a onclick="LoadInParent('#B131'); return false;" href="#B131">131</a>]). Xenosauridae is here strongly supported as the sister-group to a strongly supported
         clade containing Helodermatidae and the strongly supported Anniellidae + Anguidae
         clade (Figures&nbsp;<a name="d6347e2728"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e2731"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>). However, previous molecular analyses have placed Helodermatidae as the sister to
         Xenosauridae + (Anniellidae + Anguidae), typically with strong support <a name="d6347e2735"></a><a name="d6347e2737"></a><a name="d6347e2739"></a><a name="d6347e2741"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a></a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>].
      </p>
      <p style="line-height:160%">Within Anguidae (Figure&nbsp;<a name="d6347e2746"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>), our phylogeny indicates non-monophyly of genera within every subfamily, including
         Diploglossinae (<em>Diploglossus</em> and <em>Celestus</em> are strongly supported as paraphyletic with respect to each other and to <em>Ophiodes</em>), Anguinae (<em>Ophisaurus</em> is strongly supported as paraphyletic with respect to <em>Anguis</em>, <em>Dopasia</em>, and <em>Pseudopus</em>), and Gerrhonotinae (<em>Abronia</em> and <em>Mesaspis</em> are non-monophyletic, and <em>Coloptychon</em> is nested inside <em>Gerrhonotus</em>). Some of these problems were not reported previously (e.g. <em>Coloptychon, Abronia</em>, and <em>Mesaspis</em>), due to incomplete taxon sampling in previous studies <a name="d6347e2791"></a><a name="d6347e2793"></a><a name="d6347e2795"></a>[<a onclick="LoadInParent('#B129'); return false;" href="#B129">129</a>,<a onclick="LoadInParent('#B132'); return false;" href="#B132">132</a></a>,<a onclick="LoadInParent('#B133'); return false;" href="#B133">133</a>], but relationships within Gerrhonotinae are under detailed investigation by other
         researchers, so these issues are likely to be resolved in the near future.
      </p>
      <h4>Iguania</h4>
      <p style="line-height:160%">We find strong support (SHL = 100) for the monophyly of Iguania (Figure&nbsp;<a name="d6347e2804"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>). This clade is strongly supported by nuclear data <a name="d6347e2808"></a><a name="d6347e2810"></a><a name="d6347e2812"></a><a name="d6347e2814"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a></a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], but an apparent episode of convergent molecular evolution in several mitochondrial
         genes has seemingly misled some analyses of mtDNA, leading to weak support for Iguania
         <a name="d6347e2818"></a>[<a onclick="LoadInParent('#B134'); return false;" href="#B134">134</a>], or even separation of the acrodonts and pleurodonts <a name="d6347e2822"></a><a name="d6347e2824"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B135'); return false;" href="#B135">135</a>] in previous studies. Within Iguania (Figure&nbsp;<a name="d6347e2827"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), we find strong support (SHL = 100) for a sister-group relationship between Chamaeleonidae
         and Agamidae (Acrodonta), and for a clade of mostly New World families (Pleurodonta;
         SHL = 100).
      </p>
      <p style="line-height:160%">We find strong support for the monophyly of Chamaeleonidae and the subfamily Chamaeleoninae,
         and weak support for the paraphyly of Brookesiinae (Figures&nbsp;<a name="d6347e2832"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e2835"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15">15</a>). The sampled species of the <em>Brookesia nasus</em> group appear as the sister group to all other chamaeleonids (the latter clade weakly
         supported) as found by some previous authors <a name="d6347e2842"></a>[<a onclick="LoadInParent('#B136'); return false;" href="#B136">136</a>], though other studies have recovered a monophyletic <em>Brookesia</em><a name="d6347e2848"></a><a name="d6347e2850"></a>[<a onclick="LoadInParent('#B137'); return false;" href="#B137">137</a></a>,<a onclick="LoadInParent('#B138'); return false;" href="#B138">138</a>]. Within Chamaeleoninae (Figure&nbsp;<a name="d6347e2854"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15">15</a>), we find strong support for the monophyly of most genera and species-level relationships.
         However, we find strong support for the non-monophyly of <em>Calumma</em>, with some species strongly placed with <em>Chamaeleo</em>, others strongly placed with <em>Rieppeleon</em>, and a third set weakly placed with <em>Nadzikambia</em> + <em>Rhampoleon.</em> While non-monophly of <em>Calumma</em> has also been found in previous studies <a name="d6347e2877"></a>[<a onclick="LoadInParent('#B138'); return false;" href="#B138">138</a>], a recent study strongly supports monophyly of <em>Brookesia</em> and weakly supports monophyly of <em>Calumma</em><a name="d6347e2886"></a>[<a onclick="LoadInParent('#B139'); return false;" href="#B139">139</a>].
      </p>
      <p style="line-height:160%">Monophyly of Agamidae is strongly supported (Figure&nbsp;<a name="d6347e2891"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>; SHL = 100), contrary to some previous estimates <a name="d6347e2895"></a><a name="d6347e2897"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>]. Most relationships among agamid subfamilies and genera are strongly supported (Figure&nbsp;<a name="d6347e2900"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>), and largely congruent with earlier studies <a name="d6347e2904"></a><a name="d6347e2906"></a><a name="d6347e2908"></a><a name="d6347e2910"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>,<a onclick="LoadInParent('#B140'); return false;" href="#B140">140</a>]. There are some differences with earlier studies. For example, previous studies based
         on 29â€“44 loci <a name="d6347e2914"></a><a name="d6347e2916"></a><a name="d6347e2918"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>] placed Hydrosaurinae as sister to Amphibolurinae + (Agaminae + Draconinae) with strong
         support, whereas we place Hydrosaurinae as the sister-group to Amphibolurinae with
         weak support. Other authors <a name="d6347e2923"></a>[<a onclick="LoadInParent('#B140'); return false;" href="#B140">140</a>] placed Leiolepiedinae with Uromastycinae, but we (and most other studies) place Uromastycinae
         as the sister group to all other agamids.
      </p>
      <p style="line-height:160%">Our phylogeny indicates several taxonomic problems within amphibolurine agamids (Figure&nbsp;<a name="d6347e2928"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>). The genera <em>Moloch</em> and <em>Chelosania</em> render <em>Hypsilurus</em> paraphyletic, although the support for the relevant clades is weak. The species <em>Lophognathus gilberti</em> is placed in a strongly supported clade with <em>Chlamydosaurus</em> and <em>Amphibolurus</em>, a clade that is not closely related to the other <em>Lophognathus</em>. Many of these taxonomic problems were also noted by previous authors <a name="d6347e2954"></a>[<a onclick="LoadInParent('#B141'); return false;" href="#B141">141</a>].
      </p>
      <p style="line-height:160%">Within agamine agamids (Figure&nbsp;<a name="d6347e2959"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>), most relationships are well supported and monophyly of all sampled genera is strongly
         supported. In contrast, within draconine agamids (Figure&nbsp;<a name="d6347e2962"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>), many intergeneric relationships are weakly supported, and some genera are non-monophyletic
         (Figure&nbsp;<a name="d6347e2965"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>; see also <a name="d6347e2969"></a>[<a onclick="LoadInParent('#B142'); return false;" href="#B142">142</a>]), including <em>Gonocephalus</em> (<em>G. robinsonii</em> is only distantly related to other <em>Gonocephalus</em>) and <em>Japalura</em> (with species distributed among three distantly related clades, including one allied
         with <em>Ptyctolaemus</em>, another with <em>G. robinsonii,</em> and a third with <em>Pseudocalotes</em>).
      </p>
      <p style="line-height:160%">Recent authors suggested dividing <em>Laudakia</em> into three genera (<em>Stellagama, Paralaudakia</em>, and <em>Laudakia</em>) based on a non-phylogenetic analysis of morphology <a name="d6347e3006"></a>[<a onclick="LoadInParent('#B143'); return false;" href="#B143">143</a>]. Here, <em>Laudakia</em> (as previously defined) is strongly supported as monophyletic (Figure&nbsp;<a name="d6347e3013"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>), and this change is not necessitated by the phylogeny. Similarly, based on genetic
         and morphological data, recent authors <a name="d6347e3017"></a>[<a onclick="LoadInParent('#B144'); return false;" href="#B144">144</a>] suggested resurrecting the genus <em>Saara</em> for the basal clade of <em>Uromastyx</em> (<em>U. asmussi, U. hardwickii,</em> and <em>U. loricata</em>). However, <em>Uromastyx</em> (as previously defined) is strongly supported as monophyletic in our results (Figure&nbsp;<a name="d6347e3036"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>) and in those of the recent revision <a name="d6347e3040"></a>[<a onclick="LoadInParent('#B144'); return false;" href="#B144">144</a>], and this change is not needed. We therefore retain <em>Laudakia</em> and <em>Uromastyx</em> as previously defined, to preserve taxonomic stability in these groups <a name="d6347e3050"></a>[<a onclick="LoadInParent('#B113'); return false;" href="#B113">113</a>]. We note that recent studies have also begun to revise species limits in other groups
         such as <em>Trapelus</em><a name="d6347e3057"></a>[<a onclick="LoadInParent('#B145'); return false;" href="#B145">145</a>], and taxa such as <em>T. pallidus</em> (Figure&nbsp;<a name="d6347e3063"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>) may represent populations within other species.
      </p>
      <p style="line-height:160%">Within Pleurodonta we generally confirm the monophyly and composition of the clades
         that were ranked as families (or subfamilies) within the group (e.g. Phrynosomatidae,
         Opluridae, Leiosauridae, Leiocephalidae, and Corytophanidae; Figures&nbsp;<a name="d6347e3068"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3071"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e3074"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e3077"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>) based on previous molecular studies <a name="d6347e3081"></a><a name="d6347e3083"></a><a name="d6347e3085"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a></a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>] and earlier morphological analyses <a name="d6347e3090"></a><a name="d6347e3092"></a>[<a onclick="LoadInParent('#B146'); return false;" href="#B146">146</a></a>,<a onclick="LoadInParent('#B147'); return false;" href="#B147">147</a>].
      </p>
      <p style="line-height:160%">One important exception is the previously recognized Polychrotidae. Our results confirm
         that <em>Anolis</em> and <em>Polychrus</em> are not sister taxa (Figures&nbsp;<a name="d6347e3103"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3106"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e3109"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>), as also found in some previous molecular studies <a name="d6347e3114"></a><a name="d6347e3116"></a><a name="d6347e3118"></a>[<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a></a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>], but not others <a name="d6347e3122"></a><a name="d6347e3124"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Our results provide strong support for non-monophyly of Polychrotidae, placing <em>Polychrus</em> with Hoplocercidae (SHL = 99) and <em>Anolis</em> with Corytophanidae (SHL = 99; the latter also found by <a name="d6347e3134"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]). Recent analyses placing <em>Anolis</em> with <em>Polychrus</em> showed only weak support for this relationship <a name="d6347e3145"></a><a name="d6347e3147"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>], despite many loci (30â€“44). We support continued recognition of Dactyloidae for <em>Anolis</em> and Polychrotidae for <em>Polychrus</em><a name="d6347e3156"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>], based on a limited number of loci but extensive taxon sampling. We note that these
         families are still monophlyetic, even if they prove to be sister taxa.
      </p>
      <p style="line-height:160%">Interestingly, our results for relationships among pleurodont families differ from
         most previous studies, and are surprisingly well-supported in some cases by SHL values
         (but see below). In previous studies, many relationships among pleurodont families
         were poorly supported by Bayesian posterior probabilities and by parsimony and likelihood
         bootstrap values, though typically sampling fewer taxa or characters <a name="d6347e3162"></a><a name="d6347e3164"></a><a name="d6347e3166"></a><a name="d6347e3168"></a><a name="d6347e3170"></a><a name="d6347e3172"></a><a name="d6347e3174"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B31'); return false;" href="#B31">31</a>,<a onclick="LoadInParent('#B33'); return false;" href="#B33">33</a>,<a onclick="LoadInParent('#B148'); return false;" href="#B148">148</a>-<a onclick="LoadInParent('#B151'); return false;" href="#B151">151</a>]. Studies including 29 nuclear loci found strong concatenated Bayesian support for
         many relationships but weak support from ML bootstrap analyses for many of the same
         relationships <a name="d6347e3178"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]. The latter pattern (typically weak ML support) was also found in an analysis including
         those same 29 loci and mitochondrial data for &gt;150 species <a name="d6347e3182"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. We also find a mixture of strongly and weakly supported clades, but with many relationships
         that are incongruent with these previous studies. First, we find that Tropiduridae
         is weakly supported as the sister group to all other pleurodonts (also found by <a name="d6347e3186"></a>[<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]), followed successively (Figures&nbsp;<a name="d6347e3189"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3193"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e3196"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e3199"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>) by Iguanidae, Leiocephalidae, Crotaphytidae + Phrynosomatidae, Polychrotidae + Hoplocercidae,
         and Corytophanidae + Dactyloidae.
      </p>
      <p style="line-height:160%">The relatively strong support for the clades Crotaphytidae + Phrynosomatidae (SHL
         = 87), Polychrotidae + Hoplocercidae (SHL = 99), and Corytophanidae + Dactyloidae
         (SHL = 99) is largely unprecedented in previous studies (although Corytophanidae +
         Dactyloidae is strongly supported in some Bayesian analyses <a name="d6347e3205"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]). As in many previous analyses, deeper relationships among the families remain weakly
         supported. We also find a strongly supported clade containing Liolaemidae, Opluridae,
         and Leiosauridae (SHL = 95), with Opluridae + Leiosauridae also strongly supported
         (SHL = 99). Both clades have also been found in previous studies <a name="d6347e3209"></a><a name="d6347e3211"></a>[<a onclick="LoadInParent('#B149'); return false;" href="#B149">149</a>,<a onclick="LoadInParent('#B151'); return false;" href="#B151">151</a>], including studies based on 29 or more nuclear loci <a name="d6347e3215"></a><a name="d6347e3217"></a><a name="d6347e3219"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>,<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>].
      </p>
      <p style="line-height:160%">We note that previous studies have shown strong support for some pleurodont relationships
         (e.g. basal placement of phrynosomatids; see <a name="d6347e3226"></a>[<a onclick="LoadInParent('#B34'); return false;" href="#B34">34</a>]), only to be strongly overturned with additional data <a name="d6347e3230"></a><a name="d6347e3232"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B29'); return false;" href="#B29">29</a>]. Therefore, the relationships found here should be taken with some caution (even
         if strongly supported), with the possible exception of the recurring clade of Liolaemidae
         + (Opluridae + Leiosauridae).
      </p>
      <p style="line-height:160%">All pleurodont families are strongly supported as monophyletic (SHL &gt; 85). Within
         the pleurodont families, our results generally support the current generic-level taxonomy
         (Figures&nbsp;<a name="d6347e3237"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e3240"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e3243"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>). However, there are some exceptions. Within Tropiduridae (Figure&nbsp;<a name="d6347e3246"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>), <em>Tropidurus</em> is paraphyletic with respect to <em>Eurolophosaurus</em>, <em>Strobilurus</em>, <em>Uracentron</em>, and <em>Plica</em>. Within Opluridae (Figure&nbsp;<a name="d6347e3265"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>), the monotypic genus <em>Chalarodon</em> renders <em>Oplurus</em> paraphyletic. Two leiosaurid genera are also problematic (Figure&nbsp;<a name="d6347e3275"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>). In Enyaliinae, <em>Anisolepis</em> is paraphyletic with respect to <em>Urostrophus</em>, and this clade is nested within <em>Enyalius</em>. In Leiosaurinae, <em>Pristidactylus</em> is rendered paraphyletic by <em>Leiosaurus</em> and <em>Diplolaemus</em> (Figure&nbsp;<a name="d6347e3297"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>).
      </p>
      <p style="line-height:160%">Within Dactyloidae, a recent study re-introduced a more subdivided classification
         of anoles <a name="d6347e3303"></a>[<a onclick="LoadInParent('#B152'); return false;" href="#B152">152</a>], an issue that has been debated extensively in the past <a name="d6347e3307"></a><a name="d6347e3309"></a><a name="d6347e3311"></a><a name="d6347e3313"></a>[<a onclick="LoadInParent('#B153'); return false;" href="#B153">153</a>-<a onclick="LoadInParent('#B156'); return false;" href="#B156">156</a>]. Our results support the monophyly of all the genera recognized by recent authors
         <a name="d6347e3317"></a>[<a onclick="LoadInParent('#B152'); return false;" href="#B152">152</a>], including <em>Anolis, Audantia</em>, <em>Chamaelinorops, Dactyloa, Deiroptyx, Norops,</em> and <em>Xiphosurus</em> (see Figure&nbsp;<a name="d6347e3330"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>). However, since <em>Anolis</em> is monophyletic as traditionally defined, we retain that definition here (including
         the seven listed genera) for continuity with the recent literature <a name="d6347e3337"></a><a name="d6347e3339"></a>[<a onclick="LoadInParent('#B113'); return false;" href="#B113">113</a>,<a onclick="LoadInParent('#B157'); return false;" href="#B157">157</a>].
      </p>
      <h4>Serpentes</h4>
      <p style="line-height:160%">Relationships among the major serpent groups (Figure&nbsp;<a name="d6347e3348"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>) are generally similar to other recent studies <a name="d6347e3352"></a><a name="d6347e3354"></a><a name="d6347e3356"></a><a name="d6347e3358"></a><a name="d6347e3360"></a><a name="d6347e3362"></a><a name="d6347e3364"></a><a name="d6347e3366"></a><a name="d6347e3368"></a><a name="d6347e3370"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a></a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>,<a onclick="LoadInParent('#B158'); return false;" href="#B158">158</a>-<a onclick="LoadInParent('#B160'); return false;" href="#B160">160</a>]. We find that the blindsnakes, Scolecophidia (Figures&nbsp;<a name="d6347e3373"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3376"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>) are not monophyletic, as in previous studies <a name="d6347e3380"></a><a name="d6347e3382"></a><a name="d6347e3384"></a><a name="d6347e3386"></a><a name="d6347e3388"></a><a name="d6347e3390"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B159'); return false;" href="#B159">159</a></a>,<a onclick="LoadInParent('#B160'); return false;" href="#B160">160</a>]. Similar to some previous studies <a name="d6347e3395"></a><a name="d6347e3397"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B159'); return false;" href="#B159">159</a>], our data weakly place Anomalepididae as the sister taxon to all snakes, and the
         scolecophidian families Gerrhopilidae, Leptotyphlopidae, Typhlopidae, and Xenotyphlopidae
         as the sister-group to all other snakes excluding Anomalepididae (Figures&nbsp;<a name="d6347e3400"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3403"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>). Previous studies have also placed Anomalepididae as the sister-group to all non-scolecophidian
         snakes <a name="d6347e3407"></a><a name="d6347e3409"></a><a name="d6347e3411"></a><a name="d6347e3413"></a><a name="d6347e3415"></a><a name="d6347e3417"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a></a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B158'); return false;" href="#B158">158</a>,<a onclick="LoadInParent('#B160'); return false;" href="#B160">160</a>], in some cases with strong support <a name="d6347e3421"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. Although it might appear that recent analyses of scolecophidian relationships <a name="d6347e3425"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>] support monophyly of Scolecophidia (e.g. Figure&nbsp;<a name="d6347e3429"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a> of <a name="d6347e3433"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>]), the tree including non-snake outgroups from that study shows weak support for placing
         anomalepidids with alethinophidians, as in other studies <a name="d6347e3437"></a><a name="d6347e3439"></a><a name="d6347e3441"></a><a name="d6347e3443"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B160'); return false;" href="#B160">160</a>].
      </p>
      <p style="line-height:160%">We follow recent authors <a name="d6347e3449"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>] in recognizing Xenotyphlopidae (strongly placed as the sister taxon of Typhlopidae)
         and Gerrhopilidae (strongly placed as the sister group of Xenotyphlopidae + Typhlopidae)
         as distinct families (Figure&nbsp;<a name="d6347e3452"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>). Leptotyphlopidae is strongly supported as the sister group of a clade comprising
         Gerrhopilidae, Xenotyphlopidae, and Typhlopidae (Figure&nbsp;<a name="d6347e3455"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>). As in previous studies <a name="d6347e3459"></a><a name="d6347e3461"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>], we find strong support for non-monophyly of several typhlopid genera (<em>Afrotyphlops, Austrotyphlops, Ramphotyphlops</em>, <em>Letheobia</em>, and <em>Typhlops</em>; Figure&nbsp;<a name="d6347e3474"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>). There are also undescribed taxa (e.g. Typhlopidae sp. from Sri Lanka; <a name="d6347e3478"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]) of uncertain placement within this group (Figure&nbsp;<a name="d6347e3481"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>). The systematics of typhlopoid snakes will thus require extensive revision in the
         future, with additional taxon and character sampling.
      </p>
      <p style="line-height:160%">Within Alethinophidia (SHL = 100), Aniliidae is strongly supported (SHL = 98) as the
         sister taxon of Tropidophiidae (together comprising Anilioidea), and all other alethinophidians
         form a strongly supported sister group to this clade (SHL = 97; Figures&nbsp;<a name="d6347e3486"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3489"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). The enigmatic family Xenophidiidae is weakly placed as the sister-group to all
         alethinophidians exclusive of Anilioidea (Figures&nbsp;<a name="d6347e3492"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3495"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). The family Bolyeriidae is weakly placed as the sister-group to pythons, boas, and
         relatives (Booidea), which are strongly supported (SHL = 88). Relationships in this
         group are generally consistent with other recent molecular studies <a name="d6347e3499"></a><a name="d6347e3501"></a><a name="d6347e3503"></a><a name="d6347e3505"></a><a name="d6347e3507"></a><a name="d6347e3509"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B35'); return false;" href="#B35">35</a>-<a onclick="LoadInParent('#B37'); return false;" href="#B37">37</a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>,<a onclick="LoadInParent('#B159'); return false;" href="#B159">159</a>].
      </p>
      <p style="line-height:160%">Relationships among other alethinophidians are a mixture of strongly and weakly supported
         nodes (Figures&nbsp;<a name="d6347e3514"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3517"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). We find strong support (SHL = 100) for a clade containing Anomochilidae + Cylindrophiidae
         + Uropeltidae. This clade of three families is strongly supported (SHL = 89) as the
         sister taxon to Xenopeltidae + (Loxocemidae + Pythonidae). Together, these six families
         form a strongly supported clade (SHL = 89; Figures&nbsp;<a name="d6347e3520"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3523"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>) that is weakly supported as the sister group to the strongly supported clade of
         Boidae + Calabariidae. Within the clade of Anomochilidae, Cylindrophiidae, and Uropeltidae
         (Figure&nbsp;<a name="d6347e3526"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>), we weakly place <em>Anomochilus</em> as the sister group to Cylindrophiidae <a name="d6347e3534"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>], in contrast to previous studies which placed <em>Anomochilus</em> within <em>Cylindrophis</em><a name="d6347e3543"></a>[<a onclick="LoadInParent('#B161'); return false;" href="#B161">161</a>]. However, support for monophyly of <em>Cylindrophis</em> excluding <em>Anomochilus</em> is weak (Figure&nbsp;<a name="d6347e3553"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). As in previous studies <a name="d6347e3557"></a><a name="d6347e3559"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B162'); return false;" href="#B162">162</a>], we find several taxonomic problems within Uropeltidae (Figure&nbsp;<a name="d6347e3562"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). Specifically, <em>Rhinophis</em> and <em>Uropeltis</em> are paraphyletic with respect to each other and to <em>Pseudotyphlops</em>. The problematic taxa are primarily Sri Lankan <a name="d6347e3576"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>], and forthcoming analyses will address these issues.
      </p>
      <p style="line-height:160%">Within Pythonidae (Figure&nbsp;<a name="d6347e3581"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>), the genus <em>Python</em> is the sister group to all other genera. Some species that were traditionally referred
         to as <em>Python</em> (<em>P. reticulatus</em> and <em>P. timoriensis</em>) are instead sister to an Australasian clade consisting of <em>Antaresia</em>, <em>Apodora</em>, <em>Aspidites</em>, <em>Bothrochilus</em>, <em>Leiopython</em>, <em>Liasis</em>, and <em>Morelia</em> (Figure&nbsp;<a name="d6347e3619"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). These taxa (<em>P. reticulatus</em> and <em>P. timoriensis</em>) have been referred to as <em>Broghammerus</em>, a name originating from an act of "taxonomic vandalism" (i.e. an apparently intentional
         attempt to disrupt stable taxonomy) in a non-peer reviewed organ without data or analyses
         <a name="d6347e3632"></a><a name="d6347e3634"></a>[<a onclick="LoadInParent('#B163'); return false;" href="#B163">163</a></a>,<a onclick="LoadInParent('#B164'); return false;" href="#B164">164</a>]. However, this name was, perhaps inadvertently, subsequently used by researchers
         in peer-reviewed work <a name="d6347e3639"></a>[<a onclick="LoadInParent('#B165'); return false;" href="#B165">165</a>] and has entered into somewhat widespread usage <a name="d6347e3643"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. This name should be ignored and replaced with a suitable substitute. Within the
         Australasian clade (Figure&nbsp;<a name="d6347e3646"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>), <em>Morelia</em> is paraphyletic with respect to all other genera, and <em>Liasis</em> is non-monophyletic with respect to <em>Apodora</em>, although many of the relevant relationships are weakly supported.
      </p>
      <p style="line-height:160%">Within Boidae (Figure&nbsp;<a name="d6347e3660"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>), our results and those of other recent studies <a name="d6347e3664"></a><a name="d6347e3666"></a><a name="d6347e3668"></a><a name="d6347e3670"></a><a name="d6347e3672"></a><a name="d6347e3674"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a></a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>,<a onclick="LoadInParent('#B150'); return false;" href="#B150">150</a>,<a onclick="LoadInParent('#B166'); return false;" href="#B166">166</a>] have converged on estimated relationships that are generally similar to each other
         but which differ from traditional taxonomy <a name="d6347e3678"></a>[<a onclick="LoadInParent('#B167'); return false;" href="#B167">167</a>]. However, the classification has yet to be modified to reflect this, and we rectify
         this situation here. We find that Calabariidae is nested within Boidae <a name="d6347e3682"></a>[<a onclick="LoadInParent('#B150'); return false;" href="#B150">150</a>], but this is poorly supported, and contrary to most previous studies <a name="d6347e3686"></a><a name="d6347e3688"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a></a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>]. While <em>Calabaria</em> has been classified as an erycine boid in the past, this placement is strongly rejected
         here and in other studies <a name="d6347e3696"></a><a name="d6347e3698"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a></a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>]. If the current placement of <em>Calabaria</em> is supported in the future, it would require recognition as the subfamily Calabariinae
         in Boidae.
      </p>
      <p style="line-height:160%">The Malagasy boine genera <em>Acrantophis</em> and <em>Sanzinia</em> are placed as the sister taxa to a weakly-supported clade containing Calabariidae
         and a strongly supported clade (SHL = 99) comprising the currently recognized subfamilies
         Erycinae, Ungaliophiinae, and other boines (Figure&nbsp;<a name="d6347e3712"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>). Regardless of the position of Calabariidae, this placement of Malagasy boines renders
         Boinae paraphyletic. We therefore resurrect the subfamily Sanziniinae <a name="d6347e3716"></a>[<a onclick="LoadInParent('#B168'); return false;" href="#B168">168</a>] for <em>Acrantophis</em> and <em>Sanzinia</em>. This subfamily could be recognized as a distinct family if future studies also support
         placement of this clade as distinct from other Boidae + Calabariidae.
      </p>
      <p style="line-height:160%">The genera <em>Lichanura</em> and <em>Charina</em> are currently classified as erycines <a name="d6347e3735"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], but are strongly supported as the sister group to Ungaliophiinae, as in previous
         studies <a name="d6347e3739"></a><a name="d6347e3741"></a><a name="d6347e3743"></a><a name="d6347e3745"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>,<a onclick="LoadInParent('#B166'); return false;" href="#B166">166</a>]. We expand Ungaliophiinae to include these two genera (Figure&nbsp;<a name="d6347e3748"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>), rather than erect a new subfamily for these taxa. The subfamily Ungaliophiinae
         is placed as the sister group to a well-supported clade (SHL = 87) containing the
         rest of the traditionally recognized Erycinae and Boinae. We restrict Erycinae to
         the Old World genus <em>Eryx</em>.
      </p>
      <p style="line-height:160%">The genus <em>Candoia</em> (Boinae) from Oceania and New Guinea <a name="d6347e3761"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], is placed as the sister taxon to a moderately supported clade (SHL = 83) consisting
         of Erycinae (<em>Eryx</em>) and the remaining genera of Boinae (<em>Boa</em>, <em>Corallus</em>, <em>Epicrates</em>, and <em>Eunectes</em>). To solve the non-monophyly of Boinae with respect to Erycinae (due to <em>Candoia</em>), we place <em>Candoia</em> in a new subfamily (Candoiinae, <em>subfam. nov</em>.; see Appendix I). Boinae then comprises the four Neotropical genera that have traditionally
         been classified in this group (<em>Boa</em>, <em>Corallus</em>, <em>Epicrates</em>, and <em>Eunectes</em>). We acknowledge that non-monophyly of Boinae could be resolved in other ways (e.g.
         expanding it to include Erycinae). However, our taxonomy maintains the traditionally
         used subfamilies Boinae, Erycinae, and Ungaliophiinae, modifies them to reflect the
         phylogeny, and recognizes the phylogenetically distinct boine clades as separate subfamilies
         (Candoiinae, Sanziniinae). Within Boinae, <em>Eunectes</em> renders <em>Epicrates</em> paraphyletic, but this is not strongly supported (see also <a name="d6347e3809"></a>[<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>]).
      </p>
      <p style="line-height:160%">Our results for advanced snakes (Caenophidia) are generally similar to those of other
         recent studies <a name="d6347e3815"></a><a name="d6347e3817"></a><a name="d6347e3819"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a></a>,<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>,<a onclick="LoadInParent('#B169'); return false;" href="#B169">169</a>], and will only be briefly described. However, in contrast to most recent studies
         <a name="d6347e3823"></a><a name="d6347e3825"></a><a name="d6347e3827"></a><a name="d6347e3829"></a><a name="d6347e3831"></a><a name="d6347e3833"></a><a name="d6347e3835"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a></a>,<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a>,<a onclick="LoadInParent('#B81'); return false;" href="#B81">81</a>,<a onclick="LoadInParent('#B159'); return false;" href="#B159">159</a></a>,<a onclick="LoadInParent('#B160'); return false;" href="#B160">160</a>], Acrochordidae is here strongly placed (SHL = 95) as the sister group to Xenodermatidae.
         This clade is then the sister group to the remaining Colubroidea, which form a strongly
         supported clade (SHL = 100; Figures&nbsp;<a name="d6347e3838"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e3841"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>). This relationship has been found in some previous studies <a name="d6347e3845"></a><a name="d6347e3847"></a>[<a onclick="LoadInParent('#B169'); return false;" href="#B169">169</a></a>,<a onclick="LoadInParent('#B170'); return false;" href="#B170">170</a>], and was hypothesized by early authors <a name="d6347e3852"></a>[<a onclick="LoadInParent('#B171'); return false;" href="#B171">171</a>]. Further evidence will be required to resolve this conclusively. Analyses based on
         concatenation of 20â€“44 loci do not support this grouping <a name="d6347e3856"></a><a name="d6347e3858"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>], though preliminary species-tree analyses of &gt;400 loci do (Pyron et al., in prep.).
         Relationships in Pareatidae are similar to recent studies <a name="d6347e3862"></a>[<a onclick="LoadInParent('#B172'); return false;" href="#B172">172</a>], and the group is strongly placed as the sister taxon to colubroids excluding xenodermatids
         (SHL = 100; Figures&nbsp;<a name="d6347e3865"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a> , <a name="d6347e3868"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>), as in most recent analyses (e.g. <a name="d6347e3872"></a><a name="d6347e3874"></a><a name="d6347e3876"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a></a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]).
      </p>
      <p style="line-height:160%">The family Viperidae is the sister group to all colubroids excluding xenodermatids
         and pareatids (Figure&nbsp;<a name="d6347e3882"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), as in other recent studies. The family Viperidae is strongly supported (Figure&nbsp;<a name="d6347e3885"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>), as is the subfamily Viperinae, and the sister-group relationship between Azemiopinae
         and Crotalinae (SHL = 100). Our results generally support the existing generic-level
         taxonomy within Viperinae (Figure&nbsp;<a name="d6347e3888"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>). However, we recover a strongly supported clade within Viperinae consisting of <em>Daboia russelii</em>, <em>D</em>. <em>palaestinae</em>, <em>Macrovipera mauritanica</em>, and <em>M. deserti</em> (Figure&nbsp;<a name="d6347e3907"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>), as in previous studies <a name="d6347e3911"></a>[<a onclick="LoadInParent('#B173'); return false;" href="#B173">173</a>]. We corroborate previous suggestions that these taxa be included in <em>Daboia</em><a name="d6347e3918"></a>[<a onclick="LoadInParent('#B174'); return false;" href="#B174">174</a>], though this has not been widely adopted <a name="d6347e3922"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The other <em>Macrovipera</em> species (including the type species) remain in that genus (Figure&nbsp;<a name="d6347e3928"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>).
      </p>
      <p style="line-height:160%">Within Crotalinae (Figure&nbsp;<a name="d6347e3933"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>), a number of genera appear to be non-monophyletic. The species <em>Trimeresurus gracilis</em> is strongly supported as the sister taxon to <em>Ovophis okinavensis</em> and distantly related to other <em>Trimeresurus</em>, whereas the other <em>Ovophis</em> are strongly placed as the sister group to <em>Protobothrops</em>. A well-supported clade (SHL = 90) containing <em>Atropoides picadoi</em>, <em>Cerrophidion</em>, and <em>Porthidium</em> renders <em>Atropoides</em> paraphyletic (see also <a name="d6347e3965"></a>[<a onclick="LoadInParent('#B175'); return false;" href="#B175">175</a>]). The species <em>Bothrops pictus</em>, considered <em>incertae sedis</em> in previous studies <a name="d6347e3976"></a>[<a onclick="LoadInParent('#B176'); return false;" href="#B176">176</a>], is here strongly supported as the sister taxon to a clade containing <em>Rhinocerophis</em>, <em>Bothropoides</em>, <em>Bothriopsis</em>, and <em>Bothrops</em> (Figure&nbsp;<a name="d6347e3992"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>). Most of these relationships are strongly supported.
      </p>
      <p style="line-height:160%">Viperidae is strongly placed (SHL = 95) as the sister taxon to a well-supported clade
         (SHL = 100) containing Colubridae, Elapidae, Homalopsidae, and Lamprophiidae (Figure&nbsp;<a name="d6347e3997"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>). Monophyly of Homalopsidae is also strongly supported (Figure&nbsp;<a name="d6347e4000"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>). Within Homalopsidae, non-monophyly of the genus <em>Enhydris</em> is strongly supported (Figure&nbsp;<a name="d6347e4006"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>), and it should likely be split into multiple genera. Homalopsidae is weakly supported
         (SHL = 58) as the sister group of Elapidae + Lamprophiidae (Figure&nbsp;<a name="d6347e4009"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>). This same relationship was also weakly supported by previous analyses <a name="d6347e4014"></a><a name="d6347e4016"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>], but other studies have found strong support for placing Homalopsidae as the sister
         group of a strongly supported clade including Elapidae, Lamprophiidae, and Colubridae
         <a name="d6347e4020"></a><a name="d6347e4022"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>], including data from &gt;400 loci (Pyron et al., in prep.).
      </p>
      <p style="line-height:160%">Support for the monophyly of Lamprophiidae is strong (but excluding <em>Micrelaps</em>; see below), and most of its subfamilies are well-supported <a name="d6347e4031"></a><a name="d6347e4033"></a><a name="d6347e4035"></a><a name="d6347e4037"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a></a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B177'); return false;" href="#B177">177</a></a>,<a onclick="LoadInParent('#B178'); return false;" href="#B178">178</a>] including Atractaspidinae, Aparallactinae, Lamprophiinae, Prosymninae (weakly placed
         as the sister-group to <em>Oxyrhabdium</em>), Pseudaspidinae, Psammophiinae, and Pseudoxyrhophiinae (Figure&nbsp;<a name="d6347e4043"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>). In Lamprophiidae, most genera are monophyletic based on our sampling (Figure&nbsp;<a name="d6347e4046"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>). However, within Aparallactinae, <em>Xenocalamus</em> is strongly placed within <em>Amblyodipsas</em>, and in Atractaspidinae, <em>Homoroselaps</em> is weakly placed in <em>Atractaspis</em>. In Lamprophiinae, <em>Lamprophis</em> is paraphyletic with respect to <em>Lycodonomorphus</em> but support for the relevant clades is weak.
      </p>
      <p style="line-height:160%">The enigmatic genera <em>Buhoma</em> from Africa and <em>Psammodynastes</em> from Asia were both previously considered <em>incertae sedis</em> within Lamprophiidae <a name="d6347e4080"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. Here they are weakly placed as sister taxa, and more importantly, they form a strongly
         supported clade with the African genus <em>Pseudaspis</em> (Pseudaspidinae; SHL = 95; Figure&nbsp;<a name="d6347e4087"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>). Therefore, we expand Pseudaspidinae to include these two genera.
      </p>
      <p style="line-height:160%">The genus <em>Micrelaps</em> (putatively an aparallactine; <a name="d6347e4096"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]) is weakly placed as the sister taxon to Lamprophiidae + Elapidae. Along with <em>Oxyrhabdium</em> (see above) and <em>Montaspis</em><a name="d6347e4105"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a>], this genus is treated as <em>incertae sedis</em> in our classification (Appendix I). If future studies strongly support these relationships,
         they may require a new family for <em>Micrelaps</em> and possibly a new subfamily for <em>Oxyrhabdium</em>, though placement of these taxa has been highly variable in previous studies <a name="d6347e4119"></a><a name="d6347e4121"></a><a name="d6347e4123"></a><a name="d6347e4125"></a>[<a onclick="LoadInParent('#B40'); return false;" href="#B40">40</a></a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B81'); return false;" href="#B81">81</a>].
      </p>
      <p style="line-height:160%">Monophyly of Elapidae is strongly supported (Figure&nbsp;<a name="d6347e4130"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24">24</a>), and <em>Calliophis melanurus</em> is strongly supported as the sister group to all other elapids (see also <a name="d6347e4137"></a>[<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]). Within Elapidae (Figure&nbsp;<a name="d6347e4140"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24">24</a>), relationships are generally concordant with previous taxonomy, with some exceptions.
         The genera <em>Toxicocalamus</em>, <em>Simoselaps</em>, and <em>Echiopsis</em> are all divided across multiple clades, with strong support for many of the relevant
         branches. A recent study <a name="d6347e4154"></a>[<a onclick="LoadInParent('#B46'); return false;" href="#B46">46</a>] has provided a generic re-classification of the sea snakes (<em>Hydrophis</em> group) to resolve the extensive paraphyly of genera found in previous studies (e.g.
         <a name="d6347e4161"></a><a name="d6347e4163"></a>[<a onclick="LoadInParent('#B179'); return false;" href="#B179">179</a></a>,<a onclick="LoadInParent('#B180'); return false;" href="#B180">180</a>]). Our results support this classification.
      </p>
      <p style="line-height:160%">Monophyly of Colubridae and most of its subfamilies (sensu <a name="d6347e4169"></a><a name="d6347e4171"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]) are strongly supported (Figures&nbsp;<a name="d6347e4174"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e4177"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e4180"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>, <a name="d6347e4183"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>, <a name="d6347e4187"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>). However, relationships among many of these subfamilies are weakly supported (Figure&nbsp;<a name="d6347e4190"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), as in most previous studies <a name="d6347e4194"></a><a name="d6347e4196"></a><a name="d6347e4198"></a><a name="d6347e4200"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>,<a onclick="LoadInParent('#B45'); return false;" href="#B45">45</a>,<a onclick="LoadInParent('#B181'); return false;" href="#B181">181</a>]. The subfamilies Calamariinae and Pseudoxenodontinae are strongly supported as sister
         taxa, and weakly placed as the sister-group to the rest of Colubridae (Figure&nbsp;<a name="d6347e4203"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>). There is a weakly supported clade (Figure&nbsp;<a name="d6347e4206"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>) comprising Natricinae + (Dipsadinae + <em>Thermophis</em>), but the clade uniting the New World Dipsadinae with the Asian genus <em>Thermophis</em> is strongly supported (SHL = 100; Figure&nbsp;<a name="d6347e4216"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>). Here, we place <em>Thermophis</em> (recently in Pseudoxenodontinae <a name="d6347e4223"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]) in Dipsadinae (following <a name="d6347e4227"></a>[<a onclick="LoadInParent('#B182'); return false;" href="#B182">182</a>]), making it the first and only Asian member of this otherwise exclusively New World
         subfamily. However, despite the strong support for its placement here, placement of
         this taxon has been variable in previous analyses <a name="d6347e4231"></a><a name="d6347e4233"></a><a name="d6347e4235"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B182'); return false;" href="#B182">182</a></a>,<a onclick="LoadInParent('#B183'); return false;" href="#B183">183</a>], and we acknowledge that future analyses may support recognition of a distinct subfamily
         (Thermophiinae).
      </p>
      <p style="line-height:160%">The clade of Natricinae and Dipsadinae is weakly supported as the sister group (Figures&nbsp;<a name="d6347e4240"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e4243"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e4246"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>, <a name="d6347e4249"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>, <a name="d6347e4252"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>) to a clade containing Sibynophiinae <a name="d6347e4257"></a>[<a onclick="LoadInParent('#B181'); return false;" href="#B181">181</a>] + (Colubrinae + Grayiinae). The subfamily Colubrinae is weakly supported; we find
         that the colubrine genera <em>Ahaetulla</em>, <em>Chrysopelea</em>, and <em>Dendrelaphis</em> form a strongly supported clade that is weakly placed as the sister group to the
         rest of Colubrinae, which form a strongly supported clade (Figure&nbsp;<a name="d6347e4269"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>). This clade was also placed with Grayiinae or Sibynophiinae in many preliminary
         analyses, rendering Colubrinae paraphyletic. This group of three genera has been strongly
         supported in the past, and only weakly placed with Colubrinae <a name="d6347e4273"></a><a name="d6347e4275"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>]. It is possible that future analyses will reveal that the clade of <em>Ahaetulla, Chrysopelea,</em> and <em>Dendrelaphis</em> is placed elsewhere in Colubridae with strong support, and thus merit recognition
         as a distinct subfamily (Ahaetuliinae). A notable feature of this clade is the presence
         of gliding flight in most species of <em>Chrysopelea</em>, less-developed non-flight jumping with similar locomotor origins in <em>Dendrelaphis</em>, and homologous glide-related traits in <em>Ahaetulla</em><a name="d6347e4294"></a>[<a onclick="LoadInParent('#B184'); return false;" href="#B184">184</a>].
      </p>
      <p style="line-height:160%">Numerous colubroid genera are not included in our tree and are not clearly placed
         in subfamilies based on previous morphological evidence. In our classification, these
         genera are also considered <em>incertae sedis</em> within Colubridae, including <em>Blythia</em>, <em>Cyclocorus</em>, <em>Elapoidis</em>, <em>Gongylosoma</em>, <em>Helophis</em>, <em>Myersophis</em>, <em>Oreocalamus</em>, <em>Poecilopholis</em>, <em>Rhabdops</em>, and <em>Tetralepis</em>, as in previous classifications <a name="d6347e4335"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>].
      </p>
      <p style="line-height:160%">Our phylogeny reveals numerous taxonomic problems within Colubrinae (Figures&nbsp;<a name="d6347e4340"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e4343"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The genus <em>Boiga</em> is paraphyletic with respect to <em>Crotaphopeltis, Dipsadoboa, Telescopus, Toxicodryas,</em> and <em>Dasypeltis</em>, with strong support (Figure&nbsp;<a name="d6347e4356"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>). The genus <em>Philothamnus</em> is paraphyletic with respect to <em>Hapsidophrys</em> (Figure&nbsp;<a name="d6347e4365"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>). The genus <em>Coluber</em> is split between Old World and New World clades (Figures&nbsp;<a name="d6347e4371"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e4375"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The species <em>Hierophis spinalis</em> is sister to <em>Eirenis</em> to the exclusion of the other <em>Hierophis</em> species (Figure&nbsp;<a name="d6347e4387"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>). The genus <em>Dryocalamus</em> is nested within <em>Lycodon</em> (Figure&nbsp;<a name="d6347e4397"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The species <em>Chironius carinatus</em> and <em>C. quadricarinatus</em> are weakly placed in a clade of Neotropical colubrines only distantly related to
         the other <em>Chironius</em> species (Figure&nbsp;<a name="d6347e4409"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The genus <em>Drymobius</em> renders <em>Dendrophidion</em> paraphyletic (Figure&nbsp;<a name="d6347e4419"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The monotypic genus <em>Rhynchophis</em> renders the two species of <em>Rhadinophis</em> paraphyletic (Figure&nbsp;<a name="d6347e4428"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>). The genus <em>Coronella</em> is rendered paraphyletic (Figure&nbsp;<a name="d6347e4435"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>) by <em>Oocatochus</em> with weak support (see also <a name="d6347e4442"></a>[<a onclick="LoadInParent('#B185'); return false;" href="#B185">185</a>]). Finally, the genus <em>Rhinechis</em> is nested within <em>Zamenis</em> (Figure&nbsp;<a name="d6347e4452"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>).
      </p>
      <p style="line-height:160%">We find numerous non-monophyletic genera within Natricinae (Figure&nbsp;<a name="d6347e4458"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>), as in previous studies <a name="d6347e4462"></a><a name="d6347e4464"></a><a name="d6347e4466"></a><a name="d6347e4468"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B78'); return false;" href="#B78">78</a>,<a onclick="LoadInParent('#B186'); return false;" href="#B186">186</a>]. These non-monophyletic genera include the Asian genera <em>Amphiesma</em>, <em>Atretium</em>, and <em>Xenocrophis</em>. Among New World genera, we find <em>Regina</em> to be non-monophyletic with respect to most other genera, as in previous phylogenetic
         studies (e.g. <a name="d6347e4485"></a><a name="d6347e4487"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B186'); return false;" href="#B186">186</a>]). Also, as in previous studies (e.g. <a name="d6347e4491"></a><a name="d6347e4493"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B187'); return false;" href="#B187">187</a>]), we find that <em>Adelophis</em><a name="d6347e4499"></a>[<a onclick="LoadInParent('#B188'); return false;" href="#B188">188</a>] is nested deep within <em>Thamnophis</em><a name="d6347e4505"></a>[<a onclick="LoadInParent('#B189'); return false;" href="#B189">189</a>].
      </p>
      <p style="line-height:160%">Finally, within a weakly supported Dipsadinae (Figure&nbsp;<a name="d6347e4510"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>), we find non-monophyly of numerous genera, as in many earlier studies (e.g. <a name="d6347e4514"></a><a name="d6347e4516"></a><a name="d6347e4518"></a><a name="d6347e4520"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>-<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>,<a onclick="LoadInParent('#B190'); return false;" href="#B190">190</a>]). These problems of non-monophyly include <em>Leptodeira</em> (with respect to <em>Imantodes</em>), <em>Geophis</em> (with respect to <em>Atractus</em>), <em>Atractus</em> (with respect to <em>Geophis</em>), <em>Sibynomorphus</em> (with respect to <em>Dipsas</em>), <em>Dipsas</em> (with respect to <em>Sibynomorphus</em>), <em>Taeniophallus</em> (with respect to <em>Echinanthera</em>), and <em>Echinanthera</em> (with respect to <em>Taeniophallus</em>). Recent revisions have begun to tackle these problems <a name="d6347e4568"></a><a name="d6347e4570"></a><a name="d6347e4572"></a>[<a onclick="LoadInParent('#B42'); return false;" href="#B42">42</a></a>,<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>,<a onclick="LoadInParent('#B190'); return false;" href="#B190">190</a>], but additional taxon and character sampling will be crucial to resolve relationships
         and taxonomy.
      </p>
   </div>
</section>
<section><a name="sec3"></a><h3>Discussion</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">In this study, we provide a phylogenetic estimate for 4161 species of squamates based
         on molecular data from up to 12 genes per species, combining much of the relevant
         data used in previous molecular phylogenetic analyses. This tree provides a framework
         for future evolutionary studies, spanning from the species level to relationships
         among families, utilizing a common set of branch lengths. These estimated branch lengths
         are critically important for most phylogenetic comparative methods. To further facilitate
         use of this phylogeny in comparative studies, we provide the Newick version of this
         tree (with estimated branch lengths) in DataDryad repository 10.5061/dryad.82h0m and
         Additional file <a name="d6347e4581"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S1','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S1">1</a>: Data File S1. Our results also suggest that the branch lengths in this tree should
         not generally be compromised by missing data for some genes in some taxa.
      </p>
      <p style="line-height:160%">Our results also reveal many problems in squamate classification at nearly all phylogenetic
         levels. We make several changes to higher-level taxonomy based on this phylogeny,
         including changes to the traditionally recognized subfamilies of boid snakes (i.e.
         resurrecting Sanziniinae for the boine genera <em>Acrantophis</em> and <em>Sanzinia</em>, erecting Candoiinae for the boine genus <em>Candoia</em>, and moving <em>Lichanura</em> and <em>Charina</em> from Erycinae to Ungaliophiinae), lamprophiid snakes (expansion of Pseudaspidinae
         to include the formerly <em>incertae sedis</em> genera <em>Buhoma</em> and <em>Psammodynastes</em>), colubrid snakes (expansion of Dipsadinae to include the Asian pseudoxenodontine
         genus <em>Thermophis</em>), and gymnophthalmid lizards (recognition of Bachiinae for the tribe Bachiini, containing
         <em>Bachia</em>) and scincid lizards (synonymizing Feylininae with Scincinae to yield a total of
         three scincid subfamilies: Acontiinae, Lygosominae, and Scincinae). In Appendix I,
         we list the generic content of all families and subfamilies. We also find dozens of
         problems at the genus level, many of which have been identified previously, and which
         we defer the resolution of to future studies. Our results also highlight potential
         problems in recent proposals to modify the classification of scincid <a name="d6347e4618"></a>[<a onclick="LoadInParent('#B112'); return false;" href="#B112">112</a>] and teiid lizards <a name="d6347e4623"></a>[<a onclick="LoadInParent('#B123'); return false;" href="#B123">123</a>].
      </p>
      <p style="line-height:160%">In addition to synthesizing existing molecular data for squamate phylogeny, our analyses
         also reveal several apparently novel findings (Figures&nbsp;<a name="d6347e4628"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e4631"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F2','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F2">2</a>, <a name="d6347e4634"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F3','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F3">3</a>, <a name="d6347e4637"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F4','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F4">4</a>, <a name="d6347e4640"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F5','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F5">5</a>, <a name="d6347e4644"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e4647"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e4650"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e4653"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e4656"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>, <a name="d6347e4659"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>, <a name="d6347e4663"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F12','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F12">12</a>, <a name="d6347e4666"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F13','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F13">13</a>, <a name="d6347e4669"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>, <a name="d6347e4672"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F15','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F15">15</a>, <a name="d6347e4675"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F16','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F16">16</a>, <a name="d6347e4678"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e4682"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e4685"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>, <a name="d6347e4688"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F20','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F20">20</a>, <a name="d6347e4691"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F21','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F21">21</a>, <a name="d6347e4694"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F22','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F22">22</a>, <a name="d6347e4697"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F23','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F23">23</a>, <a name="d6347e4701"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F24','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F24">24</a>, <a name="d6347e4704"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F25','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F25">25</a>, <a name="d6347e4707"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F26','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F26">26</a>, <a name="d6347e4710"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F27','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F27">27</a>, <a name="d6347e4713"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F28','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F28">28</a>). Given space constraints, we cannot detail every deviation from previous phylogenetic
         hypotheses (especially pre-molecular studies). Nevertheless, we focus on three sets
         of examples. First, we find some relatively novel, strongly-supported relationships
         at the family level. These include the placement of Helodermatidae (as sister to Xenosauridae,
         Anguidae, and Anniellidae) and the placement of Xenodermatidae as the sister taxon
         to Acrochordidae (rendering Colubroidea paraphyletic), in contrast to most recent
         analyses of anguimorphs and snakes (see above). We also find some novel, strongly
         supported relationships among pleurodont families, but we acknowledge that these may
         be overturned in future studies.
      </p>
      <p style="line-height:160%">The second example is the higher-level relationships within Scincidae, the largest
         family of lizards <a name="d6347e4719"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. No previous studies examining higher-level relationships within the group included
         more than ~50 species <a name="d6347e4723"></a><a name="d6347e4725"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a></a>,<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>]. In this study, we sample 683 skink species (Figures&nbsp;<a name="d6347e4728"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e4731"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e4734"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e4738"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e4741"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>), and our phylogeny provides a unique resolution of higher-level skink relationships.
         Some previous researchers <a name="d6347e4745"></a>[<a onclick="LoadInParent('#B51'); return false;" href="#B51">51</a>] placed acontiines as the sister group to all other skinks, but suggested that scincines
         and lygosomines were paraphyletic with respect to each other (with feyliniines placed
         with scincines). In contrast, others <a name="d6347e4749"></a>[<a onclick="LoadInParent('#B50'); return false;" href="#B50">50</a>] suggested that acontiines, feyliniines, and lygosomines were all nested inside scincines
         (but with each of those three subfamilies as monophyletic), although many clades were
         only weakly supported. Here (Figures&nbsp;<a name="d6347e4752"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e4755"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F6','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F6">6</a>, <a name="d6347e4759"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F7','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F7">7</a>, <a name="d6347e4762"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F8','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F8">8</a>, <a name="d6347e4765"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F9','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F9">9</a>, <a name="d6347e4768"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F10','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F10">10</a>), we find that acontiines are the sister group to a strongly supported clade consisting
         of a monophyletic Scincinae and a monophyletic Lygosominae (excepting the weakly supported
         placement of <em>Ateuchosaurus</em> and placement of Feyliniinae in Scincinae).
      </p>
      <p style="line-height:160%">Third, our phylogeny reveals numerous genera that appear to be non-monophyletic, with
         many of these cases having strong support for the associated nodes. Our examples include
         genera in many families, including dibamids, diplodactylids, gekkonids, phyllodactylids,
         gerrhosaurids, scincids, teiids, gymnophthalmids, lacertids, anguids, chamaeleonids,
         agamids, tropidurids, oplurids, leiosaurids, typhlopids, pythonids, uropeltids, boids,
         viperids, lamprophiids, elapids, and colubrids (see Results). Although many problems
         noted here were found in previous studies, some seem to be new, such as placement
         of <em>Crocodilurus</em> and <em>Draceana</em> within <em>Tupinambis</em> (in Teiidae; Figure&nbsp;<a name="d6347e4785"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F11','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F11">11</a>) and <em>Coloptychon</em> within <em>Gerrhonotus</em> (in Anguidae; Figure&nbsp;<a name="d6347e4795"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F14','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F14">14</a>).
      </p>
      <p style="line-height:160%">Our study also offers an important test of higher-level squamate relationships using
         a very different sampling strategy than that used in most previous analyses. Squamates
         have traditionally been divided into two clades based on morphology, Iguania and Scleroglossa
         (e.g. <a name="d6347e4801"></a><a name="d6347e4803"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>,<a onclick="LoadInParent('#B21'); return false;" href="#B21">21</a>]). Despite considerable disagreement among morphology-based hypotheses, this basic
         division is supported by nearly all phylogenetic analyses based on morphological data
         <a name="d6347e4807"></a><a name="d6347e4809"></a><a name="d6347e4811"></a><a name="d6347e4813"></a><a name="d6347e4815"></a><a name="d6347e4817"></a><a name="d6347e4819"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B95'); return false;" href="#B95">95</a>,<a onclick="LoadInParent('#B191'); return false;" href="#B191">191</a>]. In contrast, our results and those of most previous molecular analyses strongly
         support placement of iguanians with anguimorphs and snakes <a name="d6347e4823"></a><a name="d6347e4825"></a><a name="d6347e4827"></a><a name="d6347e4829"></a><a name="d6347e4831"></a><a name="d6347e4833"></a><a name="d6347e4835"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a></a>,<a onclick="LoadInParent('#B24'); return false;" href="#B24">24</a>]. The causes of this conflict remain unclear, but may be related to morphological
         traits associated with different feeding strategies of iguanian and (traditional)
         scleroglossan squamates <a name="d6347e4839"></a><a name="d6347e4841"></a>[<a onclick="LoadInParent('#B3'); return false;" href="#B3">3</a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>].
      </p>
      <p style="line-height:160%">Additionally, analyses of morphology often place dibamids, amphisbaenians, snakes,
         and (in some cases) some scincids and anguids in a single clade <a name="d6347e4847"></a><a name="d6347e4849"></a><a name="d6347e4851"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>-<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]. Our analyses do not support such a clade (Figure&nbsp;<a name="d6347e4854"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), nor do other analyses of molecular data alone <a name="d6347e4858"></a><a name="d6347e4860"></a><a name="d6347e4862"></a><a name="d6347e4864"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], or analyses of combined molecular and morphological data <a name="d6347e4868"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>]. Instead, these morphological results seem to be explained by convergence associated
         with burrowing (e.g. <a name="d6347e4872"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>]). Overall, molecular datasets have shown overwhelmingly strong support for placement
         of dibamids and gekkonids at the base of the tree, amphisbaenians with lacertoids,
         and iguanians with snakes and anguimorphs <a name="d6347e4877"></a><a name="d6347e4879"></a><a name="d6347e4881"></a><a name="d6347e4883"></a><a name="d6347e4885"></a>[<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>-<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B23'); return false;" href="#B23">23</a>]. These results have now been corroborated with up to 22 genes (15794 bp) for 45 taxa
         <a name="d6347e4889"></a>[<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>], 25 genes (19020 bp) for 64 taxa <a name="d6347e4893"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>], and 44 genes (33717 bp) for 161 taxa <a name="d6347e4897"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]. We now support this basic topology with 4161 species sampled for up to 12 genes
         each (up to 12896 bp).
      </p>
      <p style="line-height:160%">Nevertheless, despite the overall strong support for most of the tree (i.e. 70% of
         all nodes have SHL &gt; 85), certain clades remain poorly supported (e.g. relationships
         among many pleurodont iguanian families; Figures&nbsp;<a name="d6347e4902"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>, <a name="d6347e4905"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F17','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F17">17</a>, <a name="d6347e4908"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F18','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F18">18</a>, <a name="d6347e4911"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F19','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F19">19</a>). A potential criticism of the supermatrix approach used here is that this weak support
         may occur due to missing data. However, previous studies of 8 datasets have shown
         explicitly that there is typically little relationship between branch support for
         terminal taxa and the amount of missing data <a name="d6347e4915"></a>[<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]. Instead, these patterns of weak support are more likely to reflect short underlying
         branch lengths <a name="d6347e4920"></a><a name="d6347e4922"></a><a name="d6347e4924"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>], and may be difficult to resolve even with more complete taxonomic and genomic sampling.
         Indeed, as noted above, many of the weakly supported nodes in our phylogeny are also
         weakly supported in analyses with little missing data (&lt;20%) and large numbers of
         genes (e.g. 44 genes as in <a name="d6347e4928"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]), such as the relationships of many pleurodont lizard families and colubroid snake
         families and subfamilies.
      </p>
      <p style="line-height:160%">We acknowledge that the differences between our results and previous studies (noted
         above) do not necessarily mean that our results are right and those of previous studies
         are wrong. In some cases, we provide strong support for novel relationships when previous,
         conflicting studies showed only weak support (as in scincids, see above). In other
         cases, our results disagree with other studies for clades that were strongly supported
         (e.g. placement of xenodermatids). In the best-case scenario, these conflicts may
         be resolved because our results are correct, possibly reflecting the beneficial effects
         of adding taxa and the associated subdivision of long branches <a name="d6347e4934"></a><a name="d6347e4936"></a><a name="d6347e4938"></a><a name="d6347e4940"></a><a name="d6347e4942"></a><a name="d6347e4944"></a><a name="d6347e4946"></a>[<a onclick="LoadInParent('#B25'); return false;" href="#B25">25</a></a>,<a onclick="LoadInParent('#B26'); return false;" href="#B26">26</a>,<a onclick="LoadInParent('#B28'); return false;" href="#B28">28</a>,<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>,<a onclick="LoadInParent('#B192'); return false;" href="#B192">192</a>-<a onclick="LoadInParent('#B194'); return false;" href="#B194">194</a>]. Furthermore, in many cases, we are including more genes than used in previous studies
         of particular clades, increasing sampling of characters and loci. This should generally
         reduce spurious results caused by sampling few characters and by incongruence between
         gene and species trees.
      </p>
      <p style="line-height:160%">However, other explanations for incongruence between our results and previous studies
         are also possible. Adding taxa can potentially lead to incorrect results in some cases
         (e.g. <a name="d6347e4952"></a>[<a onclick="LoadInParent('#B195'); return false;" href="#B195">195</a>]), such as when a long terminal branch is added that further subdivides a short internal
         branch. In other cases, conflicts with our results might reflect the impact of our
         sampling fewer nuclear genes and a correspondingly increased influence of mitochondrial
         data. Mitochondrial genes have relatively fast evolutionary rates (potentially exacerbating
         the impacts of long branches), and their phylogenetic resolution for a particular
         node may also reflect introgression or incomplete lineage sorting rather than the
         species phylogeny (review in <a name="d6347e4956"></a>[<a onclick="LoadInParent('#B196'); return false;" href="#B196">196</a>]). Many taxa in the matrix are represented only by mitochondrial data, and highly
         variable mitochondrial genes might also overcome the influence of less variable nuclear
         genes in combined analyses (although this scenario does not seem to be common <a name="d6347e4960"></a>[<a onclick="LoadInParent('#B196'); return false;" href="#B196">196</a>]). Such cases might explain some strongly supported conflicts between our results
         and those based on multiple nuclear loci. Another possibility is that some cases may
         represent failure to find the optimal tree (although we assume that these cases will
         likely show only weak support). We acknowledge that there are many reasons why our
         results may differ from previous studies, and the ultimate test of these novel findings
         will be corroboration in future studies that include more taxa and characters.
      </p>
      <p style="line-height:160%">This analysis also corroborates several recent studies suggesting that the supermatrix
         approach is a powerful strategy for large-scale phylogenetic inference <a name="d6347e4967"></a><a name="d6347e4969"></a><a name="d6347e4971"></a><a name="d6347e4973"></a><a name="d6347e4975"></a><a name="d6347e4977"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B72'); return false;" href="#B72">72</a></a>,<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>,<a onclick="LoadInParent('#B75'); return false;" href="#B75">75</a></a>,<a onclick="LoadInParent('#B76'); return false;" href="#B76">76</a>,<a onclick="LoadInParent('#B197'); return false;" href="#B197">197</a>]. For example, even though each species had 81% missing data on average, we found
         that most species were placed in the families and genera expected based on previous
         taxonomy, often with very strong support. Furthermore, we found that incompleteness
         of terminal taxa is not related to branch lengths (at least not terminal branch lengths),
         suggesting that missing data are not significantly biasing branch-length estimates
         (see also <a name="d6347e4981"></a><a name="d6347e4983"></a><a name="d6347e4985"></a>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a>,<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a></a>,<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>]). Also, the ML models we used have been shown to be robust to missing data in large,
         sparse supermatrices <a name="d6347e4989"></a>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a>].
      </p>
      <p style="line-height:160%">Even though we did find some subfamilies and genera to be non-monophyletic, similar
         relationships were often found in previous studies based on data matrices with only
         limited missing data (e.g. non-monophyly of boid snake subfamilies <a name="d6347e4995"></a>[<a onclick="LoadInParent('#B47'); return false;" href="#B47">47</a>], lacertid and scincid lizard genera <a name="d6347e4999"></a><a name="d6347e5001"></a>[<a onclick="LoadInParent('#B67'); return false;" href="#B67">67</a>,<a onclick="LoadInParent('#B111'); return false;" href="#B111">111</a>], and scolecophidian, dipsadine, and natricine snake genera <a name="d6347e5005"></a><a name="d6347e5007"></a><a name="d6347e5009"></a>[<a onclick="LoadInParent('#B38'); return false;" href="#B38">38</a>,<a onclick="LoadInParent('#B43'); return false;" href="#B43">43</a>,<a onclick="LoadInParent('#B186'); return false;" href="#B186">186</a>]). We suggest that further resolution of the squamate tree will be greatly facilitated
         if researchers deliberately sample mitochondrial genes and nuclear genes that include
         the set of genes used here and in recent phylogenomic studies (e.g. <a name="d6347e5013"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]), to increase overlap between genes and taxa, and decrease missing data.
      </p>
      <p style="line-height:160%">With over 5000 species remaining to be included and only 12 genes sampled, our study
         is far from the last word on squamate phylogeny. We note that new data can easily
         be added to this matrix, in terms of both new taxa and new genes. Increased sampling
         of other nuclear genes is likely to be advantageous as well. Next-generation sequencing
         strategies and phylogenomic methods should help resolve difficult nodes <a name="d6347e5019"></a><a name="d6347e5021"></a><a name="d6347e5023"></a><a name="d6347e5025"></a><a name="d6347e5027"></a><a name="d6347e5029"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B198'); return false;" href="#B198">198</a>-<a onclick="LoadInParent('#B200'); return false;" href="#B200">200</a>], as should application of species-tree methods <a name="d6347e5033"></a><a name="d6347e5035"></a>[<a onclick="LoadInParent('#B201'); return false;" href="#B201">201</a></a>,<a onclick="LoadInParent('#B202'); return false;" href="#B202">202</a>]. Species-tree analyses of 44 nuclear loci support many of these same clades across
         squamates <a name="d6347e5039"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>], and data from &gt;400 nuclear loci reinforces many of the relationships found here
         among the colubroid snake subfamilies (Pyron et al., in prep.). In addition, it would
         be useful to incorporate fossil taxa in future studies <a name="d6347e5043"></a><a name="d6347e5045"></a><a name="d6347e5047"></a><a name="d6347e5049"></a>[<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a>,<a onclick="LoadInParent('#B22'); return false;" href="#B22">22</a>,<a onclick="LoadInParent('#B86'); return false;" href="#B86">86</a>], utilizing the large morphological datasets that are now available <a name="d6347e5053"></a><a name="d6347e5055"></a>[<a onclick="LoadInParent('#B14'); return false;" href="#B14">14</a></a>,<a onclick="LoadInParent('#B15'); return false;" href="#B15">15</a>]. Despite these areas for future studies, the present tree provides a framework for
         researchers analyzing patterns of squamate evolution at both lower and higher taxonomic
         levels (e.g. <a name="d6347e5060"></a><a name="d6347e5062"></a><a name="d6347e5064"></a><a name="d6347e5066"></a>[<a onclick="LoadInParent('#B10'); return false;" href="#B10">10</a></a>,<a onclick="LoadInParent('#B11'); return false;" href="#B11">11</a>,<a onclick="LoadInParent('#B203'); return false;" href="#B203">203</a></a>,<a onclick="LoadInParent('#B204'); return false;" href="#B204">204</a>]), and for building a more complete picture of squamate phylogeny.
      </p>
   </div>
</section>
<section><a name="sec4"></a><h3>Conclusions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">In this study, we provide a phylogenetic estimate for 4161 squamate species, based
         on a supermatrix approach. Our results provide important confirmation for previous
         studies based on more limited taxon sampling, and reveal new relationships at the
         level of families, genera, and species. We also provide a phylogenetic framework for
         future comparative studies, with a large-scale tree including a common set of estimated
         branch lengths. Finally, we provide a revised classification for squamates based on
         this tree, including changes in the higher-level taxonomy of gymnophthalmid and scincid
         lizards and boid, colubrid, and lamprophiid snakes.
      </p>
   </div>
</section>
<section><a name="sec5"></a><h3>Methods</h3>
   <div class="collapsible-content">
      <h4>Initial classification</h4>
      <p style="line-height:160%">Our initial squamate classification is based on the June 2009 version of the Reptile
         Database <a name="d6347e5086"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>] (<a href="http://www.reptile-database.org/">http://www.reptile-database.org/</a> <a href="http://www.webcitation.org/query.php?url=http://www.reptile-database.org/&amp;refdoi=10.1186/1471-2148-13-93" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a>), accessed in September of 2009 when this research was begun. Minor modifications
         to this scheme were made, primarily to update changes in colubroid snake taxonomy
         <a name="d6347e5093"></a><a name="d6347e5095"></a><a name="d6347e5097"></a><a name="d6347e5099"></a><a name="d6347e5101"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>-<a onclick="LoadInParent('#B44'); return false;" href="#B44">44</a>,<a onclick="LoadInParent('#B205'); return false;" href="#B205">205</a>]. This initial taxonomic database consists of 8650 species (169 amphisbaenians, 5270
         lizards, 3209 snakes, and 2 tuataras), against which the classification of species
         in the molecular sequence database was fixed. While modifications and updates (i.e.
         new species, revisions) have been made to squamate taxonomy subsequently, these are
         minor and should have no impact on our phylogenetic results. This database represents
         ~92% of the current estimated diversity of squamates (~9400 species as of December
         2012).
      </p>
      <p style="line-height:160%">Throughout the paper, we refer to the updated version of squamate taxonomy from the
         December 2012 update of the Reptile Database <a name="d6347e5107"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], incorporating major, well-accepted changes from recent studies (summarized in <a name="d6347e5111"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]). However, for large, taxonomic groups that have recently been broken up for reasons
         other than resolving paraphyly or matters of priority (e.g. in dactyloid and scincid
         lizards; see Results), we generally retain the older, more inclusive name in the interest
         of clarity, while providing references to the recent revision. We attempt to alter
         existing classifications as little as possible (see also <a name="d6347e5115"></a>[<a onclick="LoadInParent('#B113'); return false;" href="#B113">113</a>]). Therefore, we generally only make changes when there is strong support for non-monophyly
         of currently recognized taxa and our proposed changes yield strongly supported monophyletic
         groups. Similarly, we only erect new taxa if they are strongly supported. Finally,
         although numerous genera are identified as being non-monophyletic in our tree, we
         refrain from changing genus-level taxonomy, given that our taxon sampling within many
         genera is limited.
      </p>
      <h4>Molecular data</h4>
      <p style="line-height:160%">Preliminary literature searches were conducted to identify candidate genes for which
         a substantial number of squamate species were sequenced and available on GenBank (with
         the sampled species spread across multiple families), and which were demonstrably
         useful in previous phylogenetic studies of squamates (see Introduction for references).
         Twelve genes were identified as meeting these criteria: seven nuclear genes (brain-derived
         neurotrophic factor [BDNF], oocyte maturation factor [c-mos], neurotrophin-3 [NT3],
         phosducin [PDC], G protein-coupled receptor 35 [R35], and recombination-activating
         genes 1 [RAG-1] and 2 [RAG-2]); and five mitochondrial genes (12S/16S long and short
         subunit RNAs, cytochrome <em>b</em> [cyt-<em>b</em>], and nicotinamide adenine dehydrogenase subunits 2 [ND2] and 4 [ND4]). This sampling
         of genes does not include all available markers. For example, we omitted several nuclear
         and mitochondrial genes because they were available only for a limited subset of taxa.
         We also excluded tRNAs associated with the protein-coding sequences, given their short
         lengths and difficulty in alignment across the large time scales considered here.
      </p>
      <p style="line-height:160%">To ensure maximal taxonomic coverage from the available data, searches were conducted
         on GenBank by family (stopping in October 2012), and the longest sequence for every
         species was gathered. Sequences totaling less than 250 bp for any species were not
         included. Only species in the taxonomic database were included in the sequence matrix,
         which resulted in the exclusion of numerous named taxa of ambiguous status, a few
         taxa described very recently, and many sequences labeled 'sp.' Some recently described
         phylogeographic lineages were also omitted. Species and GenBank accession numbers
         are available in Additional file <a name="d6347e5132"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/suppl/S2','',630,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/suppl/S2">2</a>: Table S1.
      </p>
      <p style="line-height:160%">With respect to the December 2012 update of the Reptile Database <a name="d6347e5138"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], we sampled 52 of 183 amphisbaenian species (28%) from 11 of 19 (58%) genera; 2847
         of 5799 lizard species (49%) from 448 of 499 genera (90%); and 1262 of 3434 snake
         species (39%) in 396 of 500 genera (80%). This yielded a total of 4161 species in
         855 genera in the final matrix, 44% of the 9416 known, extant squamate species in
         84% of 1018 genera <a name="d6347e5142"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. The species-level classification of squamates is in constant flux, and the numbers
         of species and genera changed even as this paper was under review. The extant species
         of tuatara (<em>Sphenodon punctatus</em>) was included as a non-squamate outgroup taxon (see below). We acknowledge that our
         sampling of outgroup taxa is not extensive. However, placement of <em>Sphenodon</em> as the sister group to squamates is well-established by molecular analyses with extensive
         taxon sampling (e.g. <a name="d6347e5152"></a><a name="d6347e5154"></a><a name="d6347e5156"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a>,<a onclick="LoadInParent('#B128'); return false;" href="#B128">128</a>,<a onclick="LoadInParent('#B206'); return false;" href="#B206">206</a>]) and morphological data (e.g. <a name="d6347e5161"></a>[<a onclick="LoadInParent('#B13'); return false;" href="#B13">13</a>]).
      </p>
      <p style="line-height:160%">Alignment for protein-coding sequences was relatively straightforward. We converted
         them to amino acids, and then used the translation alignment algorithm in the program
         Geneious v4.8.4 (GeneMatters Corp.), with the default cost matrix (Blosum62) and gap
         penalties (open=12, extension=3). Alignments were relatively unambiguous after being
         trimmed for quality and maximum coverage (i.e. ambiguous end regions were removed,
         and most sequences began and ended at the same point).
      </p>
      <p style="line-height:160%">For the ribosomal RNA sequences (12S and 16S sequences), alignment was more challenging.
         Preliminary global alignments using the algorithms MUSCLE <a name="d6347e5169"></a>[<a onclick="LoadInParent('#B207'); return false;" href="#B207">207</a>] and CLUSTAL <a name="d6347e5173"></a>[<a onclick="LoadInParent('#B208'); return false;" href="#B208">208</a>] under a variety of gap-cost parameters yielded low-quality results (i.e. alignments
         with large numbers of gaps and little overlap of potentially homologous characters).
         We subsequently employed a two-step strategy for these data. We first grouped sequences
         by higher taxa (i.e. Amphisbaenia, Anguimorpha, Gekkota, Iguania, Scincomorpha, and
         Serpentes, though these are not all monophyletic as previously defined), for which
         alignments were relatively straightforward under the default MUSCLE parameters.
      </p>
      <p style="line-height:160%">These were then combined using the profile alignment feature of MUSCLE, and the global
         alignment was subsequently updated using the "refine alignment" option. Minor adjustments
         were then made by eye, and ambiguously aligned end-regions were trimmed for maximum
         coverage and quality. We did not include partitions for stems and loops for the ribosomal
         sequences, although this has been shown to improve model fit in previous squamate
         studies (e.g. <a name="d6347e5179"></a>[<a onclick="LoadInParent('#B82'); return false;" href="#B82">82</a>]). Although it is possible to assign individual nucleotide positions to these partitions,
         this would have been challenging given the large number of sequences, and the potential
         for stems and loops to shift across the many species and large time scales involved.
      </p>
      <p style="line-height:160%">Each species was represented by a single terminal taxon in the matrix. In many cases,
         sequences from multiple individuals of the same species were combined, to allow us
         to combine data from different genes for the same species. We acknowledge the possibility
         that in some cases this approach may cause us to combine genes from different species
         in the same terminal taxon (e.g. due to changing taxonomy or incorrect identifications).
         Additionally, many sequences are not from vouchered specimens, and it is possible
         that misidentified species are present on GenBank and in our matrix. However, most
         of our data came from lower-level phylogenetic studies, in which the identification
         of species by previous authors should be highly accurate. In addition, any such mistakes
         should be among closely related species, and lead to minimal phylogenetic distortion,
         as the grossest errors are easily identified.
      </p>
      <p style="line-height:160%">Some species were removed after preliminary analyses, due either to obvious sequencing
         errors (e.g. high BLAST homology with unrelated families or non-squamates, excessive
         ambiguities) or a lack of overlap in genes sampled with other members of the same
         genus (leading to seemingly artificial paraphyly). We also excluded species with identical
         sequences between taxa across all genes, arbitrarily choosing the first taxon in alphabetical
         order to remain in the matrix. Additionally, we also removed a few apparent "rogue
         taxa" <a name="d6347e5187"></a><a name="d6347e5189"></a>[<a onclick="LoadInParent('#B75'); return false;" href="#B75">75</a>,<a onclick="LoadInParent('#B77'); return false;" href="#B77">77</a>]. These were identified by their poor support and suspect placement (e.g. in a clearly
         incorrect family), and were typically represented in the matrix by short fragments
         of single genes (e.g. an ND4 fragment from the enigmatic colubroid snake <em>Oreocalamus hanitchsi</em>).
      </p>
      <p style="line-height:160%">The final combined matrix contained sequence data for: 2335 species for 12S (including
         56% of all 4162 taxa, 1395 bp), 2377 for 16S (57%, 1970 bp), 730 for BDNF (18%, 714
         bp), 1671 for c-mos (40%, 903 bp), 1985 for cyt-b (48%, 1000 bp), 437 for NT3 (10%,
         675 bp), 1860 for ND2 (45%, 960 bp), 1556 for ND4 (37%, 696 bp), 393 for PDC (9%,
         395 bp), 401 for R35 (10%, 768 bp), 1379 for RAG-1 (33%, 2700 bp), and 471 for RAG2
         (11%, 720 bp). The total alignment consists of 12896 bp for 4162 taxa (4161 squamates
         and 1 outgroup). The mean length is 2497 bp of sequence data present per species from
         3.75 genes (19% of the total matrix length of 12896 bp, or 81% missing data), and
         ranges from 270â€“11153 bp (2â€“86% complete). The matrix and phylogeny (see below) are
         available in DataDryad repository 10.5061/dryad.82h0m.
      </p>
      <p style="line-height:160%">Clearly, many taxa had large amounts of missing data (some &gt;95%), and on average each
         species had 81% missing cells. However, several lines of evidence suggest that these
         missing data are not generally problematic. First, a large body of empirical and theoretical
         studies has shown that highly incomplete taxa can be accurately placed in model-based
         phylogenetic analyses (and with high levels of branch support), especially if a large
         number of characters have been sampled (recent reviews in <a name="d6347e5200"></a><a name="d6347e5202"></a>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a></a>,<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]). Second, several recent empirical studies have shown that the supermatrix approach
         (with extensive missing data in some taxa) yields generally well-supported large-scale
         trees that are generally congruent with previous taxonomies and phylogenetic estimates
         (e.g. <a name="d6347e5206"></a><a name="d6347e5208"></a><a name="d6347e5210"></a><a name="d6347e5212"></a><a name="d6347e5214"></a><a name="d6347e5216"></a><a name="d6347e5218"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B48'); return false;" href="#B48">48</a>,<a onclick="LoadInParent('#B72'); return false;" href="#B72">72</a></a>,<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>,<a onclick="LoadInParent('#B75'); return false;" href="#B75">75</a></a>,<a onclick="LoadInParent('#B76'); return false;" href="#B76">76</a>,<a onclick="LoadInParent('#B197'); return false;" href="#B197">197</a>]). Third, recent studies have also shown that there is generally little relationship
         between the amount of missing data in individual taxa and the support for their placement
         on the tree <a name="d6347e5222"></a><a name="d6347e5224"></a><a name="d6347e5226"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B73'); return false;" href="#B73">73</a>,<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]. Finally, we note that some highly incomplete taxa were unstable in their placement
         (â€œrogue taxa;" [75]), but these were removed prior to the analysis of the final matrix
         (see above).
      </p>
      <p style="line-height:160%">Our sampling design should be especially robust to the impacts of missing data for
         several reasons. Most importantly, most terminal taxon (species) had substantial data
         present (mean of 2497 bp per species) regardless of the number of missing data cells.
         Simulations (see reviews in <a name="d6347e5233"></a><a name="d6347e5235"></a>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a></a>,<a onclick="LoadInParent('#B85'); return false;" href="#B85">85</a>]) suggest that the amount of data present is a key parameter in determining the accuracy
         with which incomplete taxa are placed in phylogenies, not the amount of data absent.
         Additionally, several genes (e.g. 12S/16S, cyt-<em>b</em>, and c-mos) were shared by many (&gt;40%) of taxa. Thus, there was typically extensive
         overlap among the genes present for each taxon (as also indicated by the mean bp per
         species being much greater than the length of most genes). Limited overlap in gene
         sampling among taxa could be highly problematic, irrespective of the amount of missing
         data <em>per se</em>, but this does not appear to be a problem in our dataset. Finally, several nuclear
         genes (e.g. BDNF, c-mos, R35, and RAG-1) were congruently sampled in previous studies
         to represent most (&gt;80%) squamate families and subfamilies (e.g. <a name="d6347e5245"></a>[<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>]), providing a scaffold of well-sampled taxa spanning all major clades, as recommended
         by recent authors <a name="d6347e5249"></a>[<a onclick="LoadInParent('#B84'); return false;" href="#B84">84</a>].
      </p>
      <h4>Phylogenetic analyses</h4>
      <p style="line-height:160%">We performed phylogenetic analyses of the 12-gene concatenated matrix using Maximum
         Likelihood (ML). We assessed node support using the non-parametric Shimodaira-Hasegawa-Like
         (SHL) implementation of the approximate likelihood-ratio test (aLRT; <a name="d6347e5259"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>]). This involved a two-stage strategy. We first performed initial ML tree- inference
         using the program RAxML-Light v1.0.7 <a name="d6347e5263"></a>[<a onclick="LoadInParent('#B209'); return false;" href="#B209">209</a>], a modification of the original RAxML algorithm <a name="d6347e5267"></a>[<a onclick="LoadInParent('#B210'); return false;" href="#B210">210</a>]. This program uses the GTRCAT strategy for all genes and partitions, a high-speed
         approximation of the GTR+Î“ model (general time-reversible with gamma-distribution
         of rate heterogeneity among sites). The GTR model is the only substitution model implemented
         in RAxML <a name="d6347e5271"></a>[<a onclick="LoadInParent('#B210'); return false;" href="#B210">210</a>], and all other substitution models are simply special cases of the GTR model <a name="d6347e5275"></a>[<a onclick="LoadInParent('#B211'); return false;" href="#B211">211</a>]. Previous analyses suggest that GTR is generally the best-fitting model for these
         genes and that they should be partitioned by gene and codon position <a name="d6347e5280"></a><a name="d6347e5282"></a><a name="d6347e5284"></a><a name="d6347e5286"></a><a name="d6347e5288"></a><a name="d6347e5290"></a>[<a onclick="LoadInParent('#B16'); return false;" href="#B16">16</a></a>,<a onclick="LoadInParent('#B17'); return false;" href="#B17">17</a>,<a onclick="LoadInParent('#B19'); return false;" href="#B19">19</a></a>,<a onclick="LoadInParent('#B20'); return false;" href="#B20">20</a>,<a onclick="LoadInParent('#B36'); return false;" href="#B36">36</a>,<a onclick="LoadInParent('#B81'); return false;" href="#B81">81</a>].
      </p>
      <p style="line-height:160%">To generate an initial ML estimate for final optimization and support estimation,
         we performed 11 ML searches from 11 parsimony starting trees generated under the default
         parsimony model in RAxMLv7.2.8. This number is likely to be sufficient when datasets
         contain many characters that have strong phylogenetic signal (A. Stamatakis, <em>pers. comm.</em>). Additionally, the dataset was analyzed with these settings (GTRCAT search from
         a randomized parsimony starting tree) numerous times (&gt;20) as the final matrix was
         assembled and tested, representing hundreds of independent searches from random starting
         points. All of the estimated trees from these various analyses showed high overall
         congruence with the final topology. The concordance between the preliminary and final
         results suggests that the tree was not strongly impacted by searches stuck on local
         optima, and that it should be a good approximation of the ML tree.
      </p>
      <p style="line-height:160%">We then performed a final topology optimization and assessed support. We passed our
         best ML estimate of the phylogeny (based on GTRCAT) from RAxML-Light to RAxMLv7.2.8,
         which does an additional search (using the GTRGAMMA model) to produce a nearest-neighbor
         interchange (NNI)-optimized estimate of the ML tree. This optimization is needed to
         calculate the SHL version of the aLRT for estimating support values <a name="d6347e5301"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>]. The SHL-aLRT strategy approximates a test of the null hypothesis that the branch
         length subtending each node equals 0 (i.e. that the node can be resolved, rather than
         estimated as a polytomy) with a test of the more general null hypothesis that "the
         branch is incorrect" relative to the four next suboptimal arrangements of that node
         relative to the NNI-optimal arrangement <a name="d6347e5305"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>]. Based on initial analyses, generating sufficient ML bootstrap replicates for a tree
         of this size proved computationally intractable, so we rely on SHL values alone to
         assess support.
      </p>
      <p style="line-height:160%">The SHL approach has at least two major advantages over non-parametric bootstrapping
         for large ML trees: (i) values are apparently robust to many potential model violations
         and have the same properties as bootstrap proportions for all but the shortest branches
         <a name="d6347e5311"></a><a name="d6347e5313"></a><a name="d6347e5315"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>,<a onclick="LoadInParent('#B212'); return false;" href="#B212">212</a>], and (ii) values for short branches may be more accurate than bootstrap proportions,
         as support is evaluated based on whole-alignment likelihoods, rather than the frequency
         of re-sampled characters <a name="d6347e5319"></a><a name="d6347e5321"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>,<a onclick="LoadInParent('#B213'); return false;" href="#B213">213</a>]. Additionally, the SHL approach is orders of magnitude faster than traditional bootstrapping
         <a name="d6347e5325"></a><a name="d6347e5327"></a><a name="d6347e5329"></a>[<a onclick="LoadInParent('#B94'); return false;" href="#B94">94</a>,<a onclick="LoadInParent('#B212'); return false;" href="#B212">212</a></a>,<a onclick="LoadInParent('#B213'); return false;" href="#B213">213</a>], and it appears to be similarly robust to matrices with extensive missing data <a name="d6347e5333"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>]. As in previous studies, we take a conservative view, considering SHL values of 85
         or greater (i.e. a 15% chance that a branch is "incorrect") as strong support <a name="d6347e5337"></a><a name="d6347e5339"></a><a name="d6347e5341"></a>[<a onclick="LoadInParent('#B41'); return false;" href="#B41">41</a>,<a onclick="LoadInParent('#B212'); return false;" href="#B212">212</a></a>,<a onclick="LoadInParent('#B213'); return false;" href="#B213">213</a>].
      </p>
      <p style="line-height:160%">These analyses were performed on a 360-core SGI ICE supercomputing system ("ANDY")
         at the High Performance Computing Center at the City University of New York (CUNY).
         The final analysis was completed in 8.8 days of computer time using 188 nodes of the
         CUNY supercomputing cluster.
      </p>
      <p style="line-height:160%">Finally, we assessed the potential impact of missing data on our branch-length estimates.
         We performed linear regression (in R) of the proportional completeness of each terminal
         taxa (non-missing data in bp / maximum amount of non-missing data, 12896 bp) against
         the length of its terminal branch. This test addresses whether incomplete taxa have
         branch-length estimates that are consistently biased in one direction (shorter vs.
         longer) relative to more complete terminals. However, it does not directly test whether
         branch length estimates are correct or not, nor how branch length estimates are impacted
         by replacing non-missing data with missing data (see <a name="d6347e5349"></a>[<a onclick="LoadInParent('#B87'); return false;" href="#B87">87</a>] for results suggesting that such replacements have little effect in real data sets).
      </p>
   </div>
</section>
<section><a name="sec6"></a><h3>Appendix I</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">Note that we only provide here an account for the one subfamily newly erected in this
         study. We do not provide accounts for subfamilies with changes in content (Boinae,
         Erycinae, Dipsadinae, Pseudaspidinae, Scincinae, Ungaliophiinae), that have been resurrrected
         (Sanziniinae), or that represent elevation of lower-ranked taxa (tribe Bachiini here
         recognized as Bachiinae).
      </p>
      <h4>Candoiinae <em>subfam. nov.</em> (family Boidae)
      </h4>
      <p style="line-height:160%"><em>Type:</em> genus and species <em>Candoia carinata</em><a name="d6347e5372"></a>[<a onclick="LoadInParent('#B214'); return false;" href="#B214">214</a>].
      </p>
      <p style="line-height:160%"><em>Content:</em> one genus, 4 species; <em>C. aspera</em>, <em>C. bibroni</em>, <em>C. carinata</em>, <em>C. paulsoni.</em></p>
      <p style="line-height:160%"><em>Definition:</em> this subfamily consists of the most recent common ancestor of the extant species
         of <em>Candoia</em>, and all its descendants. These species are morphologically distinguished in part
         from other boid snakes by a flattened rostrum leading to an angular snout <a name="d6347e5398"></a>[<a onclick="LoadInParent('#B215'); return false;" href="#B215">215</a>] and a wide premaxillary floor <a name="d6347e5402"></a>[<a onclick="LoadInParent('#B167'); return false;" href="#B167">167</a>].
      </p>
      <p style="line-height:160%"><em>Distribution:</em> these snakes are primarily restricted to the South Pacific islands of New Guinea
         and Melanesia, and the eastern Indonesian archipelago <a name="d6347e5410"></a>[<a onclick="LoadInParent('#B150'); return false;" href="#B150">150</a>].
      </p>
      <p style="line-height:160%"><em>Remarks:</em> the three species from this subfamily that are sampled in our tree are strongly supported
         as monophyletic (SHL = 100), and are well supported (SHL = 87) as the sister taxon
         to a moderately supported clade consisting of Erycinae + Boinae (SHL = 83).
      </p>
      <h4>Proposed Generic Composition of Higher Taxa</h4>
      <p style="line-height:160%">Below, we list the familial and subfamilial assignment of all squamate genera from
         the December, 2012 update of the Reptile Database <a name="d6347e5424"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>], updated to reflect some recent changes and the proposed subfamily level changes
         listed above. As this classification includes numerous taxa not sampled in our tree,
         we deal with them conservatively. For traditionally recognized families and subfamilies
         that we found to be monophyletic, we include all taxa traditionally assigned to them.
         Taxa are denoted <em>incertae sedis</em> if they are of ambiguous familial or subfamilial assignment due to uncertain placement
         in our tree, or due to absence from our tree and lack of assignment by previous authors.
         This classification includes 67 families and 56 subfamilies, and accounts for &gt;9400
         squamate species in 1018 genera <a name="d6347e5431"></a>[<a onclick="LoadInParent('#B1'); return false;" href="#B1">1</a>]. Higher taxa are listed (more-or-less) phylogenetically (starting closest to the
         root; Figure&nbsp;<a name="d6347e5434"></a><a onclick="popup('http://www.biomedcentral.com/1471-2148/13/93/figure/F1','',800,470); return false;" href="http://www.biomedcentral.com/1471-2148/13/93/figure/F1">1</a>), families are listed alphabetically within higher taxa, and subfamilies and genera
         are listed alphabetically within families.
      </p>
   </div>
</section>
<section><a name="sec7"></a><h3>Squamata</h3>
   <div class="collapsible-content">
      <h4>Dibamidae (<em>Anelytropsis, Dibamus</em>)
      </h4>
      <h4>Gekkota</h4>
      <p style="line-height:160%"><strong>Carphodactylidae</strong> (<em>Carphodactylus, Nephrurus, Orraya, Phyllurus, Saltuarius, Underwoodisaurus, Uvidicolus</em>); <strong>Diplodactylidae</strong> (<em>Amalosia, Bavayia, Correlophus, Crenadactylus, Dactylocnemis, Dierogekko, Diplodactylus,
            Eurydactylodes, Hesperoedura, Hoplodactylus, Lucasium, Mniarogekko, Mokopirirakau,
            Naultinus, Nebulifera, Oedodera, Oedura, Paniegekko, Pseudothecadactylus, Rhacodactylus,
            Rhynchoedura, Strophurus, Toropuku, Tukutuku, Woodworthia</em>); <strong>Eublepharidae</strong> (<em>Aeluroscalabotes, Coleonyx, Eublepharis, Goniurosaurus, Hemitheconyx, Holodactylus</em>); <strong>Gekkonidae</strong> (<em>Afroedura, Afrogecko, Agamura, Ailuronyx, Alsophylax, Asiocolotes, Blaesodactylus,
            Bunopus, Calodactylodes, Chondrodactylus, Christinus, Cnemaspis, Colopus, Crossobamon,
            Cryptactites, Cyrtodactylus, Cyrtopodion, Dixonius, Ebenavia, Elasmodactylus, Geckolepis,
            Gehyra, Gekko, Goggia, Hemidactylus, Hemiphyllodactylus, Heteronotia, Homopholis,
            Lepidodactylus, Luperosaurus, Lygodactylus, Matoatoa, Mediodactylus, Nactus, Narudasia,
            Pachydactylus, Paragehyra, Paroedura, Perochirus, Phelsuma, Pseudoceramodactylus,
            Pseudogekko, Ptenopus, Ptychozoon, Rhinogecko, Rhoptropella, Rhoptropus, Stenodactylus,
            Tropiocolotes, Urocotyledon, Uroplatus</em>); <strong>Phyllodactylidae</strong> (<em>Asaccus, Gymnodactylus, Haemodracon, Homonota, Phyllodactylus, Phyllopezus, Ptyodactylus,
            Tarentola, Thecadactylus</em>); <strong>Pygopodidae</strong> (<em>Aprasia, Delma, Lialis, Ophidiocephalus, Paradelma, Pletholax, Pygopus</em>); <strong>Sphaerodactylidae</strong> (<em>Aristelliger, Chatogekko, Coleodactylus, Euleptes, Gonatodes, Lepidoblepharis, Pristurus,
            Pseudogonatodes, Quedenfeldtia, Saurodactylus, Sphaerodactylus, Teratoscincus</em>)
      </p>
      <h4>Scincoidea</h4>
      <p style="line-height:160%"><strong>Cordylidae,</strong> Cordylinae (<em>Chamaesaura, Cordylus, Hemicordylus, Karusasaurus, Namazonurus, Ninurta, Ouroborus,
            Pseudocordylus, Smaug</em>), Platysaurinae (<em>Platysaurus</em>); <strong>Gerrhosauridae</strong>, Gerrhosaurinae (<em>Cordylosaurus</em>, <em>Gerrhosaurus</em>, <em>Tetradactylus</em>), Zonosaurinae (<em>Tracheloptychus</em>, <em>Zonosaurus</em>); <strong>Scincidae</strong>, Acontiinae (<em>Acontias</em>, <em>Typhlosaurus</em>), Lygosominae (<em>Ablepharus, Afroablepharus, Anomalopus, Asymblepharus, Ateuchosaurus, Bartleia, Bassiana,
            Bellatorias, Caledoniscincus, Calyptotis, Carlia, Cautula, Celatiscincus, Chioninia,
            Coeranoscincus, Coggeria, Cophoscincopus, Corucia, Cryptoblepharus, Ctenotus, Cyclodomorphus,
            Dasia, Egernia, Emoia, Eremiascincus, Eroticoscincus, Eugongylus, Eulamprus, Eumecia,
            Eutropis, Fojia, Geomyersia, Geoscincus, Glaphyromorphus, Gnypetoscincus, Graciliscincus,
            Haackgreerius, Hemiergis, Hemisphaeriodon, Insulasaurus, Isopachys, Kaestlea, Kanakysaurus,
            Lacertaspis, Lacertoides, Lamprolepis, Lampropholis, Lankascincus, Larutia, Leiolopisma,
            Lepidothyris, Leptoseps, Leptosiaphos, Lerista, Liburnascincus, Liopholis, Lioscincus,
            Lipinia, Lissolepis, Lobulia, Lygisaurus, Lygosoma, Mabuya, Marmorosphax, Menetia,
            Mochlus, Morethia, Nangura, Nannoscincus, Niveoscincus, Notoscincus, Oligosoma, Ophioscincus,
            Otosaurus, Panaspis, Papuascincus, Parvoscincus, Phoboscincus, Pinoyscincus, Prasinohaema,
            Proablepharus, Pseudemoia, Ristella, Saiphos, Saproscincus, Scincella, Sigaloseps,
            Simiscincus, Sphenomorphus, Tachygyia, Tiliqua, Trachylepis, Tribolonotus, Tropidophorus,
            Tropidoscincus, Tytthoscincus, Vietnascincus</em>), Scincinae (<em>Amphiglossus, Androngo, Barkudia, Brachymeles, Chabanaudia, Chalcides, Chalcidoseps,
            Eumeces, Eurylepis, Feylinia, Gongylomorphus, Hakaria, Janetaescincus, Jarujinia,
            Madascincus, Melanoseps, Mesoscincus, Nessia, Ophiomorus, Pamelaescincus, Paracontias,
            Plestiodon, Proscelotes, Pseudoacontias, Pygomeles, Scelotes, Scincopus, Scincus,
            Scolecoseps, Sepsina, Sepsophis, Sirenoscincus, Typhlacontias, Voeltzkowia</em>); <strong>Xantusiidae</strong>, Cricosaurinae (<em>Cricosaura</em>), Lepidophyminae (<em>Lepidophyma</em>), Xantusiinae (<em>Xantusia</em>)
      </p>
      <h4>Lacertoidea (including Amphisbaenia)</h4>
      <p style="line-height:160%"><strong>Amphisbaenidae</strong> (<em>Amphisbaena, Ancylocranium, Baikia, Chirindia, Cynisca, Dalophia, Geocalamus, Loveridgea,
            Mesobaena, Monopeltis, Zygaspis</em>); <strong>Bipedidae</strong> (<em>Bipes</em>); <strong>Blanidae</strong> (<em>Blanus</em>); <strong>Cadeidae</strong> (<em>Cadea</em>); <strong>Gymnophthalmidae</strong>, Alopoglossinae (<em>Alopoglossus</em>, <em>Ptychoglossus</em>), Bachiinae (<em>Bachia</em>), Cercosaurinae (<em>Anadia, Cercosaura, Echinosaura, Euspondylus, Macropholidus, Neusticurus, Opipeuter,
            Petracola, Pholidobolus, Placosoma, Potamites, Proctoporus, Riama, Riolama, Teuchocercus</em>), Ecpleopinae (<em>Adercosaurus, Amapasaurus, Anotosaura, Arthrosaura, Colobosauroides, Dryadosaura,
            Ecpleopus, Kaieteurosaurus, Leposoma, Marinussaurus, Pantepuisaurus</em>), Gymnophthalminae (<em>Acratosaura, Alexandresaurus, Calyptommatus, Caparaonia, Colobodactylus, Colobosaura,
            Gymnophthalmus, Heterodactylus, Iphisa, Micrablepharus, Nothobachia, Procellosaurinus,
            Psilophthalmus, Scriptosaura, Stenolepis, Tretioscincus, Vanzosaura</em>), Rhachisaurinae (<em>Rhachisaurus</em>); <strong>Lacertidae</strong>, Gallotiinae (<em>Gallotia</em>, <em>Psammodromus</em>), Lacertinae (<em>Acanthodactylus, Adolfus, Algyroides, Anatololacerta, Apathya, Archaeolacerta, Atlantolacerta,
            Australolacerta, Congolacerta, Dalmatolacerta, Darevskia, Dinarolacerta, Eremias,
            Gastropholis, Heliobolus, Hellenolacerta, Holaspis, Iberolacerta, Ichnotropis, Iranolacerta,
            Lacerta, Latastia, Meroles, Mesalina, Nucras, Omanosaura, Ophisops, Parvilacerta,
            Pedioplanis, Philochortus, Phoenicolacerta, Podarcis, Poromera, Pseuderemias, Scelarcis,
            Takydromus, Teira, Timon, Tropidosaura, Zootoca</em>); <strong>Rhineuridae</strong> (<em>Rhineura</em>); <strong>Teiidae</strong>, Teiinae (<em>Ameiva</em>, <em>Aspidoscelis</em>, <em>Cnemidophorus</em>, <em>Dicrodon</em>, <em>Kentropyx</em>, <em>Teius</em>), Tupinambinae (<em>Callopistes</em>, <em>Crocodilurus</em>, <em>Dracaena</em>, <em>Tupinambis</em>); <strong>Trogonophiidae</strong> (<em>Agamodon</em>, <em>Diplometopon</em>, <em>Pachycalamus</em>, <em>Trogonophis</em>)
      </p>
      <h4>Iguania</h4>
      <p style="line-height:160%"><strong>Agamidae</strong>, Agaminae (<em>Acanthocercus, Agama, Brachysaura, Bufoniceps, Laudakia, Phrynocephalus, Pseudotrapelus,
            Trapelus, Xenagama</em>), Amphibolurinae (<em>Amphibolurus, Chelosania, Chlamydosaurus, Cryptagama, Ctenophorus, Diporiphora, Hypsilurus,
            Intellagama, Lophognathus, Moloch, Physignathus, Pogona, Rankinia, Tympanocryptis</em>), Draconinae (<em>Acanthosaura, Aphaniotis, Bronchocela, Calotes, Ceratophora, Complicitus, Cophotis,
            Coryphophylax, Dendragama, Draco, Gonocephalus, Harpesaurus, Hypsicalotes, Japalura,
            Lophocalotes, Lyriocephalus, Mantheyus, Oriocalotes, Otocryptis, Phoxophrys, Psammophilus,
            Pseudocalotes, Pseudocophotis, Ptyctolaemus, Salea, Sitana, Thaumatorhynchus</em>), Hydrosaurinae (<em>Hydrosaurus</em>), Leiolepidinae (<em>Leiolepis</em>), Uromastycinae (<em>Uromastyx</em>); <strong>Chamaeleonidae</strong>, Brookesiinae (<em>Brookesia</em>), Chamaeleoninae (<em>Archaius, Bradypodion, Calumma, Chamaeleo, Furcifer, Kinyongia, Nadzikambia, Rhampholeon,
            Rieppeleon, Trioceros</em>); <strong>Corytophanidae</strong> (<em>Basiliscus</em>, <em>Corytophanes</em>, <em>Laemanctus</em>); <strong>Crotaphytidae</strong> (<em>Crotaphytus</em>, <em>Gambelia</em>); <strong>Dactyloidae</strong> (<em>Anolis</em>); <strong>Hoplocercidae</strong> (<em>Enyalioides</em>, <em>Hoplocercus</em>, <em>Morunasaurus</em>); <strong>Iguanidae</strong> (<em>Amblyrhynchus</em>, <em>Brachylophus</em>, <em>Conolophus</em>, <em>Ctenosaura</em>, <em>Cyclura</em>, <em>Dipsosaurus</em>, <em>Iguana</em>, <em>Sauromalus</em>); <strong>Leiocephalidae</strong> (<em>Leiocephalus</em>); <strong>Leiosauridae</strong>, Enyaliinae (<em>Anisolepis</em>, <em>Enyalius</em>, <em>Urostrophus</em>), Leiosaurinae (<em>Diplolaemus</em>, <em>Leiosaurus</em>, <em>Pristidactylus</em>); <strong>Liolaemidae</strong> (<em>Ctenoblepharys</em>, <em>Liolaemus</em>, <em>Phymaturus</em>); <strong>Opluridae</strong> (<em>Chalarodon</em>, <em>Oplurus</em>); <strong>Phrynosomatidae</strong> (<em>Callisaurus</em>, <em>Cophosaurus</em>, <em>Holbrookia</em>, <em>Petrosaurus</em>, <em>Phrynosoma</em>, <em>Sceloporus</em>, <em>Uma</em>, <em>Urosaurus</em>, <em>Uta</em>); <strong>Polychrotidae</strong> (<em>Polychrus</em>); <strong>Tropiduridae</strong> (<em>Eurolophosaurus</em>, <em>Microlophus</em>, <em>Plica</em>, <em>Stenocercus</em>, <em>Strobilurus</em>, <em>Tropidurus</em>, <em>Uracentron</em>, <em>Uranoscodon</em>)
      </p>
      <h4>Anguimorpha</h4>
      <p style="line-height:160%"><strong>Anguidae</strong>, Anguinae (<em>Anguis</em>, <em>Dopasia</em>, <em>Ophisaurus</em>, <em>Pseudopus</em>), Diploglossinae (<em>Celestus</em>, <em>Diploglossus</em>, <em>Ophiodes</em>), Gerrhonotinae (<em>Abronia</em>, <em>Barisia</em>, <em>Coloptychon</em>, <em>Elgaria</em>, <em>Gerrhonotus</em>, <em>Mesaspis</em>); <strong>Anniellidae</strong> (<em>Anniella</em>); <strong>Helodermatidae</strong> (<em>Heloderma</em>); <strong>Lanthanotidae</strong> (<em>Lanthanotus</em>); <strong>Shinisauridae</strong> (<em>Shinisaurus</em>); <strong>Varanidae</strong> (<em>Varanus</em>); <strong>Xenosauridae</strong> (<em>Xenosaurus</em>)
      </p>
      <h4>Serpentes</h4>
      <p style="line-height:160%"><strong>Acrochordidae</strong> (<em>Acrochordus</em>); <strong>Aniliidae</strong> (<em>Anilius</em>); <strong>Anomalepididae</strong> (<em>Anomalepis</em>, <em>Helminthophis</em>, <em>Liotyphlops</em>, <em>Typhlophis</em>); <strong>Anomochilidae</strong> (<em>Anomochilus</em>); <strong>Boidae</strong>, Boinae (<em>Boa</em>, <em>Corallus</em>, <em>Epicrates</em>, <em>Eunectes</em>), Candoiinae (<em>Candoia</em>), Erycinae (<em>Eryx</em>), Sanziniinae (<em>Acrantophis</em>, <em>Sanzinia</em>), Ungaliophiinae (<em>Charina</em>, <em>Exiliboa</em>, <em>Lichanura</em>, <em>Ungaliophis</em>); <strong>Bolyeriidae</strong> (<em>Bolyeria</em>, <em>Casarea</em>); <strong>Calabariidae</strong> (<em>Calabaria</em>); <strong>Colubridae</strong><em>incertae sedis</em> (<em>Blythia</em>, <em>Cyclocorus</em>, <em>Elapoidis</em>, <em>Gongylosoma</em>, <em>Helophis</em>, <em>Myersophis</em>, <em>Oreocalamus, Poecilopholis</em>, <em>Rhabdops</em>, <em>Tetralepis</em>), Calamariinae (<em>Calamaria</em>, <em>Calamorhabdium</em>, <em>Collorhabdium</em>, <em>Etheridgeum</em>, <em>Macrocalamus</em>, <em>Pseudorabdion</em>, <em>Rabdion</em>), Colubrinae (<em>Aeluroglena, Ahaetulla, Aprosdoketophis, Archelaphe, Argyrogena, Arizona, Bamanophis,
            Bogertophis, Boiga, Cemophora, Chilomeniscus, Chionactis, Chironius, Chrysopelea,
            Coelognathus, Coluber, Colubroelaps, Conopsis, Coronella, Crotaphopeltis, Cyclophiops,
            Dasypeltis, Dendrelaphis, Dendrophidion, Dipsadoboa, Dispholidus, Dolichophis, Drymarchon,
            Drymobius, Drymoluber, Dryocalamus, Dryophiops, Eirenis, Elachistodon, Elaphe, Euprepiophis,
            Ficimia, Geagras, Gonyophis, Gonyosoma, Gyalopion, Hapsidophrys, Hemerophis, Hemorrhois,
            Hierophis, Lampropeltis, Leptodrymus, Leptophis, Lepturophis, Limnophis, Liopeltis,
            Lycodon, Lytorhynchus, Macroprotodon, Mastigodryas, Meizodon, Oligodon, Oocatochus,
            Opheodrys, Oreocryptophis, Orthriophis, Oxybelis, Pantherophis, Philothamnus, Phyllorhynchus,
            Pituophis, Platyceps, Pseudelaphe, Pseudoficimia, Pseustes, Ptyas, Rhadinophis, Rhamnophis,
            Rhinechis, Rhinobothryum, Rhinocheilus, Rhynchocalamus, Rhynchophis, Salvadora, Scaphiophis,
            Scolecophis, Senticolis, Simophis, Sonora, Spalerosophis, Spilotes, Stegonotus, Stenorrhina,
            Symphimus, Sympholis, Tantilla, Tantillita, Telescopus, Thelotornis, Thrasops, Toxicodryas,
            Trimorphodon, Xenelaphis, Xyelodontophis, Zamenis</em>), Dipsadinae (<em>Adelphicos, Alsophis, Amastridium, Amnesteophis, Antillophis, Apostolepis, Arrhyton,
            Atractus, Boiruna, Borikenophis, Caaeteboia, Calamodontophis, Caraiba, Carphophis,
            Cercophis, Chapinophis, Chersodromus, Clelia, Coniophanes, Conophis, Contia, Coronelaps,
            Crisantophis, Cryophis, Cubophis, Darlingtonia, Diadophis, Diaphorolepis, Dipsas,
            Ditaxodon, Drepanoides, Echinanthera, Elapomorphus, Emmochliophis, Enuliophis, Enulius,
            Erythrolamprus, Farancia, Geophis, Gomesophis, Haitiophis, Helicops, Heterodon, Hydrodynastes,
            Hydromorphus, Hydrops, Hypsiglena, Hypsirhynchus, Ialtris, Imantodes, Leptodeira,
            Lioheterophis, Lygophis, Magliophis, Manolepis, Mussurana, Ninia, Nothopsis, Ocyophis,
            Omoadiphas, Oxyrhopus, Paraphimophis, Phalotris, Philodryas, Phimophis, Plesiodipsas,
            Pliocercus, Pseudalsophis, Pseudoboa, Pseudoeryx, Pseudoleptodeira, Pseudotomodon,
            Psomophis, Ptychophis, Rhachidelus, Rhadinaea, Rhadinella, Rhadinophanes, Rodriguesophis,
            Saphenophis, Schwartzophis, Sibon, Sibynomorphus, Siphlophis, Sordellina, Synophis,
            Tachymenis, Taeniophallus, Tantalophis, Thamnodynastes, Thermophis, Tomodon, Tretanorhinus,
            Trimetopon, Tropidodipsas, Tropidodryas, Uromacer, Uromacerina, Urotheca, Xenodon,
            Xenopholis</em>), Grayiinae (<em>Grayia</em>), Natricinae (<em>Adelophis, Afronatrix, Amphiesma, Amphiesmoides, Anoplohydrus, Aspidura, Atretium,
            Balanophis, Clonophis, Hologerrhum, Hydrablabes, Hydraethiops, Iguanognathus, Lycognathophis,
            Macropisthodon, Natriciteres, Natrix, Nerodia, Opisthotropis, Parahelicops, Pararhabdophis,
            Paratapinophis, Regina, Rhabdophis, Seminatrix, Sinonatrix, Storeria, Thamnophis,
            Trachischium, Tropidoclonion, Tropidonophis, Virginia, Xenochrophis</em>), Pseudoxenodontinae (<em>Plagiopholis</em>, <em>Pseudoxenodon</em>), Sibynophiinae (<em>Scaphiodontophis</em>, <em>Sibynophis</em>); <strong>Cylindrophiidae</strong> (<em>Cylindrophis</em>); <strong>Elapidae</strong> (<em>Acanthophis, Aipysurus, Aspidelaps, Aspidomorphus, Austrelaps, Bungarus, Cacophis,
            Calliophis, Cryptophis, Demansia, Dendroaspis, Denisonia, Drysdalia, Echiopsis, Elapognathus,
            Elapsoidea, Emydocephalus, Ephalophis, Furina, Hemachatus, Hemiaspis, Hemibungarus,
            Hoplocephalus, Hydrelaps, Hydrophis, Kolpophis, Laticauda, Loveridgelaps, Maticora,
            Micropechis, Micruroides, Micrurus, Naja, Notechis, Ogmodon, Ophiophagus, Oxyuranus,
            Parahydrophis, Parapistocalamus, Parasuta, Pseudechis, Pseudohaje, Pseudolaticauda,
            Pseudonaja, Rhinoplocephalus, Salomonelaps, Simoselaps, Sinomicrurus, Suta, Thalassophis,
            Toxicocalamus, Tropidechis, Vermicella, Walterinnesia</em>); <strong>Gerrhopilidae</strong> (<em>Gerrhopilus</em>); <strong>Homalopsidae</strong> (<em>Bitia</em>, <em>Brachyorrhos</em>, <em>Cantoria</em>, <em>Cerberus</em>, <em>Djokoiskandarus</em>, <em>Enhydris</em>, <em>Erpeton</em>, <em>Fordonia</em>, <em>Gerarda</em>, <em>Heurnia</em>, <em>Homalopsis</em>, <em>Myron</em>, <em>Pseudoferania</em>); <strong>Lamprophiidae</strong><em>incertae sedis</em> (<em>Micrelaps</em>, <em>Montaspis</em>, <em>Oxyrhabdium</em>), Aparallactinae (<em>Amblyodipsas</em>, <em>Aparallactus</em>, <em>Brachyophis</em>, <em>Chilorhinophis</em>, <em>Elapotinus</em>, <em>Hypoptophis</em>, <em>Macrelaps</em>, <em>Polemon</em>, <em>Xenocalamus</em>), Atractaspidinae (<em>Atractaspis</em>, <em>Homoroselaps</em>), Lamprophiinae (<em>Boaedon</em>, <em>Bothrophthalmus</em>, <em>Chamaelycus</em>, <em>Dendrolycus</em>, <em>Gonionotophis</em>, <em>Hormonotus</em>, <em>Inyoka</em>, <em>Lamprophis</em>, <em>Lycodonomorphus</em>, <em>Lycophidion</em>, <em>Pseudoboodon</em>), Prosymninae (<em>Prosymna</em>), Psammophiinae (<em>Dipsina</em>, <em>Hemirhagerrhis</em>, <em>Malpolon</em>, <em>Mimophis</em>, <em>Psammophis</em>, <em>Psammophylax</em>, <em>Rhagerhis</em>, <em>Rhamphiophis</em>), Pseudaspidinae (<em>Buhoma, Psammodynastes</em>, <em>Pseudaspis</em>, <em>Pythonodipsas</em>), Pseudoxyrhophiine (<em>Alluaudina</em>, <em>Amplorhinus</em>, <em>Bothrolycus</em>, <em>Brygophis</em>, <em>Compsophis</em>, <em>Ditypophis</em>, <em>Dromicodryas</em>, <em>Duberria</em>, <em>Exallodontophis</em>, <em>Heteroliodon</em>, <em>Ithycyphus</em>, <em>Langaha</em>, <em>Leioheterodon</em>, <em>Liophidium</em>, <em>Liopholidophis</em>, <em>Lycodryas</em>, <em>Madagascarophis</em>, <em>Micropisthodon</em>, <em>Pararhadinaea</em>, <em>Parastenophis</em>, <em>Phisalixella</em>, <em>Pseudoxyrhopus</em>, <em>Thamnosophis</em>); <strong>Leptotyphlopidae</strong> (<em>Epacrophis</em>, <em>Epictia</em>, <em>Leptotyphlops</em>, <em>Mitophis</em>, <em>Myriopholis</em>, <em>Namibiana</em>, <em>Rena</em>, <em>Rhinoleptus</em>, <em>Siagonodon</em>, <em>Tetracheilostoma</em>, <em>Tricheilostoma</em>, <em>Trilepida</em>); <strong>Loxocemidae</strong> (<em>Loxocemus</em>); <strong>Pareatidae</strong> (<em>Aplopeltura</em>, <em>Asthenodipsas</em>, <em>Pareas</em>); <strong>Pythonidae</strong> (<em>Antaresia</em>, <em>Apodora</em>, <em>Aspidites</em>, <em>Bothrochilus</em>, <em>Broghammerus</em>, <em>Leiopython</em>, <em>Liasis</em>, <em>Morelia</em>, <em>Python</em>); <strong>Tropidophiidae</strong> (<em>Trachyboa</em>, <em>Tropidophis</em>); <strong>Typhlopidae</strong> (<em>Acutotyphlops</em>, <em>Afrotyphlops</em>, <em>Austrotyphlops</em>, <em>Cyclotyphlops</em>, <em>Grypotyphlops</em>, <em>Letheobia</em>, <em>Megatyphlops</em>, <em>Ramphotyphlops</em>, <em>Rhinotyphlops</em>, <em>Typhlops</em>); <strong>Uropeltidae</strong> (<em>Brachyophidium</em>, <em>Melanophidium</em>, <em>Platyplectrurus</em>, <em>Plectrurus</em>, <em>Pseudotyphlops</em>, <em>Rhinophis</em>, <em>Teretrurus</em>, <em>Uropeltis</em>); <strong>Viperidae</strong>, Azemiopinae (<em>Azemiops</em>), Crotalinae (<em>Agkistrodon</em>, <em>Atropoides</em>, <em>Bothriechis</em>, <em>Bothriopsis</em>, <em>Bothrocophias</em>, <em>Bothropoides</em>, <em>Bothrops</em>, <em>Calloselasma</em>, <em>Cerrophidion</em>, <em>Crotalus</em>, <em>Deinagkistrodon</em>, <em>Garthius</em>, <em>Gloydius</em>, <em>Hypnale</em>, <em>Lachesis</em>, <em>Mixcoatlus</em>, <em>Ophryacus</em>, <em>Ovophis</em>, <em>Porthidium</em>, <em>Protobothrops</em>, <em>Rhinocerophis</em>, <em>Sistrurus</em>, <em>Trimeresurus</em>, <em>Tropidolaemus</em>), Viperinae (<em>Atheris</em>, <em>Bitis</em>, <em>Causus</em>, <em>Cerastes</em>, <em>Daboia</em>, <em>Echis</em>, <em>Eristicophis</em>, <em>Macrovipera</em>, <em>Montatheris</em>, <em>Montivipera</em>, <em>Proatheris</em>, <em>Pseudocerastes</em>, <em>Vipera</em>); <strong>Xenodermatidae</strong> (<em>Achalinus</em>, <em>Fimbrios</em>, <em>Stoliczkia</em>, <em>Xenodermus</em>, <em>Xylophis</em>); <strong>Xenopeltidae</strong> (<em>Xenopeltis</em>); <strong>Xenophidiidae</strong> (<em>Xenophidion</em>); <strong>Xenotyphlopidae</strong> (<em>Xenotyphlops</em>)
      </p>
   </div>
</section>
<section><a name="sec8"></a><h3>Competing interests</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">The authors declare that they have no competing interests.</p>
   </div>
</section>
<section><a name="sec9"></a><h3>Authorsâ€™ contributions</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">RAP and JJW conceived the study. RAP, FTB, and JJW conducted analyses and checked
         results. RAP, FTB, and JJW wrote the MS. All authors read and approved the final manuscript.
      </p>
   </div>
</section>
<section><a name="ack"></a><h3>Acknowledgements</h3>
   <div class="collapsible-content">
      <p style="line-height:160%">We thank the many researchers who made this study possible through their detailed
         studies of squamate phylogeny with a (mostly) shared set of molecular markers, and
         uploading their sequence data to GenBank. We thank E. Paradis, J. Lombardo T. Guiher,
         R. Walsh, and P. Muzio for computational assistance, T. Gamble, D. Frost, D. Cannatella,
         H. Zaher, F. Grazziotin, P. Uetz, T. Jackman, and A. Bauer for taxonomic advice, and
         A. Larson, M. Vences, and five anonymous reviewers for comments on this manuscript.
         The research was supported in part by the U.S. National Science Foundation, including
         a Bioinformatics Postdoctoral grant to R.A.P. (DBI-0905765), an AToL grant to J.J.W.
         (EF-0334923), and grants to the CUNY HPCC (CNS-0958379 and CNS-0855217).
      </p>
   </div>
</section>
<section><a name="refs"></a><h3>References</h3>
   <div id="article-references" class="collapsible-content">
      <ol id="references">
         <li id="B1">
            <p><a name="B1"></a> Uetz P: <p><em>The Reptile Database</em>.
               <p><a href="http://www.reptile-database.org/">http://www.reptile-database.org/</a> <a href="http://www.webcitation.org/query.php?url=http://www.reptile-database.org/&amp;refdoi=10.1186/1471-2148-13-93" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a> Accessed December, 2012
               </p> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B1" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B1','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B2">
            <p><a name="B2"></a> Greene HW: <em>Snakes: the Evolution of Mystery in Nature</em>.  Berkeley: University of California Press;  1997. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B2" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B2','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B3">
            <p><a name="B3"></a> Vitt LJ,  Caldwell JP: <em>Herpetology</em>.  4th edition.  Burlington: Elsevier;  2009. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B3" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B3','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B4">
            <p><a name="B4"></a> Pianka ER,  Vitt LJ: <em>Lizards: Windows to the Evolution of Diversity</em>.  Berkeley: University of California Press;  2003. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B4" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B4','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B5">
            <p><a name="B5"></a> Kasturiratne A,  Wickremasinghe AR,  de Silva N,  Gunawardena NK,  Pathmeswaran A,  Premaratna R,  Savioli L,  Lalloo DG,  de Silva HJ: <strong> The global burden of snakebite: a literature analysis and modelling based on regional
                  estimates of envenoming and deaths. </strong><p><em>PLoS Med</em> 2008,  <strong>5</strong><strong>:</strong>1591-1604.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B5" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B5','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B6">
            <p><a name="B6"></a> Mahdavi A,  Ferreira L,  Sundback C,  Nichol JW,  Chan EP,  Carter DJD,  Bettinger CJ,  Patanavanich S,  Chignozha L,  Ben-Joseph E, <em>et al</em>.: <strong> A biodegradable and biocompatible gecko-inspired tissue adhesive. </strong><p><em>Proc Natl Acad Sci USA</em> 2008,  <strong>105</strong><strong>:</strong>2307-2312.  <a target="_blank" href="/pubmed/18287082">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18287082">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18287082">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B6" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B6','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B7">
            <p><a name="B7"></a> Geim AK,  Dubonos SV,  Grigorieva IV,  Novoselov KS,  Zhukov AA,  Shapoval SY: <strong> Microfabricated adhesive mimicking gecko foot-hair. </strong><p><em>Nat Mater</em> 2003,  <strong>2</strong><strong>:</strong>461-463.  <a target="_blank" href="/pubmed/12776092">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12776092">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B7" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B7','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B8">
            <p><a name="B8"></a> Huey RB,  Bennett AF: <strong> Phylogenetic studies of coadaptation: preferred temperatures versus optimal performance
                  temperatures of lizards. </strong><p><em>Evolution</em> 1987,  <strong>41</strong><strong>:</strong>1098-1115.   <a target="_blank" href="http://dx.doi.org/10.2307/2409194">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B8" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B8','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B9">
            <p><a name="B9"></a> Losos JB: <strong> The evolution of form and function: morphology and locomotor performance in West-Indian
                  <em>Anolis</em> lizards. </strong><p><em>Evolution</em> 1990,  <strong>44</strong><strong>:</strong>1189-1203.   <a target="_blank" href="http://dx.doi.org/10.2307/2409282">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B9" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B9','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B10">
            <p><a name="B10"></a> Wiens JJ,  Brandley MC,  Reeder TW: <strong> Why does a trait evolve multiple times within a clade? Repeated evolution of snakelike
                  body form in squamate reptiles. </strong><p><em>Evolution</em> 2006,  <strong>60</strong><strong>:</strong>123-141.  <a target="_blank" href="/pubmed/16568638">PubMed&nbsp;Abstract</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B10" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B10','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B11">
            <p><a name="B11"></a> Bergmann PJ,  Irschick DJ: <strong> Vertebral evolution and the diversification of squamate reptiles. </strong><p><em>Evolution</em> 2012,  <strong>66</strong><strong>:</strong>1044-1058.  <a target="_blank" href="/pubmed/22486688">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1558-5646.2011.01491.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B11" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B11','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B12">
            <p><a name="B12"></a> Losos JB,  Hillis DM,  Greene HW: <strong> Who speaks with a forked tongue? </strong><p><em>Science</em> 2012,  <strong>338</strong><strong>:</strong>1428-1429.  <a target="_blank" href="/pubmed/23239723">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23239723">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B12" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B12','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B13">
            <p><a name="B13"></a> Estes R,  de Queiroz K,  Gauthier J: <strong> Phylogenetic relationships within Squamata. </strong>In <em>Phylogenetic Relationships of the Lizard Families</em>.  Edited by Estes R, Pregill G.  Stanford: Stanford University Press;  1988:<strong>:</strong>119-281.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B13" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B13','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B14">
            <p><a name="B14"></a> Gauthier JA,  Kearney M,  Maisano JA,  Rieppel O,  Behike ADB: <strong> Assembling the Squamate Tree of Life: perspectives from the phenotype and the fossil
                  record. </strong><p><em>Bull Peabody Mus Nat Hist</em> 2012,  <strong>53</strong><strong>:</strong>3-308.   <a target="_blank" href="http://dx.doi.org/10.3374/014.053.0101">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B14" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B14','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B15">
            <p><a name="B15"></a> Conrad JL: <strong> Phylogeny and systematics of Squamata (Reptilia) based on morphology. </strong><p><em>Bull Am Mus Nat Hist</em> 2008,  <strong>310</strong><strong>:</strong>1-182.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B15" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B15','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B16">
            <p><a name="B16"></a> Mulcahy DG,  Noonan BP,  Moss T,  Townsend TM,  Reeder TW,  Sites JW Jr,  Wiens JJ: <strong> Estimating divergence times and evaluating dating methods using phylogenomic and mitochondrial
                  data in squamate reptiles. </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>65</strong><strong>:</strong>974-991.  <a target="_blank" href="/pubmed/22982760">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22982760">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B16" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B16','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B17">
            <p><a name="B17"></a> Townsend TM,  Larson A,  Louis E,  Macey JR: <strong> Molecular phylogenetics of Squamata: the position of snakes, amphisbaenians, and dibamids,
                  and the root of the squamate tree. </strong><p><em>Syst Biol</em> 2004,  <strong>53</strong><strong>:</strong>735-757.  <a target="_blank" href="/pubmed/15545252">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15545252">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B17" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B17','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B18">
            <p><a name="B18"></a> Vidal N,  Hedges SB: <strong> The phylogeny of squamate reptiles (lizards, snakes, and amphisbaenians) inferred
                  from nine nuclear protein-coding genes. </strong><p><em>CR Biol</em> 2005,  <strong>328</strong><strong>:</strong>1000-1008.   <a target="_blank" href="http://dx.doi.org/10.1016/j.crvi.2005.10.001">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B18" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B18','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B19">
            <p><a name="B19"></a> Wiens JJ,  Kuczynski CA,  Townsend T,  Reeder TW,  Mulcahy DG,  Sites JW: <strong> Combining phylogenomics and fossils in higher-level squamate reptile phylogeny: molecular
                  data change the placement of fossil taxa. </strong><p><em>Syst Biol</em> 2010,  <strong>59</strong><strong>:</strong>674-688.  <a target="_blank" href="/pubmed/20930035">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20930035">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B19" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B19','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B20">
            <p><a name="B20"></a> Wiens JJ,  Hutter CR,  Mulcahy DG,  Noonan BP,  Townsend TM,  Sites JW,  Reeder TW: <strong> Resolving the phylogeny of lizards and snakes (Squamata) with extensive sampling of
                  genes and species. </strong><p><em>Biol Lett</em> 2012,  <strong>8</strong><strong>:</strong>1043-1046.  <a target="_blank" href="/pubmed/22993238">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22993238">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B20" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B20','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B21">
            <p><a name="B21"></a> Camp CL: <strong> Classification of the lizards. </strong><p><em>Bull Am Mus Nat Hist</em> 1923,  <strong>48</strong><strong>:</strong>289-480.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B21" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B21','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B22">
            <p><a name="B22"></a> Lee MSY: <strong> Convergent evolution and character correlation in burrowing reptiles: towards a resolution
                  of squamate relationships. </strong><p><em>Biol J Linn Soc</em> 1998,  <strong>65</strong><strong>:</strong>369-453.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.1998.tb01148.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B22" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B22','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B23">
            <p><a name="B23"></a> Vidal N,  Hedges SB: <strong> Molecular evidence for a terrestrial origin of snakes. </strong><p><em>Proc R Soc B</em> 2004,  <strong>271</strong><strong>:</strong>S226-S229.  <a target="_blank" href="/pubmed/15252991">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15252991">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15252991">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B23" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B23','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B24">
            <p><a name="B24"></a> Fry BG,  Vidal N,  Norman JA,  Vonk FJ,  Scheib H,  Ramjan SFR,  Kuruppu S,  Fung K,  Hedges SB,  Richardson MK, <em>et al</em>.: <strong> Early evolution of the venom system in lizards and snakes. </strong><p><em>Nature</em> 2006,  <strong>439</strong><strong>:</strong>584-588.  <a target="_blank" href="/pubmed/16292255">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16292255">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B24" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B24','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B25">
            <p><a name="B25"></a> Zwickl DJ,  Hillis DM: <strong> Increased taxon sampling greatly reduces phylogenetic error. </strong><p><em>Syst Biol</em> 2002,  <strong>51</strong><strong>:</strong>588-598.  <a target="_blank" href="/pubmed/12228001">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12228001">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B25" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B25','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B26">
            <p><a name="B26"></a> Poe S: <strong> Evaluation of the strategy of long-branch subdivision to improve the accuracy of phylogenetic
                  methods. </strong><p><em>Syst Biol</em> 2003,  <strong>52</strong><strong>:</strong>423-428.  <a target="_blank" href="/pubmed/12775529">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12775529">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B26" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B26','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B27">
            <p><a name="B27"></a> Heath TA,  Zwickl DJ,  Kim J,  Hillis DM: <strong> Taxon sampling affects inferences of macroevolutionary processes from phylogenetic
                  trees. </strong><p><em>Syst Biol</em> 2008,  <strong>57</strong><strong>:</strong>160-166.  <a target="_blank" href="/pubmed/18300029">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18300029">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B27" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B27','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B28">
            <p><a name="B28"></a> Graybeal A: <strong> Is it better to add taxa or characters to a difficult phylogenetic problem? </strong><p><em>Syst Biol</em> 1998,  <strong>47</strong><strong>:</strong>9-17.  <a target="_blank" href="/pubmed/12064243">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1080/106351598260996">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B28" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B28','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B29">
            <p><a name="B29"></a> Blankers T,  Townsend TM,  Pepe K,  Reeder TW,  Wiens JJ: <strong> Contrasting global-scale evolutionary radiations: phylogeny, diversification, and
                  morphological evolution in the major clades of iguanian lizards. </strong><p><em>Biol J Linn Soc</em> 2013,  <strong>108</strong><strong>:</strong>127-143.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.2012.01988.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B29" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B29','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B30">
            <p><a name="B30"></a> Schulte JA,  Moreno-Roark F: <strong> Live birth among iguanian lizards predates Pliocene-Pleistocene glaciations. </strong><p><em>Biol Lett</em> 2010,  <strong>6</strong><strong>:</strong>216-218.  <a target="_blank" href="/pubmed/19812068">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19812068">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19812068">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B30" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B30','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B31">
            <p><a name="B31"></a> Frost DR,  Etheridge RE,  Janies D,  Titus TA: <strong> Total evidence, sequence alignment, evolution of polychrotid lizards, and a reclassification
                  of the Iguania (Squamata: Iguania). </strong><p><em>Am Mus Novit</em> 2001,  <strong>3343</strong><strong>:</strong>38.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B31" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B31','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B32">
            <p><a name="B32"></a> Macey JR,  Schulte JA,  Larson A,  Ananjeva NB,  Wang YZ,  Pethiyagoda R,  Rastegar-Pouyani N,  Papenfuss TJ: <strong> Evaluating trans-Tethys migration: an example using acrodont lizard phylogenetics. </strong><p><em>Syst Biol</em> 2000,  <strong>49</strong><strong>:</strong>233-256.  <a target="_blank" href="/pubmed/12118407">PubMed&nbsp;Abstract</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B32" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B32','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B33">
            <p><a name="B33"></a> Schulte JA,  Valladares JP,  Larson A: <strong> Phylogenetic relationships within Iguanidae inferred using molecular and morphological
                  data and a phylogenetic taxonomy of iguanian lizards. </strong><p><em>Herpetologica</em> 2003,  <strong>59</strong><strong>:</strong>399-419.   <a target="_blank" href="http://dx.doi.org/10.1655/02-48">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B33" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B33','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B34">
            <p><a name="B34"></a> Townsend TM,  Mulcahy DG,  Noonan BP,  Sites JW,  Kuczynski CA,  Wiens JJ,  Reeder TW: <strong> Phylogeny of iguanian lizards inferred from 29 nuclear loci, and a comparison of concatenated
                  and species-tree approaches for an ancient, rapid radiation. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>61</strong><strong>:</strong>363-380.  <a target="_blank" href="/pubmed/21787873">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21787873">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B34" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B34','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B35">
            <p><a name="B35"></a> Slowinski JB,  Lawson R: <strong> Snake phylogeny: evidence from nuclear and mitochondrial genes. </strong><p><em>Mol Phylogenet Evol</em> 2002,  <strong>24</strong><strong>:</strong>194-202.  <a target="_blank" href="/pubmed/12144756">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12144756">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B35" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B35','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B36">
            <p><a name="B36"></a> Wiens JJ,  Kuczynski CA,  Smith SA,  Mulcahy DG,  Sites JW,  Townsend TM,  Reeder TW: <strong> Branch lengths, support, and congruence: testing the phylogenomic approach with 20
                  nuclear loci in snakes. </strong><p><em>Syst Biol</em> 2008,  <strong>57</strong><strong>:</strong>420-431.  <a target="_blank" href="/pubmed/18570036">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18570036">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B36" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B36','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B37">
            <p><a name="B37"></a> Lawson R,  Slowinski JB,  Burbrink FT: <strong> A molecular approach to discerning the phylogenetic placement of the enigmatic snake
                  <em>Xenophidion schaeferi</em> among the Alethinophidia. </strong><p><em>J Zool</em> 2004,  <strong>263</strong><strong>:</strong>285-294.   <a target="_blank" href="http://dx.doi.org/10.1017/S0952836904005278">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B37" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B37','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B38">
            <p><a name="B38"></a> Vidal N,  Marin J,  Morini M,  Donnellan S,  Branch WR,  Thomas R,  Vences M,  Wynn A,  Cruaud C,  Hedges SB: <strong> Blindsnake evolutionary tree reveals long history on Gondwana. </strong><p><em>Biol Lett</em> 2010,  <strong>6</strong><strong>:</strong>558-561.  <a target="_blank" href="/pubmed/20356885">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20356885">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20356885">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B38" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B38','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B39">
            <p><a name="B39"></a> Adalsteinsson SA,  Branch WR,  Trape S,  Vitt LJ,  Hedges SB: <strong> Molecular phylogeny, classification, and biogeography of snakes of the family Leptotyphlopidae
                  (Reptilia, Squamata). </strong><p><em>Zootaxa</em> 2009,  <strong>2244</strong><strong>:</strong>1-50.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B39" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B39','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B40">
            <p><a name="B40"></a> Kelly CMR,  Barker NP,  Villet MH,  Broadley DG: <strong> Phylogeny, biogeography and classification of the snake superfamily Elapoidea: a rapid
                  radiation in the late Eocene. </strong><p><em>Cladistics</em> 2009,  <strong>25</strong><strong>:</strong>38-63.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-0031.2008.00237.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B40" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B40','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B41">
            <p><a name="B41"></a> Pyron RA,  Burbrink FT,  Colli GR,  de Oca ANM,  Vitt LJ,  Kuczynski CA,  Wiens JJ: <strong> The phylogeny of advanced snakes (Colubroidea), with discovery of a new subfamily
                  and comparison of support methods for likelihood trees. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>58</strong><strong>:</strong>329-342.  <a target="_blank" href="/pubmed/21074626">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21074626">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B41" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B41','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B42">
            <p><a name="B42"></a> Zaher H,  Grazziotin FG,  Cadle JE,  Murphy RW,  de Moura JC,  Bonatto SL: <strong> Molecular phylogeny of advanced snakes (Serpentes, Caenophidia) with an emphasis on
                  South American xenodontines: a revised classification and descriptions of new taxa. </strong><p><em>Pap Av Zool</em> 2009,  <strong>49</strong><strong>:</strong>115-153.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B42" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B42','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B43">
            <p><a name="B43"></a> Grazziotin FG,  Zaher H,  Murphy RW,  Scrocchi G,  Benavides MA,  Zhang Y-P,  Bonatto SL: <strong> Molecular phylogeny of the New World Dipsadidae (Serpentes: Colubroidea): a reappraisal. </strong><p><em>Cladistics</em> 2012,  <strong>28</strong><strong>:</strong>437-459.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-0031.2012.00393.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B43" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B43','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B44">
            <p><a name="B44"></a> Pyron RA,  Kandambi HKDK,  Hendry CR,  Pushpamal V,  Burbrink FT,  Somaweera R: <strong> Genus-level phylogeny of snakes reveals the origins of species richness in Sri Lanka. </strong><p><em>Mol Phylogenet Evol</em> 2013,  <strong>66</strong><strong>:</strong>969-978.  <a target="_blank" href="/pubmed/23261713">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23261713">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B44" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B44','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B45">
            <p><a name="B45"></a> Vidal N,  Delmas AS,  David P,  Cruaud C,  Coujoux A,  Hedges SB: <strong> The phylogeny and classification of caenophidian snakes inferred from seven nuclear
                  protein-coding genes. </strong><p><em>CR Biol</em> 2007,  <strong>330</strong><strong>:</strong>182-187.   <a target="_blank" href="http://dx.doi.org/10.1016/j.crvi.2006.10.001">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B45" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B45','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B46">
            <p><a name="B46"></a> Sanders KL,  Lee MSY,  Bertozzi T,  Rasmussen AR: <strong> Multilocus phylogeny and recent rapid radiation of the viviparous sea snakes (Elapidae:
                  Hydrophiinae). </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>66</strong><strong>:</strong>575-591.  <a target="_blank" href="/pubmed/23026811">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23026811">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B46" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B46','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B47">
            <p><a name="B47"></a> Noonan BP,  Chippindale PT: <strong> Dispersal and vicariance: the complex evolutionary history of boid snakes. </strong><p><em>Mol Phylogenet Evol</em> 2006,  <strong>40</strong><strong>:</strong>347-358.  <a target="_blank" href="/pubmed/16624591">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16624591">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B47" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B47','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B48">
            <p><a name="B48"></a> Lynch VJ,  Wagner GP: <strong> Did egg-laying boas break Dollo's law? Phylogenetic evidence for reversal to oviparity
                  in sand boas (<strong><em>Eryx</em></strong>: Boidae). </strong><p><em>Evolution</em> 2010,  <strong>64</strong><strong>:</strong>207-216.  <a target="_blank" href="/pubmed/19659599">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1558-5646.2009.00790.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B48" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B48','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B49">
            <p><a name="B49"></a> Schmitz A,  Brandley MC,  Mausfeld P,  Vences M,  Glaw F,  Nussbaum RA,  Reeder TW: <strong> Opening the black box: phylogenetics and morphological evolution of the Malagasy fossorial
                  lizards of the subfamily "Scincinae". </strong><p><em>Mol Phylogenet Evol</em> 2005,  <strong>34</strong><strong>:</strong>118-133.  <a target="_blank" href="/pubmed/15579386">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15579386">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B49" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B49','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B50">
            <p><a name="B50"></a> Brandley MC,  Schmitz A,  Reeder TW: <strong> Partitioned Bayesian analyses, partition choice, and the phylogenetic relationships
                  of scincid lizards. </strong><p><em>Syst Biol</em> 2005,  <strong>54</strong><strong>:</strong>373-390.  <a target="_blank" href="/pubmed/16012105">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16012105">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B50" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B50','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B51">
            <p><a name="B51"></a> Whiting AS,  Bauer AM,  Sites JW: <strong> Phylogenetic relationships and limb loss in sub-Saharan African scincine lizards (Squamata:
                  Scincidae). </strong><p><em>Mol Phylogenet Evol</em> 2003,  <strong>29</strong><strong>:</strong>582-598.  <a target="_blank" href="/pubmed/14615195">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14615195">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B51" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B51','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B52">
            <p><a name="B52"></a> Skinner A,  Lee MSY,  Hutchinson MN: <strong> Rapid and repeated limb loss in a clade of scincid lizards. </strong><p><em>BMC Evol Biol</em> 2008,  <strong>8</strong><strong>:</strong>310.  <a target="_blank" href="/pubmed/19014443">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2148-8-310">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19014443">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B52" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B52','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B53">
            <p><a name="B53"></a> Gamble T,  Colli GR,  Rodrigues MT,  Werneck FP,  Simons AM: <strong> Phylogeny and cryptic diversity in geckos (<em>Phyllopezus</em>; Phyllodactylidae; Gekkota) from South America's open biomes. </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>62</strong><strong>:</strong>943-953.  <a target="_blank" href="/pubmed/22182991">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22182991">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B53" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B53','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B54">
            <p><a name="B54"></a> Gamble T,  Daza JD,  Colli GR,  Vitt LJ,  Bauer AM: <strong> A new genus of miniaturized and pug-nosed gecko from South America (Sphaerodactylidae:
                  Gekkota). </strong><p><em>Zool J Linn Soc</em> 2011,  <strong>163</strong><strong>:</strong>1244-1266.  <a target="_blank" href="/pubmed/22125341">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2011.00741.x">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22125341">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B54" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B54','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B55">
            <p><a name="B55"></a> Gamble T,  Bauer AM,  Colli GR,  Greenbaum E,  Jackman TR,  Vitt LJ,  Simons AM: <strong> Coming to America: multiple origins of New World geckos. </strong><p><em>J Evol Biol</em> 2011,  <strong>24</strong><strong>:</strong>231-244.  <a target="_blank" href="/pubmed/21126276">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21126276">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21126276">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B55" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B55','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B56">
            <p><a name="B56"></a> Gamble T,  Bauer AM,  Greenbaum W,  Jackman TR: <strong> Out of the blue: a novel, trans-Atlantic clade of geckos (Gekkota, Squamata). </strong><p><em>Zool Scripta</em> 2008,  <strong>37</strong><strong>:</strong>355-366.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1463-6409.2008.00330.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B56" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B56','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B57">
            <p><a name="B57"></a> Oliver PM,  Bauer AM,  Greenbaum E,  Jackman T,  Hobbie T: <strong> Molecular phylogenetics of the arboreal Australian gecko genus <em>Oedura</em> Gray 1842 (Gekkota: Diplodactylidae): another plesiomorphic grade? </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>63</strong><strong>:</strong>255-264.  <a target="_blank" href="/pubmed/22209860">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22209860">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B57" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B57','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B58">
            <p><a name="B58"></a> Nielsen SV,  Bauer AM,  Jackman TR,  Hitchmough RA,  Daugherty CH: <strong> New Zealand geckos (Diplodactylidae): cryptic diversity in a post-Gondwanan lineage
                  with trans-Tasman affinities. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>59</strong><strong>:</strong>1-22.  <a target="_blank" href="/pubmed/21184833">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21184833">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B58" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B58','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B59">
            <p><a name="B59"></a> Gamble T,  Greenbaum E,  Jackman TR,  Russell AP,  Bauer AM: <strong> Repeated origin and loss of adhesive toepads in geckos. </strong><p><em>PLoS One</em> 2012,  <strong>7</strong><strong>:</strong>e39429.  <a target="_blank" href="/pubmed/22761794">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22761794">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22761794">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B59" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B59','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B60">
            <p><a name="B60"></a> Wood PL Jr,  Heinicke MP,  Jackman TR,  Bauer AM: <strong> Phylogeny of bent-toed geckos (<em>Cyrtodactylus</em>) reveals a west to east pattern of diversification. </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>65</strong><strong>:</strong>992-1003.  <a target="_blank" href="/pubmed/22982633">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22982633">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B60" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B60','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B61">
            <p><a name="B61"></a> Giugliano LG,  Collevatti RG,  Colli GR: <strong> Molecular dating and phylogenetic relationships among Teiidae (Squamata) inferred
                  by molecular and morphological data. </strong><p><em>Mol Phylogenet Evol</em> 2007,  <strong>45</strong><strong>:</strong>168-179.  <a target="_blank" href="/pubmed/17618129">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17618129">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B61" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B61','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B62">
            <p><a name="B62"></a> Reeder TW,  Cole CJ,  Dessauer HC: <strong> Phylogenetic relationships of whiptail lizards of the genus <em>Cnemidophorus</em> (Squamata: Teiidae): a test of monophyly, reevalution of karyotypic evolution, and
                  review of hybrid origins. </strong><p><em>Am Mus Novit</em> 2002,  <strong>3365</strong><strong>:</strong>1-61.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B62" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B62','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B63">
            <p><a name="B63"></a> Pellegrino KCM,  Rodrigues MT,  Yonenaga-Yassuda Y,  Sites JW: <strong> A molecular perspective on the evolution of microteiid lizards (Squamata, Gymnophthalmidae),
                  and a new classification for the family. </strong><p><em>Biol J Linn Soc</em> 2001,  <strong>74</strong><strong>:</strong>315-338.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B63" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B63','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B64">
            <p><a name="B64"></a> Castoe TA,  Doan TM,  Parkinson CL: <strong> Data partitions and complex models in Bayesian analysis: the phylogeny of gymnophthalmid
                  lizards. </strong><p><em>Syst Biol</em> 2004,  <strong>53</strong><strong>:</strong>448-469.  <a target="_blank" href="/pubmed/15503673">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15503673">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B64" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B64','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B65">
            <p><a name="B65"></a> Pavlicev M,  Mayer W: <strong> Fast radiation of the subfamily Lacertinae (Reptilia: Lacertidae): history or methodical
                  artefact? </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>52</strong><strong>:</strong>727-734.  <a target="_blank" href="/pubmed/19427911">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19427911">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B65" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B65','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B66">
            <p><a name="B66"></a> Mayer W,  Pavilcev M: <strong> The phylogeny of the family Lacertidae (Reptilia) based on nuclear DNA sequences:
                  convergent adaptations to arid habitats within the subfamily Eremiainae. </strong><p><em>Mol Phylogenet Evol</em> 2007,  <strong>44</strong><strong>:</strong>1155-1163.  <a target="_blank" href="/pubmed/17616472">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17616472">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B66" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B66','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B67">
            <p><a name="B67"></a> Fu JZ: <strong> Toward the phylogeny of the family Lacertidae: why 4708 base pairs of mtDNA sequences
                  cannot draw the picture. </strong><p><em>Biol J Linn Soc</em> 2000,  <strong>71</strong><strong>:</strong>203-217.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B67" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B67','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B68">
            <p><a name="B68"></a> Arnold EN,  Arribas O,  Carranza S: <strong> Systematics of the Palaearctic and Oriental lizard tribe Lacertini (Squamata: Lacertidae:
                  Lacertinae), with descriptions of eight new genera. </strong><p><em>Zootaxa</em> 2007,  <strong>1430</strong><strong>:</strong>1-86.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B68" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B68','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B69">
            <p><a name="B69"></a> Greenbaum E,  Villanueva CO,  Kusamba C,  Aristote MM,  Branch WR: <strong> A molecular phylogeny of equatorial African Lacertidae, with the description of a
                  new genus and species from eastern Democratic Republic of the Congo. </strong><p><em>Zool J Linn Soc</em> 2011,  <strong>163</strong><strong>:</strong>913-942.  <a target="_blank" href="/pubmed/22121299">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2011.00732.x">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22121299">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B69" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B69','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B70">
            <p><a name="B70"></a> Mott T,  Vieites DR: <strong> Molecular phylogenetics reveals extreme morphological homoplasy in Brazilian worm
                  lizards challenging current taxonomy. </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>51</strong><strong>:</strong>190-200.  <a target="_blank" href="/pubmed/19405188">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1016/j.ympev.2009.01.014">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B70" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B70','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B71">
            <p><a name="B71"></a> Kearney M,  Stuart BL: <strong> Repeated evolution of limblessness and digging heads in worm lizards revealed by DNA
                  from old bones. </strong><p><em>Proc R Soc B</em> 2004,  <strong>271</strong><strong>:</strong>1677-1683.  <a target="_blank" href="/pubmed/15306287">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15306287">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15306287">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B71" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B71','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B72">
            <p><a name="B72"></a> Driskell AC,  Ane C,  Burleigh JG,  McMahon MM,  O'Meara BC,  Sanderson MJ: <strong> Prospects for building the tree of life from large sequence databases. </strong><p><em>Science</em> 2004,  <strong>306</strong><strong>:</strong>1172-1174.  <a target="_blank" href="/pubmed/15539599">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15539599">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B72" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B72','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B73">
            <p><a name="B73"></a> Wiens JJ,  Fetzner JW,  Parkinson CL,  Reeder TW: <strong> Hylid frog phylogeny and sampling strategies for speciose clades. </strong><p><em>Syst Biol</em> 2005,  <strong>54</strong><strong>:</strong>719-748.  <a target="_blank" href="/pubmed/16243759">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16243759">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B73" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B73','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B74">
            <p><a name="B74"></a> de Queiroz A,  Gatesy J: <strong> The supermatrix approach to systematics. </strong><p><em>Trends Ecol Evol</em> 2007,  <strong>22</strong><strong>:</strong>34-41.  <a target="_blank" href="/pubmed/17046100">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17046100">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B74" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B74','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B75">
            <p><a name="B75"></a> Thomson RC,  Shaffer HB: <strong> Sparse supermatrices for phylogenetic inference: taxonomy, alignment, rogue taxa,
                  and the phylogeny of living turtles. </strong><p><em>Syst Biol</em> 2010,  <strong>59</strong><strong>:</strong>42-58.  <a target="_blank" href="/pubmed/20525619">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20525619">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B75" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B75','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B76">
            <p><a name="B76"></a> Pyron RA,  Wiens JJ: <strong> A large-scale phylogeny of Amphibia including over 2,800 species, and a revised classification
                  of extant frogs, salamanders, and caecilians. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>61</strong><strong>:</strong>543-583.  <a target="_blank" href="/pubmed/21723399">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21723399">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B76" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B76','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B77">
            <p><a name="B77"></a> Hinchcliff CE,  Roalson EH: <strong> Using supermatrices for phylogenetic inquiry: an example using the sedges. </strong><p><em>Syst Biol</em> 2013,  <strong>62</strong><strong>:</strong>205-219.  <a target="_blank" href="/pubmed/23103590">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23103590">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B77" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B77','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B78">
            <p><a name="B78"></a> Guo P,  Liu Q,  Xu Y,  Jiang K,  Hou M,  Ding L,  Pyron RA,  Burbrink FT: <strong> Out of Asia: natricine snakes support the Cenozoic Beringian Dispersal Hypothesis. </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>63</strong><strong>:</strong>825-833.  <a target="_blank" href="/pubmed/22430561">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22430561">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B78" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B78','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B79">
            <p><a name="B79"></a> Pyron RA,  Burbrink FT: <strong> Neogene diversification and taxonomic stability in the snake tribe Lampropeltini (Serpentes:
                  Colubridae). </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>52</strong><strong>:</strong>524-529.  <a target="_blank" href="/pubmed/19236930">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19236930">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B79" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B79','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B80">
            <p><a name="B80"></a> Burbrink FT,  Lawson R: <strong> How and when did Old World ratsnakes disperse into the New World? </strong><p><em>Mol Phylogenet Evol</em> 2007,  <strong>43</strong><strong>:</strong>173-189.  <a target="_blank" href="/pubmed/17113316">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17113316">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B80" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B80','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B81">
            <p><a name="B81"></a> Lawson R,  Slowinski JB,  Crother BI,  Burbrink FT: <strong> Phylogeny of the Colubroidea (Serpentes): new evidence from mitochondrial and nuclear
                  genes. </strong><p><em>Mol Phylogenet Evol</em> 2005,  <strong>37</strong><strong>:</strong>581-601.  <a target="_blank" href="/pubmed/16172004">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16172004">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B81" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B81','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B82">
            <p><a name="B82"></a> Wiens JJ,  Kuczynski CA,  Arif S,  Reeder TW: <strong> Phylogenetic relationships of phrynosomatid lizards based on nuclear and mitochondrial
                  data, and a revised phylogeny for <em>Sceloporus</em>. </strong><p><em>Mol Phylogenet Evol</em> 2010,  <strong>54</strong><strong>:</strong>150-161.  <a target="_blank" href="/pubmed/19751839">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19751839">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B82" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B82','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B83">
            <p><a name="B83"></a> Lemmon AR,  Brown JM,  Stanger-Hall K,  Lemmon EM: <strong> The effect of ambiguous data on phylogenetic estimates obtained by maximum likelihood
                  and Bayesian inference. </strong><p><em>Syst Biol</em> 2009,  <strong>58</strong><strong>:</strong>130-145.  <a target="_blank" href="/pubmed/20525573">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20525573">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B83" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B83','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B84">
            <p><a name="B84"></a> Roure B,  Baurain D,  Philippe H: <strong> Impact of missing data on phylogenies inferred from empirical phylogenomic datasets. </strong><p><em>Mol Biol Evol</em> 2013,  <strong>30</strong><strong>:</strong>197-214.  <a target="_blank" href="/pubmed/22930702">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22930702">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B84" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B84','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B85">
            <p><a name="B85"></a> Wiens JJ,  Morrill MC: <strong> Missing data in phylogenetic analysis: reconciling results from simulations and empirical
                  data. </strong><p><em>Syst Biol</em> 2011,  <strong>60</strong><strong>:</strong>719-731.  <a target="_blank" href="/pubmed/21447483">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21447483">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B85" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B85','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B86">
            <p><a name="B86"></a> Pyron RA: <strong> Divergence time estimation using fossils as terminal taxa and the origins of Lissamphibia. </strong><p><em>Syst Biol</em> 2011,  <strong>60</strong><strong>:</strong>466-481.  <a target="_blank" href="/pubmed/21540408">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21540408">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B86" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B86','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B87">
            <p><a name="B87"></a> Wiens JJ,  Tiu J: <strong> Highly incomplete taxa can rescue phylogenetic analyses from the negative impacts
                  of limited taxon sampling. </strong><p><em>PLoS One</em> 2012,  <strong>7</strong><strong>:</strong>e42925.  <a target="_blank" href="/pubmed/22900065">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22900065">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22900065">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B87" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B87','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B88">
            <p><a name="B88"></a> Scanlon JD,  Lee MSY: <strong> The Pleistocene serpent <em>Wonambi</em> and the early evolution of snakes. </strong><p><em>Nature</em> 2000,  <strong>403</strong><strong>:</strong>416-420.  <a target="_blank" href="/pubmed/10667791">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10667791">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B88" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B88','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B89">
            <p><a name="B89"></a> Tchernov E,  Rieppel O,  Zaher H,  Polcyn MJ,  Jacobs LL: <strong> A fossil snake with limbs. </strong><p><em>Science</em> 2000,  <strong>287</strong><strong>:</strong>2010-2012.  <a target="_blank" href="/pubmed/10720326">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10720326">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B89" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B89','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B90">
            <p><a name="B90"></a> Rage J-C: <em>Serpentes</em>.  Stuttgart: Gustav Fischer Verlag;  1984. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B90" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B90','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B91">
            <p><a name="B91"></a> Estes R: <em>Sauria terrestria, Amphisbaenia</em>.  Stuttgart: Gustav Fischer Verlag;  1983. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B91" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B91','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B92">
            <p><a name="B92"></a> Holman JA: <em>Fossil Snakes of North America: Origin, Evolution, Distribution, Paleoecology</em>.  Bloomington: Indiana University Press;  2000. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B92" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B92','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B93">
            <p><a name="B93"></a> Rage J-C: <strong> Fossil history. </strong>In <em>Snakes: Ecology and Evolutionary Biology</em>.  Edited by Seigel RA, Collins JT, Novak SS.  New York: McMillan;  1987:<strong>:</strong>51-76.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B93" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B93','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B94">
            <p><a name="B94"></a> Anisimova M,  Gascuel O: <strong> Approximate likelihood-ratio test for branches: a fast, accurate, and powerful alternative. </strong><p><em>Syst Biol</em> 2006,  <strong>55</strong><strong>:</strong>539-552.  <a target="_blank" href="/pubmed/16785212">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16785212">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B94" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B94','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B95">
            <p><a name="B95"></a> Lee MSY: <strong> Squamate phylogeny, taxon sampling, and data congruence. </strong><p><em>Org Div Evol</em> 2005,  <strong>5</strong><strong>:</strong>25-45.   <a target="_blank" href="http://dx.doi.org/10.1016/j.ode.2004.05.003">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B95" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B95','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B96">
            <p><a name="B96"></a> Townsend TM,  Leavitt DH,  Reeder TW: <strong> Intercontinental dispersal by a microendemic burrowing reptile (Dibamidae). </strong><p><em>Proc R Soc B</em> 2011,  <strong>278</strong><strong>:</strong>2568-2574.  <a target="_blank" href="/pubmed/21270029">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21270029">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21270029">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B96" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B96','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B97">
            <p><a name="B97"></a> Kluge AG: <strong> Cladistic relationships in the Gekkonoidea (Squamata, Sauria). </strong><p><em>Misc Pub Mus Zool Univ Mich</em> 1987,  <strong>173</strong><strong>:</strong>1-54.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B97" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B97','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B98">
            <p><a name="B98"></a> Gamble T,  Bauer AM,  Greenbaum E,  Jackman TR: <strong> Evidence for Gondwanan vicariance in an ancient clade of gecko lizards. </strong><p><em>J Biogeogr</em> 2008,  <strong>35</strong><strong>:</strong>88-104.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B98" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B98','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B99">
            <p><a name="B99"></a> Melville J,  Schulte JA,  Larson A: <strong> A molecular study of phylogenetic relationships and evolution of antipredator strategies
                  in Australian <em>Diplodactylus</em> geckos, subgenus <em>Strophurus</em>. </strong><p><em>Biol J Linn Soc</em> 2004,  <strong>82</strong><strong>:</strong>123-138.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.2004.00324.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B99" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B99','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B100">
            <p><a name="B100"></a> Jennings WB,  Pianka ER,  Donnellan S: <strong> Systematics of the lizard family Pygopodidae with implications for the diversification
                  of Australian temperate biotas. </strong><p><em>Syst Biol</em> 2003,  <strong>52</strong><strong>:</strong>757-780.  <a target="_blank" href="/pubmed/14668116">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=14668116">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B100" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B100','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B101">
            <p><a name="B101"></a> Bauer AM,  Jackman TR,  Sadlier RA,  Whitaker AH: <strong> Revision of the giant geckos of New Caledonia (Reptilia: Diplodactylidae: <em>Rhacodactylus</em>). </strong><p><em>Zootaxa</em> 2012,  <strong>3404</strong><strong>:</strong>1-52.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B101" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B101','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B102">
            <p><a name="B102"></a> Brown RM,  Siler CD,  Das I,  Min Y: <strong> Testing the phylogenetic affinities of Southeast Asia's rarest geckos: flap-legged
                  geckos (<em>Luperosaurus</em>), flying geckos (<em>Ptychozoon</em>) and their relationship to the pan-Asian genus <em>Gekko</em>. </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>63</strong><strong>:</strong>915-921.  <a target="_blank" href="/pubmed/22425705">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22425705">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B102" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B102','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B103">
            <p><a name="B103"></a> Vicario S,  Caccone A,  Gauthier J: <strong> Xantusiid "night" lizards: a puzzling phylogenetic problem revisited using likelihood-based
                  Bayesian methods on mtDNA sequences. </strong><p><em>Mol Phylogenet Evol</em> 2003,  <strong>26</strong><strong>:</strong>243-261.  <a target="_blank" href="/pubmed/12565035">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12565035">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B103" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B103','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B104">
            <p><a name="B104"></a> Hedges SB,  Bezy RL,  Maxson LR: <strong> Phylogenetic relationships and biogeography of xantusiid lizards, inferred from mitochondrial
                  DNA sequences. </strong><p><em>Mol Biol Evol</em> 1991,  <strong>8</strong><strong>:</strong>767-780.  <a target="_blank" href="/pubmed/1775064">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=1775064">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B104" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B104','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B105">
            <p><a name="B105"></a> Smith HM: <strong> The name of the non-nominotypical subfamily of the lizard family Xantusiidae. </strong><p><em>Syst Zool</em> 1987,  <strong>36</strong><strong>:</strong>326-328.   <a target="_blank" href="http://dx.doi.org/10.2307/2413069">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B105" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B105','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B106">
            <p><a name="B106"></a> Savage JM: <em>The Amphibians and Reptiles of Costa Rica: a Herpetofauna between Two Continents,
                  between Two Seas</em>.  Chicago: University of Chicago Press;  2002. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B106" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B106','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B107">
            <p><a name="B107"></a> Crother BI,  Miyamoto MM,  Presch WF: <strong> Phylogeny and biogeography of the lizard family Xantusiidae. </strong><p><em>Syst Zool</em> 1986,  <strong>35</strong><strong>:</strong>37-45.   <a target="_blank" href="http://dx.doi.org/10.2307/2413289">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B107" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B107','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B108">
            <p><a name="B108"></a> Stanley EL,  Bauer AM,  Jackman TR,  Branch WR,  Mouton PLFN: <strong> Between a rock and a hard polytomy: rapid radiation in the rupicolous girdled lizards
                  (Squamata: Cordylidae). </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>58</strong><strong>:</strong>53-70.  <a target="_blank" href="/pubmed/20816817">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20816817">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B108" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B108','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B109">
            <p><a name="B109"></a> Frost DR,  Janies D,  Mouton PIN,  Titus TA: <strong> A molecular perspective on the phylogeny of the girdled lizards (Cordylidae, Squamata). </strong><p><em>Am Mus Novit</em> 2001,  <strong>3310</strong><strong>:</strong>1-10.   <a target="_blank" href="http://dx.doi.org/10.1206/0003-0082(2001)310<0001:AMPOTP&gt;2.0.CO;2">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B109" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B109','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B110">
            <p><a name="B110"></a> Greer AE: <strong> A subfamilial classification of scincid lizards. </strong><p><em>Bull Mus Comp Zool</em> 1970,  <strong>139</strong><strong>:</strong>151-183.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B110" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B110','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B111">
            <p><a name="B111"></a> Smith SA,  Sadlier RA,  Bauer AM,  Austin CC,  Jackman T: <strong> Molecular phylogeny of the scincid lizards of New Caledonia and adjacent areas: evidence
                  for a single origin of the endemic skinks of Tasmantis. </strong><p><em>Mol Phylogenet Evol</em> 2007,  <strong>43</strong><strong>:</strong>1151-1166.  <a target="_blank" href="/pubmed/17400482">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17400482">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B111" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B111','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B112">
            <p><a name="B112"></a> Hedges SB,  Conn CE: <strong> A new skink fauna from Caribbean islands (Squamata, Mabuyidae, Mabuyinae). </strong><p><em>Zootaxa</em> 2012,  <strong>3288</strong><strong>:</strong>1-244.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B112" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B112','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B113">
            <p><a name="B113"></a> Vences M,  Guayasamin JM,  Miralles A,  de La Riva I: <strong> To name or not to name: criteria to promote economy of change in supraspecific Linnaean
                  classification schemes. </strong><p><em>Zootaxa</em> 2013,  <strong>3636</strong><strong>:</strong>201-244.   <a target="_blank" href="http://dx.doi.org/10.11646/zootaxa.3636.2.1">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B113" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B113','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B114">
            <p><a name="B114"></a> Bauer AM: <strong> On the identity of <em>Lacerta punctata</em> Linnaeus, 1758, the type species of the genus <em>Euprepis</em> Wagler, 1830, and the generic assignment of Afro-Malagasy skinks. </strong><p><em>Afr J Herpetol</em> 2003,  <strong>52</strong><strong>:</strong>1-7.   <a target="_blank" href="http://dx.doi.org/10.1080/21564574.2003.9635472">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B114" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B114','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B115">
            <p><a name="B115"></a> Austin JJ,  Arnold EN: <strong> Using ancient and recent DNA to explore relationships of extinct and endangered <em>Leiolopisma</em> skinks (Reptilia: Scincidae) in the Mascarene islands. </strong><p><em>Mol Phylogenet Evol</em> 2006,  <strong>39</strong><strong>:</strong>503-511.  <a target="_blank" href="/pubmed/16473026">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16473026">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B115" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B115','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B116">
            <p><a name="B116"></a> Skinner A: <strong> Phylogenetic relationships and rate of early diversification of Australian <em>Sphenomorphus</em> group scincids (Scincoidea, Squamata). </strong><p><em>Biol J Linn Soc</em> 2007,  <strong>92</strong><strong>:</strong>347-366.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.2007.00843.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B116" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B116','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B117">
            <p><a name="B117"></a> Linkem CW,  Diesmos AC,  Brown RM: <strong> Molecular systematics of the Philippine forest skinks (Squamata: Scincidae: <em>Sphenomorphus</em>): testing morphological hypotheses of interspecific relationships. </strong><p><em>Zool J Linn Soc</em> 2011,  <strong>163</strong><strong>:</strong>1217-1243.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2011.00747.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B117" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B117','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B118">
            <p><a name="B118"></a> Siler CD,  Diesmos AC,  Alcala AC,  Brown RM: <strong> Phylogeny of Philippine slender skinks (Scincidae: <em>Brachymeles</em>) reveals underestimated species diversity, complex biogeographical relationships,
                  and cryptic patterns of lineage diversification. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>59</strong><strong>:</strong>53-65.  <a target="_blank" href="/pubmed/21255665">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21255665">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B118" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B118','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B119">
            <p><a name="B119"></a> Brandley MC,  Ota H,  Hikida T,  de Oca ANM,  Feria-Ortiz M,  Guo XG,  Wang YZ: <strong> The phylogenetic systematics of blue-tailed skinks (<em>Plestiodon</em>) and the family Scincidae. </strong><p><em>Zool J Linn Soc</em> 2012,  <strong>165</strong><strong>:</strong>163-189.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2011.00801.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B119" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B119','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B120">
            <p><a name="B120"></a> Crottini A,  Dordel J,  Kohler J,  Glaw F,  Schmitz A,  Vences M: <strong> A multilocus phylogeny of Malagasy scincid lizards elucidates the relationships of
                  the fossorial genera <em>Androngo</em> and <em>Cryptoscincus</em>. </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>53</strong><strong>:</strong>345-350.  <a target="_blank" href="/pubmed/19482092">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19482092">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B120" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B120','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B121">
            <p><a name="B121"></a> Sindaco R,  Metallinou M,  Pupin F,  Fasola M,  Carranza S: <strong> Forgotten in the ocean: systematics, biogeography and evolution of the <em>Trachylepis</em> skinks of the Socotra Archipelago. </strong><p><em>Zool Scripta</em> 2012,  <strong>41</strong><strong>:</strong>346-362.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1463-6409.2012.00540.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B121" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B121','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B122">
            <p><a name="B122"></a> Miralles A,  Fuenmayor GR,  Bonillo C,  Schargel WE,  Barros T,  Garcia-Perez JE,  Barrio-Amoros CL: <strong> Molecular systematics of Caribbean skinks of the genus <em>Mabuya</em> (Reptilia, Scincidae), with descriptions of two new species from Venezuela. </strong><p><em>Zool J Linn Soc</em> 2009,  <strong>156</strong><strong>:</strong>598-616.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2008.00487.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B122" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B122','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B123">
            <p><a name="B123"></a> Harvey MB,  Ugueto GN,  Gutberlet RL: <strong> Review of teiid morphology with a revised taxonomy and phylogeny of the Teiidae (Lepidosauria:
                  Squamata). </strong><p><em>Zootaxa</em> 2012,  <strong>3459</strong><strong>:</strong>1-156.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B123" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B123','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B124">
            <p><a name="B124"></a> Doan TM,  Castoe TA: <strong> Phylogenetic taxonomy of the Cercosaurini (Squamata: Gymnophthalmidae), with new genera
                  for species of <em>Neusticurus</em> and <em>Proctoporus</em>. </strong><p><em>Zool J Linn Soc</em> 2005,  <strong>143</strong><strong>:</strong>405-416.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2005.00145.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B124" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B124','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B125">
            <p><a name="B125"></a> Vidal N,  Azvolinsky A,  Cruaud C,  Hedges SB: <strong> Origin of tropical American burrowing reptiles by transatlantic rafting. </strong><p><em>Biol Lett</em> 2008,  <strong>4</strong><strong>:</strong>115-118.  <a target="_blank" href="/pubmed/18077239">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18077239">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=18077239">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B125" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B125','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B126">
            <p><a name="B126"></a> Vidal N,  Hedges SB: <strong> The molecular evolutionary tree of lizards, snakes, and amphisbaenians. </strong><p><em>CR Biol</em> 2009,  <strong>332</strong><strong>:</strong>129-139.   <a target="_blank" href="http://dx.doi.org/10.1016/j.crvi.2008.07.010">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B126" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B126','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B127">
            <p><a name="B127"></a> Lee MSY: <strong> Hidden support from unpromising data sets strongly unites snakes with anguimorph 'lizards'. </strong><p><em>J Evol Biol</em> 2009,  <strong>22</strong><strong>:</strong>1308-1316.  <a target="_blank" href="/pubmed/19490385">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19490385">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B127" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B127','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B128">
            <p><a name="B128"></a> Hugall AF,  Foster R,  Lee MSY: <strong> Calibration choice, rate smoothing, and the pattern of tetrapod diversification according
                  to the long nuclear gene RAG-1. </strong><p><em>Syst Biol</em> 2007,  <strong>56</strong><strong>:</strong>543-563.  <a target="_blank" href="/pubmed/17654361">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17654361">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B128" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B128','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B129">
            <p><a name="B129"></a> Macey JR,  Schulte JA,  Larson A,  Tuniyev BS,  Orlov N,  Papenfuss TJ: <strong> Molecular phylogenetics, tRNA evolution, and historical biogeography in anguid lizards
                  and related taxonomic families. </strong><p><em>Mol Phylogenet Evol</em> 1999,  <strong>12</strong><strong>:</strong>250-272.  <a target="_blank" href="/pubmed/10413621">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10413621">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B129" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B129','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B130">
            <p><a name="B130"></a> Conrad JL,  Ast JC,  Montanari S,  Norell MA: <strong> A combined evidence phylogenetic analysis of Anguimorpha (Reptilia: Squamata). </strong><p><em>Cladistics</em> 2011,  <strong>27</strong><strong>:</strong>230-277.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-0031.2010.00330.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B130" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B130','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B131">
            <p><a name="B131"></a> Collar DC,  Schulte JA,  Losos JB: <strong> Evolution of extreme body size disparity in monitor lizards (<em>Varanus</em>). </strong><p><em>Evolution</em> 2011,  <strong>65</strong><strong>:</strong>2664-2680.  <a target="_blank" href="/pubmed/21884063">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1558-5646.2011.01335.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B131" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B131','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B132">
            <p><a name="B132"></a> Chippindale PT,  Ammerman LK,  Campbell JA: <strong> Molecular approaches to phylogeny of <em>Abronia</em> (Anguidae: Gerrhonotinae), with emphasis on relationships in subgenus <em>Auriculabronia</em>. </strong><p><em>Copeia</em> 1998,  <strong>4</strong><strong>:</strong>883-892.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B132" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B132','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B133">
            <p><a name="B133"></a> Conroy CJ,  Bryson RW,  Lazcano D,  Knight A: <strong> Phylogenetic placement of the pygmy alligator lizard based on mitochondrial DNA. </strong><p><em>J Herp</em> 2005,  <strong>39</strong><strong>:</strong>142-147.   <a target="_blank" href="http://dx.doi.org/10.1670/0022-1511(2005)039[0142:PPOTPA]2.0.CO;2">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B133" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B133','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B134">
            <p><a name="B134"></a> Okajima Y,  Kumazawa Y: <strong> Mitochondrial genomes of acrodont lizards: timing of gene rearrangements and phylogenetic
                  and biogeographic implications. </strong><p><em>BMC Evol Biol</em> 2010,  <strong>10</strong><strong>:</strong>141.  <a target="_blank" href="/pubmed/20465814">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2148-10-141">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20465814">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B134" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B134','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B135">
            <p><a name="B135"></a> Castoe TA,  de Koning APJ,  Kim HM,  Gu WJ,  Noonan BP,  Naylor G,  Jiang ZJ,  Parkinson CL,  Pollock DD: <strong> Evidence for an ancient adaptive episode of convergent molecular evolution. </strong><p><em>Proc Natl Acad Sci USA</em> 2009,  <strong>106</strong><strong>:</strong>8986-8991.  <a target="_blank" href="/pubmed/19416880">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19416880">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19416880">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B135" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B135','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B136">
            <p><a name="B136"></a> Raxworthy CJ,  Forstner MRJ,  Nussbaum RA: <strong> Chameleon radiation by oceanic dispersal. </strong><p><em>Nature</em> 2002,  <strong>415</strong><strong>:</strong>784-787.  <a target="_blank" href="/pubmed/11845207">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11845207">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B136" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B136','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B137">
            <p><a name="B137"></a> Townsend TM,  Vieites DR,  Glaw F,  Vences M: <strong> Testing species-level diversification hypotheses in Madagascar: the case of microendemic
                  <em>Brookesia</em> leaf chameleons. </strong><p><em>Syst Biol</em> 2009,  <strong>58</strong><strong>:</strong>641-656.  <a target="_blank" href="/pubmed/20525615">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20525615">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B137" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B137','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B138">
            <p><a name="B138"></a> Townsend TM,  Tolley KA,  Glaw F,  Bohme W,  Vences M: <strong> Eastward from Africa: palaeocurrent-mediated chameleon dispersal to the Seychelles
                  islands. </strong><p><em>Biol Lett</em> 2011,  <strong>7</strong><strong>:</strong>225-228.  <a target="_blank" href="/pubmed/20826471">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20826471">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=20826471">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B138" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B138','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B139">
            <p><a name="B139"></a> Tolley KA,  Townsend TM,  Vences M: <strong> Large-scale phylogeny of chameleons suggests African origins and Eocene diversification. </strong><p><em>Proc R Soc B</em> 2013,  <strong>280</strong><strong>:</strong>20130184.  <a target="_blank" href="/pubmed/23536596">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23536596">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B139" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B139','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B140">
            <p><a name="B140"></a> Honda M,  Ota H,  Kobayashi M,  Nabhitabhata J,  Yong H-S,  Sengoku S,  Hikida T: <strong> Phylogenetic relationships of the family Agamidae (Reptilia: Iguania) inferred from
                  mitochondrial DNA sequences. </strong><p><em>Zool Sci</em> 2000,  <strong>17</strong><strong>:</strong>527-537.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B140" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B140','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B141">
            <p><a name="B141"></a> Hugall AF,  Foster R,  Hutchinson M,  Lee MSY: <strong> Phylogeny of Australasian agamid lizards based on nuclear and mitochondrial genes:
                  implications for morphological evolution and biogeography. </strong><p><em>Biol J Linn Soc</em> 2008,  <strong>93</strong><strong>:</strong>343-358.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.2007.00911.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B141" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B141','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B142">
            <p><a name="B142"></a> Honda M,  Ota H,  Sengoku S,  Yong HS,  Hikida T: <strong> Molecular evaluation of phylogenetic significances in the highly divergent karyotypes
                  of the genus <em>Gonocephalus</em> (Reptilia: Agamidae) from tropical Asia. </strong><p><em>Zool Sci</em> 2002,  <strong>19</strong><strong>:</strong>129-133.  <a target="_blank" href="/pubmed/12025399">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.2108/zsj.19.129">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B142" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B142','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B143">
            <p><a name="B143"></a> Baig KJ,  Wagner P,  Ananjeva NB,  Bohme W: <strong> A morphology-based taxonomic revision of <em>Laudakia</em> Gray, 1845 (Squamata: Agamidae). </strong><p><em>Vertebr Zool</em> 2012,  <strong>62</strong><strong>:</strong>213-260.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B143" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B143','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B144">
            <p><a name="B144"></a> Wilms TM,  Bohme W,  Wagner P,  Lutzmann N,  Schmitz A: <strong> On the phylogeny and taxonomy of the genus <em>Uromastyx</em> Merrem, 1820 (Reptilia: Squamata: Agamidae: Uromastycinae): resurrection of the genus
                  <em>Saara</em> Gray, 1845. </strong><p><em>Bonner zoologische BeitrÃ¤ge</em> 2009,  <strong>56</strong><strong>:</strong>55-99.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B144" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B144','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B145">
            <p><a name="B145"></a> Wagner P,  Melville J,  Wilms TM,  Schmitz A: <strong> Opening a box of cryptic taxa: the first review of the North African desert lizards
                  in the <em>Trapelus mutabilis</em> Merrem, 1820 complex (Squamata: Agamidae) with descriptions of new taxa. </strong><p><em>Zool J Linn Soc</em> 2011,  <strong>163</strong><strong>:</strong>884-912.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2011.00726.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B145" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B145','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B146">
            <p><a name="B146"></a> Etheridge R,  de Queiroz K: <strong> A phylogeny of Iguanidae. </strong>In <em>Phylogenetic Relationships of the Lizard Families: Essays Commemorating Charles L
                  Camp</em>.  Edited by Estes RD, Pregill GK.  Stanford: Stanford University Press;  1988:<strong>:</strong>283-367.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B146" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B146','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B147">
            <p><a name="B147"></a> Frost DR,  Etheridge R: <strong> A phylogenetic analysis and taxonomy of iguanian lizards (Reptilia: Squamata). </strong><p><em>Misc Publ Univ Kansas</em> 1989,  <strong>81</strong><strong>:</strong>1-65.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B147" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B147','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B148">
            <p><a name="B148"></a> Macey JR,  Larson A,  Ananjeva NB,  Papenfuss TJ: <strong> Evolutionary shifts in three major structural features of the mitochondrial genome
                  among iguanian lizards. </strong><p><em>J Mol Evol</em> 1997,  <strong>44</strong><strong>:</strong>660-674.  <a target="_blank" href="/pubmed/9169559">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9169559">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B148" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B148','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B149">
            <p><a name="B149"></a> Noonan BP,  Chippindale PT: <strong> Vicariant origin of Malagasy reptiles supports late Cretaceous antarctic land bridge. </strong><p><em>Am Nat</em> 2006,  <strong>168</strong><strong>:</strong>730-741.  <a target="_blank" href="/pubmed/17109316">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17109316">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B149" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B149','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B150">
            <p><a name="B150"></a> Noonan BP,  Sites JW: <strong> Tracing the origins of iguanid lizards and boine snakes of the Pacific. </strong><p><em>Am Nat</em> 2010,  <strong>175</strong><strong>:</strong>61-72.  <a target="_blank" href="/pubmed/19929634">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19929634">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B150" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B150','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B151">
            <p><a name="B151"></a> Schulte JA,  Cartwright EM: <strong> Phylogenetic relationships among iguanian lizards using alternative partitioning methods
                  and TSHZ1: a new phylogenetic marker for reptiles. </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>50</strong><strong>:</strong>391-396.  <a target="_blank" href="/pubmed/19026752">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19026752">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B151" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B151','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B152">
            <p><a name="B152"></a> Nicholson KE,  Crother BI,  Guyer C,  Savage JM: <strong> It is time for a new classification of anoles (Squamata: Dactyloidae). </strong><p><em>Zootaxa</em> 2012,  <strong>3477</strong><strong>:</strong>1-108.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B152" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B152','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B153">
            <p><a name="B153"></a> Poe S: <strong> Phylogeny of anoles. </strong><p><em>Herp Monogr</em> 2004,  <strong>18</strong><strong>:</strong>37-89.   <a target="_blank" href="http://dx.doi.org/10.1655/0733-1347(2004)018[0037:POA]2.0.CO;2">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B153" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B153','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B154">
            <p><a name="B154"></a> Guyer C,  Savage JM: <strong> Cladistic relationships among anoles (Sauria, Iguanidae). </strong><p><em>Syst Zool</em> 1986,  <strong>35</strong><strong>:</strong>509-531.   <a target="_blank" href="http://dx.doi.org/10.2307/2413112">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B154" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B154','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B155">
            <p><a name="B155"></a> Losos JB: <em>Lizards in an Evolutionary Tree: the Ecology of Adaptive Radiation in Anoles</em>.  Berkeley: University of California Press;  2009. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B155" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B155','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B156">
            <p><a name="B156"></a> Cannatella DC,  de Queiroz K: <strong> Phylogenetic systematics of the anoles: is a new taxonomy warranted? </strong><p><em>Syst Zool</em> 1989,  <strong>38</strong><strong>:</strong>57-69.   <a target="_blank" href="http://dx.doi.org/10.2307/2992437">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B156" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B156','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B157">
            <p><a name="B157"></a> Poe S: <strong> 1986 redux: new genera of anoles (Squamata: Dactyloidae) are unwarranted. </strong><p><em>Zootaxa</em> 2013,  <strong>3626</strong><strong>:</strong>295-299.   <a target="_blank" href="http://dx.doi.org/10.11646/zootaxa.3626.2.7">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B157" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B157','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B158">
            <p><a name="B158"></a> Heise PJ,  Maxson LR,  Dowling HG,  Hedges SB: <strong> Higher-level snake phylogeny inferred from mitochondrial DNA sequences of 12S rRNA
                  and 16S rRNA genes. </strong><p><em>Mol Biol Evol</em> 1995,  <strong>12</strong><strong>:</strong>259-265.  <a target="_blank" href="/pubmed/7700153">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=7700153">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B158" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B158','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B159">
            <p><a name="B159"></a> Burbrink FT,  Pyron RA: <strong> The taming of the skew: estimating proper confidence intervals for divergence dates. </strong><p><em>Syst Biol</em> 2008,  <strong>57</strong><strong>:</strong>317-328.  <a target="_blank" href="/pubmed/18432552">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18432552">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B159" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B159','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B160">
            <p><a name="B160"></a> Pyron RA,  Burbrink FT: <strong> Extinction, ecological opportunity, and the origins of global snake diversity. </strong><p><em>Evolution</em> 2012,  <strong>66</strong><strong>:</strong>163-178.  <a target="_blank" href="/pubmed/22220872">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1558-5646.2011.01437.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B160" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B160','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B161">
            <p><a name="B161"></a> Gower DJ,  Vidal N,  Spinks JN,  McCarthy CJ: <strong> The phylogenetic position of Anomochilidae (Reptilia: Serpentes): first evidence from
                  DNA sequences. </strong><p><em>J Zool Syst Evol Res</em> 2005,  <strong>43</strong><strong>:</strong>315-320.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1439-0469.2005.00315.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B161" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B161','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B162">
            <p><a name="B162"></a> Cadle JE,  Dessauer HC,  Gans C,  Gartside DF: <strong> Phylogenetic relationships and molecular evolution in uropeltid snakes (Serpentes,
                  Uropeltidae): allozymes and albumin immunology. </strong><p><em>Biol J Linn Soc</em> 1990,  <strong>40</strong><strong>:</strong>293-320.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.1990.tb00541.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B162" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B162','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B163">
            <p><a name="B163"></a> Wallach V,  Wuster W,  Broadley DG: <strong> In praise of subgenera: taxonomic status of cobras of the genus <em>Naja</em> Laurenti (Serpentes: Elapidae). </strong><p><em>Zootaxa</em> 2009,  <strong>2236</strong><strong>:</strong>26-36.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B163" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B163','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B164">
            <p><a name="B164"></a> Williams D,  Wuster W,  Fry BG: <strong> The good, the bad and the ugly: Australian snake taxonomists and a history of the
                  taxonomy of Australia's venomous snakes. </strong><p><em>Toxicon</em> 2006,  <strong>48</strong><strong>:</strong>919-930.  <a target="_blank" href="/pubmed/16999982">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16999982">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B164" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B164','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B165">
            <p><a name="B165"></a> Rawlings LH,  Rabosky DL,  Donnellan SC,  Hutchinson MN: <strong> Python phylogenetics: inference from morphology and mitochondrial DNA. </strong><p><em>Biol J Linn Soc</em> 2008,  <strong>93</strong><strong>:</strong>603-619.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1095-8312.2007.00904.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B165" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B165','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B166">
            <p><a name="B166"></a> Burbrink FT: <strong> Inferring the phylogenetic position of <em>Boa constrictor</em> among the Boinae. </strong><p><em>Mol Phylogenet Evol</em> 2005,  <strong>34</strong><strong>:</strong>167-180.  <a target="_blank" href="/pubmed/15579390">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15579390">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B166" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B166','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B167">
            <p><a name="B167"></a> Kluge AG: <strong> Boine snake phylogeny and research cycles. </strong><p><em>Misc Publ Mus Zool Univ Mich</em> 1991,  <strong>178</strong><strong>:</strong>1-58.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B167" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B167','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B168">
            <p><a name="B168"></a> Romer AS: <em>Osteology of the Reptiles</em>.  Chicago: University of Chicago Press;  1956. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B168" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B168','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B169">
            <p><a name="B169"></a> Kelly CMR,  Barker NP,  Villet MH: <strong> Phylogenetics of advanced snakes (Caenophidia) based on four mitochondrial genes. </strong><p><em>Syst Biol</em> 2003,  <strong>52</strong><strong>:</strong>439-459.  <a target="_blank" href="/pubmed/12857637">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=12857637">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B169" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B169','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B170">
            <p><a name="B170"></a> Kraus F,  Brown WM: <strong> Phylogenetic relationships of colubroid snakes based on mitochondrial DNA sequences. </strong><p><em>Zool J Linn Soc</em> 1998,  <strong>122</strong><strong>:</strong>455-487.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.1998.tb02159.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B170" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B170','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B171">
            <p><a name="B171"></a> Boulenger GA: <em>Catalogue of snakes in the British Museum</em>.  London: British Museum of Natural History;  1894. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B171" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B171','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B172">
            <p><a name="B172"></a> Guo P,  Wu Y,  He S,  Shi H,  Zhao E: <strong> Systematics and molecular phylogenetics of Asian snail-eating snakes (Pareatidae). </strong><p><em>Zootaxa</em> 2011,  <strong>3001</strong><strong>:</strong>57-64.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B172" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B172','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B173">
            <p><a name="B173"></a> Garrigues T,  Dauga C,  Ferquel E,  Choumet V,  Failloux AB: <strong> Molecular phylogeny of <em>Vipera</em> Laurenti, 1768 and the related genera <em>Macrovipera</em> (Reuss, 1927) and <em>Daboia</em> (Gray, 1842), with comments about neurotoxic <em>Vipera aspis aspis</em> populations. </strong><p><em>Mol Phylogenet Evol</em> 2005,  <strong>35</strong><strong>:</strong>35-47.  <a target="_blank" href="/pubmed/15737580">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15737580">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B173" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B173','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B174">
            <p><a name="B174"></a> Lenk P,  Kalyabina S,  Wink M,  Joger U: <strong> Evolutionary relationships among the true vipers (Reptilia: Viperidae) inferred from
                  mitochondrial DNA sequences. </strong><p><em>Mol Phylogenet Evol</em> 2001,  <strong>19</strong><strong>:</strong>94-104.  <a target="_blank" href="/pubmed/11286494">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11286494">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B174" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B174','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B175">
            <p><a name="B175"></a> Castoe TA,  Parkinson CL: <strong> Bayesian mixed models and the phylogeny of pitvipers (Viperidae: Serpentes). </strong><p><em>Mol Phylogenet Evol</em> 2006,  <strong>39</strong><strong>:</strong>91-110.  <a target="_blank" href="/pubmed/16504544">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16504544">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B175" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B175','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B176">
            <p><a name="B176"></a> Fenwick AM,  Gutberlet RL,  Evans JA,  Parkinson CL: <strong> Morphological and molecular evidence for phylogeny and classification of South American
                  pitvipers, genera <em>Bothrops</em>, <em>Bothriopsis</em>, and <em>Bothrocophias</em> (Serpentes: Viperidae). </strong><p><em>Zool J Linn Soc</em> 2009,  <strong>156</strong><strong>:</strong>617-640.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.2008.00495.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B176" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B176','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B177">
            <p><a name="B177"></a> Kelly CMR,  Branch WR,  Broadley DG,  Barker NP,  Villet MH: <strong> Molecular systematics of the African snake family Lamprophiidae Fitzinger, 1843 (Serpentes:
                  Elapoidea), with particular focus on the genera <em>Lamprophis</em> Fitzinger 1843 and <em>Mehelya</em> Csiki 1903. </strong><p><em>Mol Phylogenet Evol</em> 2011,  <strong>58</strong><strong>:</strong>415-426.  <a target="_blank" href="/pubmed/21095234">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21095234">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B177" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B177','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B178">
            <p><a name="B178"></a> Vidal N,  Branch WR,  Pauwels OSG,  Hedges SB,  Broadley DG,  Wink M,  Cruaud C,  Joger U,  Nagy ZT: <strong> Dissecting the major African snake radiation: a molecular phylogeny of the Lamprophiidae
                  Fitzinger (Serpentes, Caenophidia). </strong><p><em>Zootaxa</em> 2008,  <strong>1945</strong><strong>:</strong>51-66.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B178" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B178','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B179">
            <p><a name="B179"></a> Sanders KL,  Lee MSY,  Leys R,  Foster R,  Keogh JS: <strong> Molecular phylogeny and divergence dates for Australasian elapids and sea snakes (Hydrophiinae):
                  evidence from seven genes for rapid evolutionary radiations. </strong><p><em>J Evol Biol</em> 2008,  <strong>21</strong><strong>:</strong>682-695.  <a target="_blank" href="/pubmed/18384538">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18384538">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B179" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B179','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B180">
            <p><a name="B180"></a> Sanders KL,  Lee MSY: <strong> Molecular evidence for a rapid late-Miocene radiation of Australasian venomous snakes
                  (Elapidae, Colubroidea). </strong><p><em>Mol Phylogenet Evol</em> 2008,  <strong>46</strong><strong>:</strong>1165-1173.  <a target="_blank" href="/pubmed/18222097">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=18222097">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B180" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B180','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B181">
            <p><a name="B181"></a> Chen X,  Huang S,  Guo P,  Colli GR,  de Oca AN M,  Vitt LJ,  Pyron RA,  Burbrink FT: <strong> Understanding the formation of ancient intertropical disjunct distributions using
                  Asian and Neotropical hinged-teeth snakes (<em>Sibynophis</em> and <em>Scaphiodontophis</em>: Serpentes: Colubridae). </strong><p><em>Mol Phylogenet Evol</em> 2012,  <strong>66</strong><strong>:</strong>254-261.  <a target="_blank" href="/pubmed/23044403">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=23044403">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B181" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B181','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B182">
            <p><a name="B182"></a> Huang S,  Liu SY,  Guo P,  Zhang YP,  Zhao EM: <strong> What are the closest relatives of the hot-spring snakes (Colubridae, <em>Thermophis</em>), the relict species endemic to the Tibetan Plateau? </strong><p><em>Mol Phylogenet Evol</em> 2009,  <strong>51</strong><strong>:</strong>438-446.  <a target="_blank" href="/pubmed/19249375">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19249375">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B182" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B182','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B183">
            <p><a name="B183"></a> He M,  Feng JC,  Liu SY,  Guo P,  Zhao EM: <strong> The phylogenetic position of <em>Thermophis</em> (Serpentes: Colubridae), an endemic snake from the Qinghai-Xizang Plateau, China. </strong><p><em>J Nat Hist</em> 2009,  <strong>43</strong><strong>:</strong>479-488.   <a target="_blank" href="http://dx.doi.org/10.1080/00222930802389825">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B183" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B183','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B184">
            <p><a name="B184"></a> Socha JJ: <strong> Gliding flight in <strong><em>Chrysopelea</em></strong>: turning a snake into a wing. </strong><p><em>Integr Comp Biol</em> 2011,  <strong>51</strong><strong>:</strong>969-982.  <a target="_blank" href="/pubmed/21816808">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21816808">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B184" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B184','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B185">
            <p><a name="B185"></a> Helfenberger N: <strong> Phylogenetic relationship of Old World ratsnakes based on visceral organ topography,
                  osteology, and allozyme variation. </strong><p><em>Russ J Herp</em> 2001,  <strong>8</strong><strong>:</strong>1-56.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B185" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B185','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B186">
            <p><a name="B186"></a> Alfaro ME,  Arnold SJ: <strong> Molecular systematics and evolution of <em>Regina</em> and the Thamnophiine snakes. </strong><p><em>Mol Phylogenet Evol</em> 2001,  <strong>21</strong><strong>:</strong>408-423.  <a target="_blank" href="/pubmed/11741383">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11741383">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B186" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B186','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B187">
            <p><a name="B187"></a> de Queiroz A,  Lawson R,  Lemos-Espinal JA: <strong> Phylogenetic relationships of North American garter snakes (<em>Thamnophis</em>) based on four mitochondrial genes: How much DNA sequence is enough? </strong><p><em>Mol Phylogenet Evol</em> 2002,  <strong>22</strong><strong>:</strong>315-329.  <a target="_blank" href="/pubmed/11820851">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11820851">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B187" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B187','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B188">
            <p><a name="B188"></a> Cope ED: <strong> Eleventh contribution to the herpetology of tropical America: DugÃ©s, A. </strong><p><em>Proc Amer Philos Soc</em> 1879,  <strong>18</strong><strong>:</strong>261-277.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B188" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B188','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B189">
            <p><a name="B189"></a> Fitzinger L: <em>Systema Reptilium. Fasciculus Primus, Amblyglossae</em>.  Vienna: Apud BraumÃ¼ller and Seidel Bibliopolas;  1843. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B189" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B189','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B190">
            <p><a name="B190"></a> Vidal N,  Dewynter M,  Gower DJ: <strong> Dissecting the major American snake radiation: a molecular phylogeny of the Dipsadidae
                  Bonaparte (Serpentes, Caenophidia). </strong><p><em>CR Biol</em> 2010,  <strong>333</strong><strong>:</strong>48-55.   <a target="_blank" href="http://dx.doi.org/10.1016/j.crvi.2009.11.003">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B190" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B190','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B191">
            <p><a name="B191"></a> Caldwell MW: <strong> Squamate phylogeny and the relationships of snakes and mosasauroids. </strong><p><em>Zool J Linn Soc</em> 1999,  <strong>125</strong><strong>:</strong>115-147.   <a target="_blank" href="http://dx.doi.org/10.1111/j.1096-3642.1999.tb00587.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B191" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B191','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B192">
            <p><a name="B192"></a> Heath TA,  Hedtke SM,  Hillis DM: <strong> Taxon sampling and the accuracy of phylogenetic analyses. </strong><p><em>J Syst Evol</em> 2008,  <strong>46</strong><strong>:</strong>239-257.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B192" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B192','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B193">
            <p><a name="B193"></a> Hedtke SM,  Townsend TM,  Hillis DM: <strong> Resolution of phylogenetic conflict in large data sets by increased taxon sampling. </strong><p><em>Syst Biol</em> 2006,  <strong>55</strong><strong>:</strong>522-529.  <a target="_blank" href="/pubmed/16861214">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16861214">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B193" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B193','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B194">
            <p><a name="B194"></a> Rannala B,  Huelsenbeck JP,  Yang ZH,  Nielsen R: <strong> Taxon sampling and the accuracy of large phylogenies. </strong><p><em>Syst Biol</em> 1998,  <strong>47</strong><strong>:</strong>702-710.  <a target="_blank" href="/pubmed/12066312">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1080/106351598260680">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B194" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B194','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B195">
            <p><a name="B195"></a> Poe S,  Swofford DL: <strong> Taxon sampling revisited. </strong><p><em>Nature</em> 1999,  <strong>398</strong><strong>:</strong>299-300.  <a target="_blank" href="/pubmed/10192331">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=10192331">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B195" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B195','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B196">
            <p><a name="B196"></a> Fisher-Reid MC,  Wiens JJ: <strong> What are the consequences of combining nuclear and mitochondrial data for phylogenetic
                  analysis? Lessons from <em>Plethodon</em> salamanders and 13 other vertebrate clades. </strong><p><em>BMC Evol Biol</em> 2011,  <strong>11</strong><strong>:</strong>300.  <a target="_blank" href="/pubmed/21995558">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://dx.doi.org/10.1186/1471-2148-11-300">BioMed&nbsp;Central&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21995558">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B196" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B196','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B197">
            <p><a name="B197"></a> McMahon MM,  Sanderson MJ: <strong> Phylogenetic supermatrix analysis of GenBank sequences from 2228 papilionoid legumes. </strong><p><em>Syst Biol</em> 2006,  <strong>55</strong><strong>:</strong>818-836.  <a target="_blank" href="/pubmed/17060202">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17060202">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B197" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B197','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B198">
            <p><a name="B198"></a> Lemmon AR,  Emme SA,  Lemmon EM: <strong> Anchored hybrid enrichment for massively high-throughput phylogenomics. </strong><p><em>Syst Biol</em> 2012,  <strong>61</strong><strong>:</strong>727-744.  <a target="_blank" href="/pubmed/22605266">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22605266">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B198" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B198','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B199">
            <p><a name="B199"></a> Faircloth BC,  McCormack JE,  Crawford NG,  Harvey MG,  Brumfield RT,  Glenn TC: <strong> Ultraconserved elements anchor thousands of genetic markers spanning multiple evolutionary
                  timescales. </strong><p><em>Syst Biol</em> 2012,  <strong>61</strong><strong>:</strong>717-726.  <a target="_blank" href="/pubmed/22232343">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22232343">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B199" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B199','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B200">
            <p><a name="B200"></a> Crawford NG,  Faircloth BC,  McCormack JE,  Brumfield D,  Winker K,  Glenn TC: <strong> More than 1000 ultraconserved elements provide evidence that turtles are the sister
                  group of archosaurs. </strong><p><em>Biol Lett</em> 2012,  <strong>8</strong><strong>:</strong>783-786.  <a target="_blank" href="/pubmed/22593086">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22593086">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B200" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B200','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B201">
            <p><a name="B201"></a> Edwards SV,  Liu L,  Pearl DK: <strong> High-resolution species trees without concatenation. </strong><p><em>Proc Natl Acad Sci USA</em> 2007,  <strong>104</strong><strong>:</strong>5936-5941.  <a target="_blank" href="/pubmed/17392434">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17392434">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=17392434">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B201" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B201','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B202">
            <p><a name="B202"></a> Heled J,  Drummond AJ: <strong> Bayesian inference of species trees from multilocus data. </strong><p><em>Mol Biol Evol</em> 2010,  <strong>27</strong><strong>:</strong>570-580.  <a target="_blank" href="/pubmed/19906793">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=19906793">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=19906793">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B202" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B202','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B203">
            <p><a name="B203"></a> Ricklefs RE,  Losos JB,  Townsend TM: <strong> Evolutionary diversification of clades of squamate reptiles. </strong><p><em>J Evol Biol</em> 2007,  <strong>20</strong><strong>:</strong>1751-1762.  <a target="_blank" href="/pubmed/17714293">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17714293">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B203" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B203','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B204">
            <p><a name="B204"></a> Brandley MC,  Huelsenbeck JP,  Wiens JJ: <strong> Rates and patterns in the evolution of snake-like body form in squamate reptiles:
                  evidence for repeated re-evolution of lost digits and long-term persistence of intermediate
                  body forms. </strong><p><em>Evolution</em> 2008,  <strong>62</strong><strong>:</strong>2042-2064.  <a target="_blank" href="/pubmed/18507743">PubMed&nbsp;Abstract</a> |  <a target="_blank" href="http://dx.doi.org/10.1111/j.1558-5646.2008.00430.x">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B204" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B204','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B205">
            <p><a name="B205"></a> Pyron RA,  Burbrink FT: <strong> Systematics of the Common Kingsnake (<em>Lampropeltis getula</em>; Serpentes: Colubridae) and the burden of heritage in taxonomy. </strong><p><em>Zootaxa</em> 2009,  <strong>2241</strong><strong>:</strong>22-32.  <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B205" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B205','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B206">
            <p><a name="B206"></a> Pyron RA: <strong> A likelihood method for assessing molecular divergence time estimates and the placement
                  of fossil calibrations. </strong><p><em>Syst Biol</em> 2010,  <strong>59</strong><strong>:</strong>185-194.  <a target="_blank" href="/pubmed/20525629">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20525629">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B206" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B206','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B207">
            <p><a name="B207"></a> Edgar RC: <strong> MUSCLE: multiple sequence alignment with high accuracy and high throughput. </strong><p><em>Nucleic Acids Res</em> 2004,  <strong>32</strong><strong>:</strong>1792-1797.  <a target="_blank" href="/pubmed/15034147">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15034147">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15034147">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B207" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B207','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B208">
            <p><a name="B208"></a> Larkin MA,  Blackshields G,  Brown NP,  Chenna R,  McGettigan PA,  McWilliam H,  Valentin F,  Wallace IM,  Wilm A,  Lopez R, <em>et al</em>.: <strong> Clustal W and Clustal X version 2.0. </strong><p><em>Bioinformatics</em> 2007,  <strong>23</strong><strong>:</strong>2947-2948.  <a target="_blank" href="/pubmed/17846036">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17846036">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B208" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B208','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B209">
            <p><a name="B209"></a> Stamatakis A,  Aberer AJ,  Goll C,  Smith SA,  Berger SA,  Izquierdo-Carrasco F: <strong> RAxML-Light: a tool for computing terabyte phylogenies. </strong><p><em>Bioinformatics</em> 2012,  <strong>28</strong><strong>:</strong>2064-2066.  <a target="_blank" href="/pubmed/22628519">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=22628519">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=22628519">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B209" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B209','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B210">
            <p><a name="B210"></a> Stamatakis A: <strong> RAxML-VI-HPC: maximum likelihood-based phylogenetic analyses with thousands of taxa
                  and mixed models. </strong><p><em>Bioinformatics</em> 2006,  <strong>22</strong><strong>:</strong>2688-2690.  <a target="_blank" href="/pubmed/16928733">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16928733">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B210" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B210','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B211">
            <p><a name="B211"></a> Felsenstein J: <em>Inferring Phylogenies</em>.  Sunderland: Sinauer Associates;  2004. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B211" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B211','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B212">
            <p><a name="B212"></a> Anisimova M,  Gil M,  Dufayard JF,  Dessimoz C,  Gascuel O: <strong> Survey of branch support methods demonstrates accuracy, power, and robustness of fast
                  likelihood-based approximation schemes. </strong><p><em>Syst Biol</em> 2011,  <strong>60</strong><strong>:</strong>685-699.  <a target="_blank" href="/pubmed/21540409">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=21540409">Publisher&nbsp;Full&nbsp;Text</a> | <a target="_blank" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=21540409">PubMed&nbsp;Central&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B212" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B212','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B213">
            <p><a name="B213"></a> Guindon S,  Dufayard JF,  Lefort V,  Anisimova M,  Hordijk W,  Gascuel O: <strong> New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the
                  performance of PhyML 3.0. </strong><p><em>Syst Biol</em> 2010,  <strong>59</strong><strong>:</strong>307-321.  <a target="_blank" href="/pubmed/20525638">PubMed&nbsp;Abstract</a> | <a target="_blank" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=20525638">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B213" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B213','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B214">
            <p><a name="B214"></a> Schneider JG: <em>Historiae Amphibiorum Naturalis et Literariae. Fasciculus Secundus continens Crocodilos,
                  Scincos, Chamaesauras, Boas, Pseudoboas, Elapes, Angues, Amphisbaenas et Caecilias</em>.  Jena: Frommani;  1801. <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B214" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B214','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
         <li id="B215">
            <p><a name="B215"></a> McDowell SB: <strong> A catalog of the snakes of New Guinea and the Solomons, with special reference to
                  those in the Bernice P. Bishop museum. Part III. Boinae and Acrochordoidea (Reptilia,
                  Serpentes). </strong><p><em>J Herp</em> 1979,  <strong>13</strong><strong>:</strong>1-92.   <a target="_blank" href="http://dx.doi.org/10.2307/1563761">Publisher&nbsp;Full&nbsp;Text</a> <a href="/sfx_links?ui=1471-2148-13-93&amp;bibl=B215" onclick="popup('/sfx_links?ui=1471-2148-13-93&amp;bibl=B215','SFXMenu','460','420'); return false;"><img src="/sfx_links?getImage" alt="OpenURL" align="absmiddle"></a></p>
               <p class="totext"><script type="text/javascript">totext()</script></p>
            </p>
         </li>
      </ol>
   </div>
</section>
    <br class="clearall">
<div class="article-alert-signup-div">

<a class="close png_bg" id="closeHelpBox" >close</a>	
<div class="article-alert-signup rounded" > 
<span class="blurb left">Sign up to receive new article alerts from <em>BMC Evolutionary Biology</em></span>

	
				<button name="articleAlertPreference.status" type="button" class="w74 right" onClick="window.location='/bmcevolbiol/my/preferences'">Sign up</button>
	
</div>
</div></div>
    </div></div>
</div>


		</div>
</div>
						
		<dl class="google-ad wide ">
	<dt class="hide">
					<a class="banner-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising"></a>
				
			</dt>
		<dd>

		<!-- OAS AD 'Bottom' begin -->
		<script type="text/javascript">
		OAS_AD('Bottom');

		</script>
		<!-- OAS AD 'Bottom' end -->
		</dd>

		</dl>

		<noscript>
			<dl class="google-ad  noscript">  
    <dt class="hide"><a href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</dt>
    
    <dd>
        <a href="http://oas.biomedcentral.com/RealMedia/ads/click_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/11276636203@Bottom?">
            <img src="http://oas.biomedcentral.com/RealMedia/ads/adstream_nx.ads/biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93/16291602864@Bottom?" alt="advert"/>
        </a>
    </dd>
			</dl>
		</noscript>
				    <hr class="hide" />
	
    

						
   
  
      
  
                  
  
                  
  
            
  
            <script>window.bmcIsMobile = "classic";</script>
            
            


<div id="footer">

						
			<span class="views"><a id="mobile-view" href="?fmt_view=mobile">Mobile view</a> | <strong>Desktop view</strong></span>
			
	<div class="content">
    <div class="rounded ">
<div class="wrap-inner ">
        	
    <ul class="desktop">
                                                    
				        <li class="noborder"><a href="http://www.biomedcentral.com/about/tandc">Terms and Conditions</a></li>
<li><a href="http://www.biomedcentral.com/about/privacy">Privacy statement</a></li>
<li><a href="http://www.biomedcentral.com/presscenter">Press</a></li>
	<li><a href="http://www.biomedcentral.com/advertisers">Information for advertisers</a></li>
<li><a href="http://www.biomedcentral.com/about/bmcjobs">Jobs at BMC</a></li>
<li><a href="http://www.biomedcentral.com/support">Support</a></li>
<li><a href="http://www.biomedcentral.com/about/contact">Contact us</a></li>


    
    	    </ul>
    <p id="copyright">&copy; 2014 
	BioMed Central Ltd unless otherwise stated. Part of Springer Science+Business&nbsp;Media.</p>
        </div>
</div>

	</div>

</div>
<div class="springer">
    <a href="http://www.springer.com/" target="blank"><img style="float:right; margin:0;" src="http://www.biomedcentral.com/images/Springer_Branding_footer_logo.png" /></a>
</div>
					    

<script>
// getElementsByClassName Polyfill
var getElementsByClassName=function(e,t,n){if(document.getElementsByClassName){getElementsByClassName=function(e,t,n){n=n||document;var r=n.getElementsByClassName(e),i=t?new RegExp("\\b"+t+"\\b","i"):null,s=[],o;for(var u=0,a=r.length;u<a;u+=1){o=r[u];if(!i||i.test(o.nodeName)){s.push(o)}}return s}}else if(document.evaluate){getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i="",s="http://www.w3.org/1999/xhtml",o=document.documentElement.namespaceURI===s?s:null,u=[],a,f;for(var l=0,c=r.length;l<c;l+=1){i+="[contains(concat(' ', @class, ' '), ' "+r[l]+" ')]"}try{a=document.evaluate(".//"+t+i,n,o,0,null)}catch(h){a=document.evaluate(".//"+t+i,n,null,0,null)}while(f=a.iterateNext()){u.push(f)}return u}}else{getElementsByClassName=function(e,t,n){t=t||"*";n=n||document;var r=e.split(" "),i=[],s=t==="*"&&n.all?n.all:n.getElementsByTagName(t),o,u=[],a;for(var f=0,l=r.length;f<l;f+=1){i.push(new RegExp("(^|\\s)"+r[f]+"(\\s|$)"))}for(var c=0,h=s.length;c<h;c+=1){o=s[c];a=false;for(var p=0,d=i.length;p<d;p+=1){a=i[p].test(o.className);if(!a){break}}if(a){u.push(o)}}return u}}return getElementsByClassName(e,t,n)}
</script>

						



<script type="text/javascript">
if (typeof jQuery == 'undefined') { document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.min.js' type='text/javascript'%3E%3C/script%3E")); }


document.documentElement.id = "js";

var site = {
			abbreviation: "/bmcevolbiol",
	        type: "BMC_CORE",
        portalId: "9001",
                section: "browse",
                page: "fulltext",
    		
									homePortal: "biomedcentral.com",
		
			siteShowAds: "true",
				showAds: "true",
				url: "biomedcentral.com/bmcevolbiol",
			portal: false,
	        adpage_url: "biomedcentral.com/bmcevolbiol/article/10.1186/1471/2148/13/93",
        id: "1006",
	name: "BMC Evolutionary Biology",
	plugins: {
		scrollable: (getElementsByClassName("scrollable").length > 0),
				flowplayer: ((document.getElementById("flowplayer") != null) || (getElementsByClassName("myPlayer").length > 0) || (document.getElementById("videoarticle") != null) || (getElementsByClassName("flowplayer").length > 0) || (document.getElementById("audio") != null) || (document.getElementById("elifeplayer") != null)),
		lightbox: ((document.getElementById("image-highlight") != null) || (getElementsByClassName("lightbox").length > 0)),
		thickbox: (getElementsByClassName("thickbox").length > 0),
		altmetric: ((document.getElementById("altmetric") != null) || (getElementsByClassName("altmetric").length > 0))
	}
	
};



if(typeof quoteSearchUri == "undefined") { quoteSearchUri=null; }
</script>



        <script type="text/javascript" src="http://www.bmcimg.com/javascript/journals/behaviours-0.js"></script>






		<script>
	if (site.plugins.scrollable == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.scrollable-1.0.1.min.js' %3E%3C/script%3E")); 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.tools.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.flowplayer == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/flowplayer-3.2.6.min.js'%3E%3C/script%3E"));
	}

	if (site.plugins.lightbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/browse_imagehighlight.js'%3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/jquery.colorbox.js' %3E%3C/script%3E"));
	}

	if (site.plugins.thickbox == true) { 
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-compressed.js' %3E%3C/script%3E"));
		document.write(decodeURIComponent("%3Cscript src='/javascript/plugins/thickbox-flashhidden-patch.js' %3E%3C/script%3E"));
	}
	</script>





            <script type="text/javascript" src="http://www.bmcimg.com/javascript/plugins/jquery.fancybox-1.3.1.pack-0.js"></script>
            <script type="text/javascript" src="http://www.bmcimg.com/javascript/plugins/jquery.debug-0.js"></script>
                      <script type="text/javascript" src="http://www.bmcimg.com/javascript/email_preferences/email_preferences-0.js"></script>
                <script type="text/javascript" src="http://www.bmcimg.com/javascript/articles/articles-0.js"></script>
		<div id="fb-root"></div>











        <script type="text/javascript" src="/javascript/plugins/jquery.tagcloud.min-0.js"></script>




<!--[if lte IE 6]>
            <script type="text/javascript" src="/javascript/plugins/DD_belatedPNG_0.0.8a-0.js"></script>
    <script type="text/javascript">
        DD_belatedPNG.fix('.png_bg, ul.primary-nav li a, ul.primary-nav li a span, ul.secondary-nav li a, ul.secondary-nav li a span, ul li i, .plus-button, .facebook, .twitter, .fancy-bg, #fancybox-close');
    </script>
	                <link rel="stylesheet" type="text/css" href="http://www.bmcimg.com/css/hacks/ie6-0.css"/>
    <![endif]-->

<!-- Claudia Whitcombe wanted this comment here -->

       <div class="hide">
      <dl class="google-ad" id="x96banner">
        <dt style="display: block;" class="hide">
          <a class="skyscraper-ad" href="http://www.biomedcentral.com/advertisers/digital_advertising">Advertisement</a>
       </dt>
        <dd>
          <!-- OAS AD 'x96' begin -->
          <script type="text/javascript">
            OAS_AD('x96');
          </script>
        </dd>
      </dl>
    </div>  
    
</body>
</html>

